<!DOCTYPE html><html lang="en">
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<title>[0704.2010] A study of structural properties on profile HMMs</title><meta property="og:description" content="Profile hidden Markov Models (pHMMs) are a popular and very useful tool in the detection of the remote homologue protein families. Unfortunately, their performance is not always satisfactory when proteins are in the “t…">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="A study of structural properties on profile HMMs">
<meta name="twitter:image:src" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta name="twitter:image:alt" content="ar5iv logo">
<meta property="og:title" content="A study of structural properties on profile HMMs">
<meta property="og:site_name" content="ar5iv">
<meta property="og:image" content="https://ar5iv.labs.arxiv.org/assets/ar5iv_card.png">
<meta property="og:type" content="article">
<meta property="og:url" content="https://ar5iv.labs.arxiv.org/html/0704.2010">

<!--Generated on Sun Mar 10 12:20:08 2024 by LaTeXML (version 0.8.8) http://dlmf.nist.gov/LaTeXML/.-->
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

<script>
  function detectColorScheme(){
    var theme="light";
    var current_theme = localStorage.getItem("ar5iv_theme");
    if(current_theme){
      if(current_theme == "dark"){
        theme = "dark";
      } }
    else if(!window.matchMedia) { return false; }
    else if(window.matchMedia("(prefers-color-scheme: dark)").matches) {
      theme = "dark"; }
    if (theme=="dark") {
      document.documentElement.setAttribute("data-theme", "dark");
    } else {
      document.documentElement.setAttribute("data-theme", "light"); } }

  detectColorScheme();

  function toggleColorScheme(){
    var current_theme = localStorage.getItem("ar5iv_theme");
    if (current_theme) {
      if (current_theme == "light") {
        localStorage.setItem("ar5iv_theme", "dark"); }
      else {
        localStorage.setItem("ar5iv_theme", "light"); } }
    else {
        localStorage.setItem("ar5iv_theme", "dark"); }
    detectColorScheme(); }
</script>
<link media="all" rel="stylesheet" href="/assets/ar5iv-fonts.0.7.9.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv.0.7.9.min.css"><link media="all" rel="stylesheet" href="/assets/ar5iv-site.0.2.2.css">
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<article class="ltx_document ltx_authors_1line">
<h1 class="ltx_title ltx_title_document">A study of structural properties on profile HMMs</h1>
<div class="ltx_authors">
<span class="ltx_creator ltx_role_author">
<span class="ltx_personname">Juliana S Bernardes <sup id="id8.2.id1" class="ltx_sup">a</sup><span id="footnote1" class="ltx_note ltx_role_footnote"><sup class="ltx_note_mark">1</sup><span class="ltx_note_outer"><span class="ltx_note_content"><sup class="ltx_note_mark">1</sup><span class="ltx_tag ltx_tag_note">1</span>to whom correspondence should be addressed</span></span></span>
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Alberto M. R. Dávila <sup id="id9.2.id1" class="ltx_sup">b</sup>
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Vítor Santos Costa<sup id="id10.2.id1" class="ltx_sup">c</sup>
</span></span>
<span class="ltx_author_before">  </span><span class="ltx_creator ltx_role_author">
<span class="ltx_personname"> Gerson Zaverucha<sup id="id11.5.id1" class="ltx_sup">a</sup>
</span><span class="ltx_author_notes">
<span class="ltx_contact ltx_role_address"><sup id="id12.6.id1" class="ltx_sup">a</sup>COPPE Engenharia de Sistemas e Computação, UFRJ, Rio de Janeiro,
<sup id="id13.7.id2" class="ltx_sup">b</sup>Instituto Oswaldo Cruz, Fiocruz, Rio de Janeiro, Brazil,
<sup id="id14.8.id3" class="ltx_sup">c</sup>CRACS and DCC-FCUP, Universidade do Porto, Portugal
</span></span></span>
</div>
<div class="ltx_dates">(2005; 2005)</div>

<div class="ltx_abstract">
<h6 class="ltx_title ltx_title_abstract">Abstract</h6>
<section id="S1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">1 </span>Motivation:</h2>

<div id="S1.p1" class="ltx_para">
<p id="S1.p1.1" class="ltx_p">Profile hidden Markov Models (pHMMs) are a popular and very useful tool in the detection of the remote homologue protein families. Unfortunately, their performance is not always satisfactory when proteins are in the “twilight zone”. We present HMMER-STRUCT, a model construction algorithm and tool that tries to improve pHMM performance by using structural information while training pHMMs. As a first step, HMMER-STRUCT constructs a set of pHMMs. Each pHMM is constructed by weighting each residue in an aligned protein according to a specific structural property of the residue. Properties used were primary, secondary and tertiary structures, accessibility and packing. HMMER-STRUCT then prioritizes the results by voting.</p>
</div>
<section id="S2" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">2 </span>Results:</h2>

<div id="S2.p1" class="ltx_para">
<p id="S2.p1.1" class="ltx_p">We used the SCOP database to perform our experiments. Throughout, we apply leave-one-family-out cross-validation over protein superfamilies. First, we used the MAMMOTH-mult structural aligner to align the training set proteins. Then, we performed two sets of experiments. In a first experiment, we compared structure weighted models against standard pHMMs and against each other. In a second experiment, we compared the voting model against individual pHMMs. We compare method performance through ROC curves and through Precision/Recall curves, and assess significance through the paired two tailed t-test. Our results show significant performance improvements of all structurally weighted models over default HMMER, and a significant improvement in sensitivity of the combined models over both the original model and the structurally weighted models.</p>
</div>
<section id="S3" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">3 </span>Availability:</h2>

<div id="S3.p1" class="ltx_para">
<p id="S3.p1.1" class="ltx_p">The HMMER-STRUCT tool has been implemented as Perl scripts and as C source code. The structure weighting procedure is available as a patch to the HMMER program. All the test sets, train sets, programs and scripts used in this study are available in
<span id="S3.p1.1.1" class="ltx_ERROR undefined">\href</span>http://wiki.biowebdb.org/index.php/Hmmer-structhttp://wiki.biowebdb.org/index.php/Hmmer-struct.</p>
</div>
<section id="S4" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">4 </span>Contact:</h2>

<div id="S4.p1" class="ltx_para">
<span id="S4.p1.1" class="ltx_ERROR undefined">\href</span>
<p id="S4.p1.2" class="ltx_p">julianab@cos.ufrj.brjulianab@cos.ufrj.br</p>
</div>
</section>
</section>
</section>
</section>
</div>
<section id="S5" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">5 </span>Introduction</h2>

<div id="S5.p1" class="ltx_para">
<p id="S5.p1.1" class="ltx_p">One of the major tasks in computational molecular biology is to aid large-scale protein annotation and biological knowledge discovery. Functional characterization of unknown-function proteins is often inferred through sequence similarity search methods, such as BLAST <cite class="ltx_cite ltx_citemacro_citep">(Altschul <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib2" title="" class="ltx_ref">1990</a>)</cite> and FASTA <cite class="ltx_cite ltx_citemacro_citep">(Pearson,, <a href="#bib.bib34" title="" class="ltx_ref">1985</a>)</cite>. However, when the evolutionary relationship among proteins is distant, methods based on profile hidden Markov models (pHMMs) <cite class="ltx_cite ltx_citemacro_citep">(Eddy,, <a href="#bib.bib14" title="" class="ltx_ref">1996</a>; Krogh <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib27" title="" class="ltx_ref">1994</a>)</cite> are known to outperform methods based on sequence similarity search <cite class="ltx_cite ltx_citemacro_citep">(Gough <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib19" title="" class="ltx_ref">2001</a>; Park <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib33" title="" class="ltx_ref">1998</a>)</cite>.</p>
</div>
<div id="S5.p2" class="ltx_para">
<p id="S5.p2.1" class="ltx_p">Profile Hidden Markov Models are probabilistic models that are often used to represent groups of homolog sequences. These models have been a key tool in protein annotation, and are highly effective for scoring similar sequences. Unfortunately, the performance of pHMMs degrades for sequences in the twilight zone, that is, for homologue sequences with low identity (below 30%). This limitation has motivated a number of different approaches to increase pHMM performance. Proposals include new scoring functions, new null models <cite class="ltx_cite ltx_citemacro_citep">(Karplus <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib26" title="" class="ltx_ref">2005</a>)</cite> and prior probability <cite class="ltx_cite ltx_citemacro_citep">(Brown <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib10" title="" class="ltx_ref">1993</a>)</cite>. Researchers have also combined other information with pHMMs: T-HMM <cite class="ltx_cite ltx_citemacro_citep">(Qian <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib36" title="" class="ltx_ref">2004</a>)</cite> uses phylogenetic information; HMM-STR <cite class="ltx_cite ltx_citemacro_citep">(Bystroff <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib11" title="" class="ltx_ref">2000</a>)</cite>, combines pHMMs and support vector machines <cite class="ltx_cite ltx_citemacro_citep">(Schölkopf <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib38" title="" class="ltx_ref">1999</a>)</cite>.</p>
</div>
<div id="S5.p3" class="ltx_para">
<p id="S5.p3.1" class="ltx_p">The observation that homologue proteins tend to preserve structure suggests that structural information should be extremely relevant in detecting homologues. In fact, it has been shown that pHMMs trained with multiple sequence alignments based on protein’s structural alignment can have better performance than pHMMs based on state-of-the art aligners that apply primary sequence information only, when remote homology detections are assessed <cite class="ltx_cite ltx_citemacro_citep">(Bernardes <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib8" title="" class="ltx_ref">2007</a>)</cite>. In this vein, researchers have proposed special alphabets to represent structural elements in pHMMs <cite class="ltx_cite ltx_citemacro_citep">(Goyon <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib20" title="" class="ltx_ref">2004</a>)</cite>, or modifying pHMM structure to add protein three-dimensional information <cite class="ltx_cite ltx_citemacro_citep">(Alexandrov <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib1" title="" class="ltx_ref">2004</a>)</cite>. Although such methods are more powerful than pHMMs, arguably they are computationally more expensive both in training and in classification, and to the best of our knowledge have not become widely used.</p>
</div>
<div id="S5.p4" class="ltx_para">
<p id="S5.p4.1" class="ltx_p">We present a novel method to apply structural information in protein classification. In contrast to the previous approaches, our method relies on pHMMs. Our main contribution is a residue weighting-algorithm that incorporates protein structural information into pHMMs. Further, we apply different structural properties to train a library of 5 pHMMs from a homologue protein set. The properties we consider are primary, secondary, and tertiary structure, also used in previous methods. We also apply two properties that, to the best of our knowledge, have not been used in this task before, but that are often important in this domain: solvent accessibility and residue packing. The classification of a unknown-function protein
is then obtained by combining the classification from the library of pHMMs. The main advantage of our method is that structural information is only used to train the pHMMs. Notice that scoring is still performed using sequence data, as opposed to <cite class="ltx_cite ltx_citemacro_citep">(Alexandrov <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib1" title="" class="ltx_ref">2004</a>)</cite>. Our method was implemented by extending the HMMER package and experimental evaluation using the SCOP database showed significant improvement over HMMER.</p>
</div>
<div id="S5.p5" class="ltx_para">
<span id="S5.p5.1" class="ltx_ERROR undefined">{methods}</span>
</div>
</section>
<section id="S6" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">6 </span>Methods</h2>

<div id="S6.p1" class="ltx_para">
<p id="S6.p1.3" class="ltx_p">In our experiments, a protein homologue set is aligned by utilizing the MAMMOTH-mult <cite class="ltx_cite ltx_citemacro_citep">(Attwood <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib5" title="" class="ltx_ref">2005</a>)</cite> aligner. MAMMOTH produces two outputs: one represents the multiple sequence alignment (based on spatial coordinates’ similarity) and the other the structural alignment. These outputs are used to build weight matrices <math id="S6.p1.1.m1.1" class="ltx_Math" alttext="M^{{}^{\prime}}_{s}" display="inline"><semantics id="S6.p1.1.m1.1a"><msubsup id="S6.p1.1.m1.1.1" xref="S6.p1.1.m1.1.1.cmml"><mi id="S6.p1.1.m1.1.1.2.2" xref="S6.p1.1.m1.1.1.2.2.cmml">M</mi><mi id="S6.p1.1.m1.1.1.3" xref="S6.p1.1.m1.1.1.3.cmml">s</mi><msup id="S6.p1.1.m1.1.1.2.3" xref="S6.p1.1.m1.1.1.2.3.cmml"><mi id="S6.p1.1.m1.1.1.2.3a" xref="S6.p1.1.m1.1.1.2.3.cmml"></mi><mo id="S6.p1.1.m1.1.1.2.3.1" xref="S6.p1.1.m1.1.1.2.3.1.cmml">′</mo></msup></msubsup><annotation-xml encoding="MathML-Content" id="S6.p1.1.m1.1b"><apply id="S6.p1.1.m1.1.1.cmml" xref="S6.p1.1.m1.1.1"><csymbol cd="ambiguous" id="S6.p1.1.m1.1.1.1.cmml" xref="S6.p1.1.m1.1.1">subscript</csymbol><apply id="S6.p1.1.m1.1.1.2.cmml" xref="S6.p1.1.m1.1.1"><csymbol cd="ambiguous" id="S6.p1.1.m1.1.1.2.1.cmml" xref="S6.p1.1.m1.1.1">superscript</csymbol><ci id="S6.p1.1.m1.1.1.2.2.cmml" xref="S6.p1.1.m1.1.1.2.2">𝑀</ci><apply id="S6.p1.1.m1.1.1.2.3.cmml" xref="S6.p1.1.m1.1.1.2.3"><ci id="S6.p1.1.m1.1.1.2.3.1.cmml" xref="S6.p1.1.m1.1.1.2.3.1">′</ci></apply></apply><ci id="S6.p1.1.m1.1.1.3.cmml" xref="S6.p1.1.m1.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.p1.1.m1.1c">M^{{}^{\prime}}_{s}</annotation></semantics></math>, which represent residue structural weights for each protein. These matrices were used on pHMM training stage. The section <a href="#S6.SS2" title="6.2 Sequence Weighting ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">6.2</span></a> will give more details on the building of <math id="S6.p1.2.m2.1" class="ltx_Math" alttext="M^{{}^{\prime}}_{s}" display="inline"><semantics id="S6.p1.2.m2.1a"><msubsup id="S6.p1.2.m2.1.1" xref="S6.p1.2.m2.1.1.cmml"><mi id="S6.p1.2.m2.1.1.2.2" xref="S6.p1.2.m2.1.1.2.2.cmml">M</mi><mi id="S6.p1.2.m2.1.1.3" xref="S6.p1.2.m2.1.1.3.cmml">s</mi><msup id="S6.p1.2.m2.1.1.2.3" xref="S6.p1.2.m2.1.1.2.3.cmml"><mi id="S6.p1.2.m2.1.1.2.3a" xref="S6.p1.2.m2.1.1.2.3.cmml"></mi><mo id="S6.p1.2.m2.1.1.2.3.1" xref="S6.p1.2.m2.1.1.2.3.1.cmml">′</mo></msup></msubsup><annotation-xml encoding="MathML-Content" id="S6.p1.2.m2.1b"><apply id="S6.p1.2.m2.1.1.cmml" xref="S6.p1.2.m2.1.1"><csymbol cd="ambiguous" id="S6.p1.2.m2.1.1.1.cmml" xref="S6.p1.2.m2.1.1">subscript</csymbol><apply id="S6.p1.2.m2.1.1.2.cmml" xref="S6.p1.2.m2.1.1"><csymbol cd="ambiguous" id="S6.p1.2.m2.1.1.2.1.cmml" xref="S6.p1.2.m2.1.1">superscript</csymbol><ci id="S6.p1.2.m2.1.1.2.2.cmml" xref="S6.p1.2.m2.1.1.2.2">𝑀</ci><apply id="S6.p1.2.m2.1.1.2.3.cmml" xref="S6.p1.2.m2.1.1.2.3"><ci id="S6.p1.2.m2.1.1.2.3.1.cmml" xref="S6.p1.2.m2.1.1.2.3.1">′</ci></apply></apply><ci id="S6.p1.2.m2.1.1.3.cmml" xref="S6.p1.2.m2.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.p1.2.m2.1c">M^{{}^{\prime}}_{s}</annotation></semantics></math>. Basically, our approach builds five pHMMs. The simplest pHMM is built from MAMMOTH’s multiple sequence alignment, by keeping the default sequence-weighting algorithm of HMMER. This model is called <span id="S6.p1.3.1" class="ltx_text ltx_font_italic">pHMM1D</span>. In order to aid to build the remaining pHMMs our approach generates <math id="S6.p1.3.m3.1" class="ltx_Math" alttext="M^{{}^{\prime}}_{s}" display="inline"><semantics id="S6.p1.3.m3.1a"><msubsup id="S6.p1.3.m3.1.1" xref="S6.p1.3.m3.1.1.cmml"><mi id="S6.p1.3.m3.1.1.2.2" xref="S6.p1.3.m3.1.1.2.2.cmml">M</mi><mi id="S6.p1.3.m3.1.1.3" xref="S6.p1.3.m3.1.1.3.cmml">s</mi><msup id="S6.p1.3.m3.1.1.2.3" xref="S6.p1.3.m3.1.1.2.3.cmml"><mi id="S6.p1.3.m3.1.1.2.3a" xref="S6.p1.3.m3.1.1.2.3.cmml"></mi><mo id="S6.p1.3.m3.1.1.2.3.1" xref="S6.p1.3.m3.1.1.2.3.1.cmml">′</mo></msup></msubsup><annotation-xml encoding="MathML-Content" id="S6.p1.3.m3.1b"><apply id="S6.p1.3.m3.1.1.cmml" xref="S6.p1.3.m3.1.1"><csymbol cd="ambiguous" id="S6.p1.3.m3.1.1.1.cmml" xref="S6.p1.3.m3.1.1">subscript</csymbol><apply id="S6.p1.3.m3.1.1.2.cmml" xref="S6.p1.3.m3.1.1"><csymbol cd="ambiguous" id="S6.p1.3.m3.1.1.2.1.cmml" xref="S6.p1.3.m3.1.1">superscript</csymbol><ci id="S6.p1.3.m3.1.1.2.2.cmml" xref="S6.p1.3.m3.1.1.2.2">𝑀</ci><apply id="S6.p1.3.m3.1.1.2.3.cmml" xref="S6.p1.3.m3.1.1.2.3"><ci id="S6.p1.3.m3.1.1.2.3.1.cmml" xref="S6.p1.3.m3.1.1.2.3.1">′</ci></apply></apply><ci id="S6.p1.3.m3.1.1.3.cmml" xref="S6.p1.3.m3.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.p1.3.m3.1c">M^{{}^{\prime}}_{s}</annotation></semantics></math>. The matrices used in building <span id="S6.p1.3.2" class="ltx_text ltx_font_italic">pHMM2D</span>, <span id="S6.p1.3.3" class="ltx_text ltx_font_italic">pHMMAcc</span> and <span id="S6.p1.3.4" class="ltx_text ltx_font_italic">pHMMOi</span>, incorporate secondary structure, residue solvent accessibility, and residue packing information, respectively. In order to build these matrices were used both MAMMOTH sequence alignment plus structural properties obtained using the <span id="S6.p1.3.5" class="ltx_text ltx_font_italic">joy</span> package <cite class="ltx_cite ltx_citemacro_citep">(Mizuguchi <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib31" title="" class="ltx_ref">1998</a>)</cite> to collect these information from PDB coordinates <cite class="ltx_cite ltx_citemacro_citep">(Helen <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib23" title="" class="ltx_ref">2000</a>)</cite>. Last, we used MAMMOTH’s multiple sequence alignment plus structural alignment to build a matrix based on homologue core structures <cite class="ltx_cite ltx_citemacro_citep">(Matsuo <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib29" title="" class="ltx_ref">1999</a>)</cite>. That matrix was used to build <span id="S6.p1.3.6" class="ltx_text ltx_font_italic">pHMM3D</span>. Figure <a href="#S6.F1" title="Figure 1 ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a> shows the proposed method.</p>
</div>
<figure id="S6.F1" class="ltx_figure"><img src="/html/0704.2010/assets/x1.png" id="S6.F1.g1" class="ltx_graphics ltx_centering ltx_img_portrait" width="236" height="305" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 1: </span>First, a homologue protein set is aligned by using MAMMOTH-mult aligner. The aligner produces a multiple sequence alignment and a structural alignment. The multiple sequence alignment is used to build a conventional pHMM using HMMER package, <em id="S6.F1.8.1" class="ltx_emph ltx_font_italic">pHMM1D</em>. Aligned sequences are fed to the <span id="S6.F1.9.2" class="ltx_text ltx_font_italic">joy</span> tool. The <span id="S6.F1.10.3" class="ltx_text ltx_font_italic">joy</span> output is used to construct weight matrices, which are then used to build secondary <em id="S6.F1.11.4" class="ltx_emph ltx_font_italic">pHMM2D</em>, accessibility <em id="S6.F1.12.5" class="ltx_emph ltx_font_italic">pHMMAcc</em>, and packing <em id="S6.F1.13.6" class="ltx_emph ltx_font_italic">pHMMOi</em> models. Finally, the structural alignment is used to find the homologue core structure, which is then used to construct <em id="S6.F1.14.7" class="ltx_emph ltx_font_italic">pHMM3D</em>.</figcaption>
</figure>
<section id="S6.SS1" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">6.1 </span>Profile HMMs</h3>

<div id="S6.SS1.p1" class="ltx_para">
<p id="S6.SS1.p1.1" class="ltx_p">Profile HMMs represent conserved regions in sequences as sequences of <em id="S6.SS1.p1.1.1" class="ltx_emph ltx_font_italic">match</em> (M) states. Inserted material is represented as <em id="S6.SS1.p1.1.2" class="ltx_emph ltx_font_italic">insert</em> states (I), and deleted regions as <em id="S6.SS1.p1.1.3" class="ltx_emph ltx_font_italic">delete</em> states (D). The parameters of pHMMs are probabilities of two events: a <em id="S6.SS1.p1.1.4" class="ltx_emph ltx_font_italic">transition probability</em> from a state to another state, and a probability that a specific state will emit a specific residue (say, a specific amino-acid when comparing proteins), called <em id="S6.SS1.p1.1.5" class="ltx_emph ltx_font_italic">emission probability</em>. Obviously, only match and insert states generate characters and have an emission probability distribution; delete states are quiet. In the case of proteins, emission distributions have 20 entries, one per amino-acid.</p>
</div>
<div id="S6.SS1.p2" class="ltx_para">
<p id="S6.SS1.p2.2" class="ltx_p">Possible transitions define the structure of the pHMM. Systems such as SAM <cite class="ltx_cite ltx_citemacro_citep">(Hughey <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib25" title="" class="ltx_ref">1996</a>)</cite> allow transitions between all types of states, totaling 3 transitions per state, hence 9 per node. On the other hand, the HMMER system relies on the Plan7 model <cite class="ltx_cite ltx_citemacro_citep">(Eddy,, <a href="#bib.bib15" title="" class="ltx_ref">1998</a>)</cite>, which disallows <math id="S6.SS1.p2.1.m1.1" class="ltx_Math" alttext="I\rightarrow D" display="inline"><semantics id="S6.SS1.p2.1.m1.1a"><mrow id="S6.SS1.p2.1.m1.1.1" xref="S6.SS1.p2.1.m1.1.1.cmml"><mi id="S6.SS1.p2.1.m1.1.1.2" xref="S6.SS1.p2.1.m1.1.1.2.cmml">I</mi><mo stretchy="false" id="S6.SS1.p2.1.m1.1.1.1" xref="S6.SS1.p2.1.m1.1.1.1.cmml">→</mo><mi id="S6.SS1.p2.1.m1.1.1.3" xref="S6.SS1.p2.1.m1.1.1.3.cmml">D</mi></mrow><annotation-xml encoding="MathML-Content" id="S6.SS1.p2.1.m1.1b"><apply id="S6.SS1.p2.1.m1.1.1.cmml" xref="S6.SS1.p2.1.m1.1.1"><ci id="S6.SS1.p2.1.m1.1.1.1.cmml" xref="S6.SS1.p2.1.m1.1.1.1">→</ci><ci id="S6.SS1.p2.1.m1.1.1.2.cmml" xref="S6.SS1.p2.1.m1.1.1.2">𝐼</ci><ci id="S6.SS1.p2.1.m1.1.1.3.cmml" xref="S6.SS1.p2.1.m1.1.1.3">𝐷</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p2.1.m1.1c">I\rightarrow D</annotation></semantics></math> and <math id="S6.SS1.p2.2.m2.1" class="ltx_Math" alttext="D\rightarrow I" display="inline"><semantics id="S6.SS1.p2.2.m2.1a"><mrow id="S6.SS1.p2.2.m2.1.1" xref="S6.SS1.p2.2.m2.1.1.cmml"><mi id="S6.SS1.p2.2.m2.1.1.2" xref="S6.SS1.p2.2.m2.1.1.2.cmml">D</mi><mo stretchy="false" id="S6.SS1.p2.2.m2.1.1.1" xref="S6.SS1.p2.2.m2.1.1.1.cmml">→</mo><mi id="S6.SS1.p2.2.m2.1.1.3" xref="S6.SS1.p2.2.m2.1.1.3.cmml">I</mi></mrow><annotation-xml encoding="MathML-Content" id="S6.SS1.p2.2.m2.1b"><apply id="S6.SS1.p2.2.m2.1.1.cmml" xref="S6.SS1.p2.2.m2.1.1"><ci id="S6.SS1.p2.2.m2.1.1.1.cmml" xref="S6.SS1.p2.2.m2.1.1.1">→</ci><ci id="S6.SS1.p2.2.m2.1.1.2.cmml" xref="S6.SS1.p2.2.m2.1.1.2">𝐷</ci><ci id="S6.SS1.p2.2.m2.1.1.3.cmml" xref="S6.SS1.p2.2.m2.1.1.3">𝐼</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p2.2.m2.1c">D\rightarrow I</annotation></semantics></math> transitions.</p>
</div>
<div id="S6.SS1.p3" class="ltx_para">
<p id="S6.SS1.p3.5" class="ltx_p">Emission probabilities are calculated by the equation <a href="#S6.E1" title="In 6.1 Profile HMMs ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>, where <math id="S6.SS1.p3.1.m1.1" class="ltx_Math" alttext="c_{j}(\sigma)" display="inline"><semantics id="S6.SS1.p3.1.m1.1a"><mrow id="S6.SS1.p3.1.m1.1.2" xref="S6.SS1.p3.1.m1.1.2.cmml"><msub id="S6.SS1.p3.1.m1.1.2.2" xref="S6.SS1.p3.1.m1.1.2.2.cmml"><mi id="S6.SS1.p3.1.m1.1.2.2.2" xref="S6.SS1.p3.1.m1.1.2.2.2.cmml">c</mi><mi id="S6.SS1.p3.1.m1.1.2.2.3" xref="S6.SS1.p3.1.m1.1.2.2.3.cmml">j</mi></msub><mo lspace="0em" rspace="0em" id="S6.SS1.p3.1.m1.1.2.1" xref="S6.SS1.p3.1.m1.1.2.1.cmml">​</mo><mrow id="S6.SS1.p3.1.m1.1.2.3.2" xref="S6.SS1.p3.1.m1.1.2.cmml"><mo stretchy="false" id="S6.SS1.p3.1.m1.1.2.3.2.1" xref="S6.SS1.p3.1.m1.1.2.cmml">(</mo><mi id="S6.SS1.p3.1.m1.1.1" xref="S6.SS1.p3.1.m1.1.1.cmml">σ</mi><mo stretchy="false" id="S6.SS1.p3.1.m1.1.2.3.2.2" xref="S6.SS1.p3.1.m1.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS1.p3.1.m1.1b"><apply id="S6.SS1.p3.1.m1.1.2.cmml" xref="S6.SS1.p3.1.m1.1.2"><times id="S6.SS1.p3.1.m1.1.2.1.cmml" xref="S6.SS1.p3.1.m1.1.2.1"></times><apply id="S6.SS1.p3.1.m1.1.2.2.cmml" xref="S6.SS1.p3.1.m1.1.2.2"><csymbol cd="ambiguous" id="S6.SS1.p3.1.m1.1.2.2.1.cmml" xref="S6.SS1.p3.1.m1.1.2.2">subscript</csymbol><ci id="S6.SS1.p3.1.m1.1.2.2.2.cmml" xref="S6.SS1.p3.1.m1.1.2.2.2">𝑐</ci><ci id="S6.SS1.p3.1.m1.1.2.2.3.cmml" xref="S6.SS1.p3.1.m1.1.2.2.3">𝑗</ci></apply><ci id="S6.SS1.p3.1.m1.1.1.cmml" xref="S6.SS1.p3.1.m1.1.1">𝜎</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p3.1.m1.1c">c_{j}(\sigma)</annotation></semantics></math> is the observed frequency of residue <math id="S6.SS1.p3.2.m2.1" class="ltx_Math" alttext="\sigma" display="inline"><semantics id="S6.SS1.p3.2.m2.1a"><mi id="S6.SS1.p3.2.m2.1.1" xref="S6.SS1.p3.2.m2.1.1.cmml">σ</mi><annotation-xml encoding="MathML-Content" id="S6.SS1.p3.2.m2.1b"><ci id="S6.SS1.p3.2.m2.1.1.cmml" xref="S6.SS1.p3.2.m2.1.1">𝜎</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p3.2.m2.1c">\sigma</annotation></semantics></math> in <math id="S6.SS1.p3.3.m3.1" class="ltx_Math" alttext="j" display="inline"><semantics id="S6.SS1.p3.3.m3.1a"><mi id="S6.SS1.p3.3.m3.1.1" xref="S6.SS1.p3.3.m3.1.1.cmml">j</mi><annotation-xml encoding="MathML-Content" id="S6.SS1.p3.3.m3.1b"><ci id="S6.SS1.p3.3.m3.1.1.cmml" xref="S6.SS1.p3.3.m3.1.1">𝑗</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p3.3.m3.1c">j</annotation></semantics></math> column of the alignment, and <math id="S6.SS1.p3.4.m4.1" class="ltx_Math" alttext="\alpha(\sigma)" display="inline"><semantics id="S6.SS1.p3.4.m4.1a"><mrow id="S6.SS1.p3.4.m4.1.2" xref="S6.SS1.p3.4.m4.1.2.cmml"><mi id="S6.SS1.p3.4.m4.1.2.2" xref="S6.SS1.p3.4.m4.1.2.2.cmml">α</mi><mo lspace="0em" rspace="0em" id="S6.SS1.p3.4.m4.1.2.1" xref="S6.SS1.p3.4.m4.1.2.1.cmml">​</mo><mrow id="S6.SS1.p3.4.m4.1.2.3.2" xref="S6.SS1.p3.4.m4.1.2.cmml"><mo stretchy="false" id="S6.SS1.p3.4.m4.1.2.3.2.1" xref="S6.SS1.p3.4.m4.1.2.cmml">(</mo><mi id="S6.SS1.p3.4.m4.1.1" xref="S6.SS1.p3.4.m4.1.1.cmml">σ</mi><mo stretchy="false" id="S6.SS1.p3.4.m4.1.2.3.2.2" xref="S6.SS1.p3.4.m4.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS1.p3.4.m4.1b"><apply id="S6.SS1.p3.4.m4.1.2.cmml" xref="S6.SS1.p3.4.m4.1.2"><times id="S6.SS1.p3.4.m4.1.2.1.cmml" xref="S6.SS1.p3.4.m4.1.2.1"></times><ci id="S6.SS1.p3.4.m4.1.2.2.cmml" xref="S6.SS1.p3.4.m4.1.2.2">𝛼</ci><ci id="S6.SS1.p3.4.m4.1.1.cmml" xref="S6.SS1.p3.4.m4.1.1">𝜎</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p3.4.m4.1c">\alpha(\sigma)</annotation></semantics></math> represents the pseudo counts of residue <math id="S6.SS1.p3.5.m5.1" class="ltx_Math" alttext="\sigma" display="inline"><semantics id="S6.SS1.p3.5.m5.1a"><mi id="S6.SS1.p3.5.m5.1.1" xref="S6.SS1.p3.5.m5.1.1.cmml">σ</mi><annotation-xml encoding="MathML-Content" id="S6.SS1.p3.5.m5.1b"><ci id="S6.SS1.p3.5.m5.1.1.cmml" xref="S6.SS1.p3.5.m5.1.1">𝜎</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p3.5.m5.1c">\sigma</annotation></semantics></math>, which are obtained from Dirichlet mixtures, as seen in <cite class="ltx_cite ltx_citemacro_citep">(Brown <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib10" title="" class="ltx_ref">1993</a>)</cite>.</p>
</div>
<div id="S6.SS1.p4" class="ltx_para">
<table id="S6.E1" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S6.E1.m1.5" class="ltx_Math" alttext="\\
e_{j}(\sigma)=\dfrac{c_{j}(\sigma)+\alpha(\sigma)}{\sum_{k}c_{j}(\sigma_{k})+\alpha(\sigma_{k})}" display="block"><semantics id="S6.E1.m1.5a"><mrow id="S6.E1.m1.5.6" xref="S6.E1.m1.5.6.cmml"><mrow id="S6.E1.m1.5.6.2" xref="S6.E1.m1.5.6.2.cmml"><msub id="S6.E1.m1.5.6.2.2" xref="S6.E1.m1.5.6.2.2.cmml"><mi id="S6.E1.m1.5.6.2.2.2" xref="S6.E1.m1.5.6.2.2.2.cmml">e</mi><mi id="S6.E1.m1.5.6.2.2.3" xref="S6.E1.m1.5.6.2.2.3.cmml">j</mi></msub><mo lspace="0em" rspace="0em" id="S6.E1.m1.5.6.2.1" xref="S6.E1.m1.5.6.2.1.cmml">​</mo><mrow id="S6.E1.m1.5.6.2.3.2" xref="S6.E1.m1.5.6.2.cmml"><mo stretchy="false" id="S6.E1.m1.5.6.2.3.2.1" xref="S6.E1.m1.5.6.2.cmml">(</mo><mi id="S6.E1.m1.5.5" xref="S6.E1.m1.5.5.cmml">σ</mi><mo stretchy="false" id="S6.E1.m1.5.6.2.3.2.2" xref="S6.E1.m1.5.6.2.cmml">)</mo></mrow></mrow><mo id="S6.E1.m1.5.6.1" xref="S6.E1.m1.5.6.1.cmml">=</mo><mfrac id="S6.E1.m1.4.4" xref="S6.E1.m1.4.4.cmml"><mrow id="S6.E1.m1.2.2.2" xref="S6.E1.m1.2.2.2.cmml"><mrow id="S6.E1.m1.2.2.2.4" xref="S6.E1.m1.2.2.2.4.cmml"><msub id="S6.E1.m1.2.2.2.4.2" xref="S6.E1.m1.2.2.2.4.2.cmml"><mi id="S6.E1.m1.2.2.2.4.2.2" xref="S6.E1.m1.2.2.2.4.2.2.cmml">c</mi><mi id="S6.E1.m1.2.2.2.4.2.3" xref="S6.E1.m1.2.2.2.4.2.3.cmml">j</mi></msub><mo lspace="0em" rspace="0em" id="S6.E1.m1.2.2.2.4.1" xref="S6.E1.m1.2.2.2.4.1.cmml">​</mo><mrow id="S6.E1.m1.2.2.2.4.3.2" xref="S6.E1.m1.2.2.2.4.cmml"><mo stretchy="false" id="S6.E1.m1.2.2.2.4.3.2.1" xref="S6.E1.m1.2.2.2.4.cmml">(</mo><mi id="S6.E1.m1.1.1.1.1" xref="S6.E1.m1.1.1.1.1.cmml">σ</mi><mo stretchy="false" id="S6.E1.m1.2.2.2.4.3.2.2" xref="S6.E1.m1.2.2.2.4.cmml">)</mo></mrow></mrow><mo id="S6.E1.m1.2.2.2.3" xref="S6.E1.m1.2.2.2.3.cmml">+</mo><mrow id="S6.E1.m1.2.2.2.5" xref="S6.E1.m1.2.2.2.5.cmml"><mi id="S6.E1.m1.2.2.2.5.2" xref="S6.E1.m1.2.2.2.5.2.cmml">α</mi><mo lspace="0em" rspace="0em" id="S6.E1.m1.2.2.2.5.1" xref="S6.E1.m1.2.2.2.5.1.cmml">​</mo><mrow id="S6.E1.m1.2.2.2.5.3.2" xref="S6.E1.m1.2.2.2.5.cmml"><mo stretchy="false" id="S6.E1.m1.2.2.2.5.3.2.1" xref="S6.E1.m1.2.2.2.5.cmml">(</mo><mi id="S6.E1.m1.2.2.2.2" xref="S6.E1.m1.2.2.2.2.cmml">σ</mi><mo stretchy="false" id="S6.E1.m1.2.2.2.5.3.2.2" xref="S6.E1.m1.2.2.2.5.cmml">)</mo></mrow></mrow></mrow><mrow id="S6.E1.m1.4.4.4" xref="S6.E1.m1.4.4.4.cmml"><mrow id="S6.E1.m1.3.3.3.1" xref="S6.E1.m1.3.3.3.1.cmml"><msub id="S6.E1.m1.3.3.3.1.2" xref="S6.E1.m1.3.3.3.1.2.cmml"><mo id="S6.E1.m1.3.3.3.1.2.2" xref="S6.E1.m1.3.3.3.1.2.2.cmml">∑</mo><mi id="S6.E1.m1.3.3.3.1.2.3" xref="S6.E1.m1.3.3.3.1.2.3.cmml">k</mi></msub><mrow id="S6.E1.m1.3.3.3.1.1" xref="S6.E1.m1.3.3.3.1.1.cmml"><msub id="S6.E1.m1.3.3.3.1.1.3" xref="S6.E1.m1.3.3.3.1.1.3.cmml"><mi id="S6.E1.m1.3.3.3.1.1.3.2" xref="S6.E1.m1.3.3.3.1.1.3.2.cmml">c</mi><mi id="S6.E1.m1.3.3.3.1.1.3.3" xref="S6.E1.m1.3.3.3.1.1.3.3.cmml">j</mi></msub><mo lspace="0em" rspace="0em" id="S6.E1.m1.3.3.3.1.1.2" xref="S6.E1.m1.3.3.3.1.1.2.cmml">​</mo><mrow id="S6.E1.m1.3.3.3.1.1.1.1" xref="S6.E1.m1.3.3.3.1.1.1.1.1.cmml"><mo stretchy="false" id="S6.E1.m1.3.3.3.1.1.1.1.2" xref="S6.E1.m1.3.3.3.1.1.1.1.1.cmml">(</mo><msub id="S6.E1.m1.3.3.3.1.1.1.1.1" xref="S6.E1.m1.3.3.3.1.1.1.1.1.cmml"><mi id="S6.E1.m1.3.3.3.1.1.1.1.1.2" xref="S6.E1.m1.3.3.3.1.1.1.1.1.2.cmml">σ</mi><mi id="S6.E1.m1.3.3.3.1.1.1.1.1.3" xref="S6.E1.m1.3.3.3.1.1.1.1.1.3.cmml">k</mi></msub><mo stretchy="false" id="S6.E1.m1.3.3.3.1.1.1.1.3" xref="S6.E1.m1.3.3.3.1.1.1.1.1.cmml">)</mo></mrow></mrow></mrow><mo id="S6.E1.m1.4.4.4.3" xref="S6.E1.m1.4.4.4.3.cmml">+</mo><mrow id="S6.E1.m1.4.4.4.2" xref="S6.E1.m1.4.4.4.2.cmml"><mi id="S6.E1.m1.4.4.4.2.3" xref="S6.E1.m1.4.4.4.2.3.cmml">α</mi><mo lspace="0em" rspace="0em" id="S6.E1.m1.4.4.4.2.2" xref="S6.E1.m1.4.4.4.2.2.cmml">​</mo><mrow id="S6.E1.m1.4.4.4.2.1.1" xref="S6.E1.m1.4.4.4.2.1.1.1.cmml"><mo stretchy="false" id="S6.E1.m1.4.4.4.2.1.1.2" xref="S6.E1.m1.4.4.4.2.1.1.1.cmml">(</mo><msub id="S6.E1.m1.4.4.4.2.1.1.1" xref="S6.E1.m1.4.4.4.2.1.1.1.cmml"><mi id="S6.E1.m1.4.4.4.2.1.1.1.2" xref="S6.E1.m1.4.4.4.2.1.1.1.2.cmml">σ</mi><mi id="S6.E1.m1.4.4.4.2.1.1.1.3" xref="S6.E1.m1.4.4.4.2.1.1.1.3.cmml">k</mi></msub><mo stretchy="false" id="S6.E1.m1.4.4.4.2.1.1.3" xref="S6.E1.m1.4.4.4.2.1.1.1.cmml">)</mo></mrow></mrow></mrow></mfrac></mrow><annotation-xml encoding="MathML-Content" id="S6.E1.m1.5b"><apply id="S6.E1.m1.5.6.cmml" xref="S6.E1.m1.5.6"><eq id="S6.E1.m1.5.6.1.cmml" xref="S6.E1.m1.5.6.1"></eq><apply id="S6.E1.m1.5.6.2.cmml" xref="S6.E1.m1.5.6.2"><times id="S6.E1.m1.5.6.2.1.cmml" xref="S6.E1.m1.5.6.2.1"></times><apply id="S6.E1.m1.5.6.2.2.cmml" xref="S6.E1.m1.5.6.2.2"><csymbol cd="ambiguous" id="S6.E1.m1.5.6.2.2.1.cmml" xref="S6.E1.m1.5.6.2.2">subscript</csymbol><ci id="S6.E1.m1.5.6.2.2.2.cmml" xref="S6.E1.m1.5.6.2.2.2">𝑒</ci><ci id="S6.E1.m1.5.6.2.2.3.cmml" xref="S6.E1.m1.5.6.2.2.3">𝑗</ci></apply><ci id="S6.E1.m1.5.5.cmml" xref="S6.E1.m1.5.5">𝜎</ci></apply><apply id="S6.E1.m1.4.4.cmml" xref="S6.E1.m1.4.4"><divide id="S6.E1.m1.4.4.5.cmml" xref="S6.E1.m1.4.4"></divide><apply id="S6.E1.m1.2.2.2.cmml" xref="S6.E1.m1.2.2.2"><plus id="S6.E1.m1.2.2.2.3.cmml" xref="S6.E1.m1.2.2.2.3"></plus><apply id="S6.E1.m1.2.2.2.4.cmml" xref="S6.E1.m1.2.2.2.4"><times id="S6.E1.m1.2.2.2.4.1.cmml" xref="S6.E1.m1.2.2.2.4.1"></times><apply id="S6.E1.m1.2.2.2.4.2.cmml" xref="S6.E1.m1.2.2.2.4.2"><csymbol cd="ambiguous" id="S6.E1.m1.2.2.2.4.2.1.cmml" xref="S6.E1.m1.2.2.2.4.2">subscript</csymbol><ci id="S6.E1.m1.2.2.2.4.2.2.cmml" xref="S6.E1.m1.2.2.2.4.2.2">𝑐</ci><ci id="S6.E1.m1.2.2.2.4.2.3.cmml" xref="S6.E1.m1.2.2.2.4.2.3">𝑗</ci></apply><ci id="S6.E1.m1.1.1.1.1.cmml" xref="S6.E1.m1.1.1.1.1">𝜎</ci></apply><apply id="S6.E1.m1.2.2.2.5.cmml" xref="S6.E1.m1.2.2.2.5"><times id="S6.E1.m1.2.2.2.5.1.cmml" xref="S6.E1.m1.2.2.2.5.1"></times><ci id="S6.E1.m1.2.2.2.5.2.cmml" xref="S6.E1.m1.2.2.2.5.2">𝛼</ci><ci id="S6.E1.m1.2.2.2.2.cmml" xref="S6.E1.m1.2.2.2.2">𝜎</ci></apply></apply><apply id="S6.E1.m1.4.4.4.cmml" xref="S6.E1.m1.4.4.4"><plus id="S6.E1.m1.4.4.4.3.cmml" xref="S6.E1.m1.4.4.4.3"></plus><apply id="S6.E1.m1.3.3.3.1.cmml" xref="S6.E1.m1.3.3.3.1"><apply id="S6.E1.m1.3.3.3.1.2.cmml" xref="S6.E1.m1.3.3.3.1.2"><csymbol cd="ambiguous" id="S6.E1.m1.3.3.3.1.2.1.cmml" xref="S6.E1.m1.3.3.3.1.2">subscript</csymbol><sum id="S6.E1.m1.3.3.3.1.2.2.cmml" xref="S6.E1.m1.3.3.3.1.2.2"></sum><ci id="S6.E1.m1.3.3.3.1.2.3.cmml" xref="S6.E1.m1.3.3.3.1.2.3">𝑘</ci></apply><apply id="S6.E1.m1.3.3.3.1.1.cmml" xref="S6.E1.m1.3.3.3.1.1"><times id="S6.E1.m1.3.3.3.1.1.2.cmml" xref="S6.E1.m1.3.3.3.1.1.2"></times><apply id="S6.E1.m1.3.3.3.1.1.3.cmml" xref="S6.E1.m1.3.3.3.1.1.3"><csymbol cd="ambiguous" id="S6.E1.m1.3.3.3.1.1.3.1.cmml" xref="S6.E1.m1.3.3.3.1.1.3">subscript</csymbol><ci id="S6.E1.m1.3.3.3.1.1.3.2.cmml" xref="S6.E1.m1.3.3.3.1.1.3.2">𝑐</ci><ci id="S6.E1.m1.3.3.3.1.1.3.3.cmml" xref="S6.E1.m1.3.3.3.1.1.3.3">𝑗</ci></apply><apply id="S6.E1.m1.3.3.3.1.1.1.1.1.cmml" xref="S6.E1.m1.3.3.3.1.1.1.1"><csymbol cd="ambiguous" id="S6.E1.m1.3.3.3.1.1.1.1.1.1.cmml" xref="S6.E1.m1.3.3.3.1.1.1.1">subscript</csymbol><ci id="S6.E1.m1.3.3.3.1.1.1.1.1.2.cmml" xref="S6.E1.m1.3.3.3.1.1.1.1.1.2">𝜎</ci><ci id="S6.E1.m1.3.3.3.1.1.1.1.1.3.cmml" xref="S6.E1.m1.3.3.3.1.1.1.1.1.3">𝑘</ci></apply></apply></apply><apply id="S6.E1.m1.4.4.4.2.cmml" xref="S6.E1.m1.4.4.4.2"><times id="S6.E1.m1.4.4.4.2.2.cmml" xref="S6.E1.m1.4.4.4.2.2"></times><ci id="S6.E1.m1.4.4.4.2.3.cmml" xref="S6.E1.m1.4.4.4.2.3">𝛼</ci><apply id="S6.E1.m1.4.4.4.2.1.1.1.cmml" xref="S6.E1.m1.4.4.4.2.1.1"><csymbol cd="ambiguous" id="S6.E1.m1.4.4.4.2.1.1.1.1.cmml" xref="S6.E1.m1.4.4.4.2.1.1">subscript</csymbol><ci id="S6.E1.m1.4.4.4.2.1.1.1.2.cmml" xref="S6.E1.m1.4.4.4.2.1.1.1.2">𝜎</ci><ci id="S6.E1.m1.4.4.4.2.1.1.1.3.cmml" xref="S6.E1.m1.4.4.4.2.1.1.1.3">𝑘</ci></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.E1.m1.5c">\\
e_{j}(\sigma)=\dfrac{c_{j}(\sigma)+\alpha(\sigma)}{\sum_{k}c_{j}(\sigma_{k})+\alpha(\sigma_{k})}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(1)</span></td>
</tr></tbody>
</table>
</div>
<div id="S6.SS1.p5" class="ltx_para">
<p id="S6.SS1.p5.7" class="ltx_p">In the same way, the transition probability can be found through the equation <a href="#S6.E2" title="In 6.1 Profile HMMs ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>, where <math id="S6.SS1.p5.1.m1.1" class="ltx_Math" alttext="c_{kl}" display="inline"><semantics id="S6.SS1.p5.1.m1.1a"><msub id="S6.SS1.p5.1.m1.1.1" xref="S6.SS1.p5.1.m1.1.1.cmml"><mi id="S6.SS1.p5.1.m1.1.1.2" xref="S6.SS1.p5.1.m1.1.1.2.cmml">c</mi><mrow id="S6.SS1.p5.1.m1.1.1.3" xref="S6.SS1.p5.1.m1.1.1.3.cmml"><mi id="S6.SS1.p5.1.m1.1.1.3.2" xref="S6.SS1.p5.1.m1.1.1.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.SS1.p5.1.m1.1.1.3.1" xref="S6.SS1.p5.1.m1.1.1.3.1.cmml">​</mo><mi id="S6.SS1.p5.1.m1.1.1.3.3" xref="S6.SS1.p5.1.m1.1.1.3.3.cmml">l</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS1.p5.1.m1.1b"><apply id="S6.SS1.p5.1.m1.1.1.cmml" xref="S6.SS1.p5.1.m1.1.1"><csymbol cd="ambiguous" id="S6.SS1.p5.1.m1.1.1.1.cmml" xref="S6.SS1.p5.1.m1.1.1">subscript</csymbol><ci id="S6.SS1.p5.1.m1.1.1.2.cmml" xref="S6.SS1.p5.1.m1.1.1.2">𝑐</ci><apply id="S6.SS1.p5.1.m1.1.1.3.cmml" xref="S6.SS1.p5.1.m1.1.1.3"><times id="S6.SS1.p5.1.m1.1.1.3.1.cmml" xref="S6.SS1.p5.1.m1.1.1.3.1"></times><ci id="S6.SS1.p5.1.m1.1.1.3.2.cmml" xref="S6.SS1.p5.1.m1.1.1.3.2">𝑘</ci><ci id="S6.SS1.p5.1.m1.1.1.3.3.cmml" xref="S6.SS1.p5.1.m1.1.1.3.3">𝑙</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p5.1.m1.1c">c_{kl}</annotation></semantics></math> is the observed frequency of transitions between state <math id="S6.SS1.p5.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S6.SS1.p5.2.m2.1a"><mi id="S6.SS1.p5.2.m2.1.1" xref="S6.SS1.p5.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S6.SS1.p5.2.m2.1b"><ci id="S6.SS1.p5.2.m2.1.1.cmml" xref="S6.SS1.p5.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p5.2.m2.1c">k</annotation></semantics></math> and state <math id="S6.SS1.p5.3.m3.1" class="ltx_Math" alttext="l" display="inline"><semantics id="S6.SS1.p5.3.m3.1a"><mi id="S6.SS1.p5.3.m3.1.1" xref="S6.SS1.p5.3.m3.1.1.cmml">l</mi><annotation-xml encoding="MathML-Content" id="S6.SS1.p5.3.m3.1b"><ci id="S6.SS1.p5.3.m3.1.1.cmml" xref="S6.SS1.p5.3.m3.1.1">𝑙</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p5.3.m3.1c">l</annotation></semantics></math>, where <math id="S6.SS1.p5.4.m4.5" class="ltx_Math" alttext="k,l\in\{M,I,D\}" display="inline"><semantics id="S6.SS1.p5.4.m4.5a"><mrow id="S6.SS1.p5.4.m4.5.6" xref="S6.SS1.p5.4.m4.5.6.cmml"><mrow id="S6.SS1.p5.4.m4.5.6.2.2" xref="S6.SS1.p5.4.m4.5.6.2.1.cmml"><mi id="S6.SS1.p5.4.m4.4.4" xref="S6.SS1.p5.4.m4.4.4.cmml">k</mi><mo id="S6.SS1.p5.4.m4.5.6.2.2.1" xref="S6.SS1.p5.4.m4.5.6.2.1.cmml">,</mo><mi id="S6.SS1.p5.4.m4.5.5" xref="S6.SS1.p5.4.m4.5.5.cmml">l</mi></mrow><mo id="S6.SS1.p5.4.m4.5.6.1" xref="S6.SS1.p5.4.m4.5.6.1.cmml">∈</mo><mrow id="S6.SS1.p5.4.m4.5.6.3.2" xref="S6.SS1.p5.4.m4.5.6.3.1.cmml"><mo stretchy="false" id="S6.SS1.p5.4.m4.5.6.3.2.1" xref="S6.SS1.p5.4.m4.5.6.3.1.cmml">{</mo><mi id="S6.SS1.p5.4.m4.1.1" xref="S6.SS1.p5.4.m4.1.1.cmml">M</mi><mo id="S6.SS1.p5.4.m4.5.6.3.2.2" xref="S6.SS1.p5.4.m4.5.6.3.1.cmml">,</mo><mi id="S6.SS1.p5.4.m4.2.2" xref="S6.SS1.p5.4.m4.2.2.cmml">I</mi><mo id="S6.SS1.p5.4.m4.5.6.3.2.3" xref="S6.SS1.p5.4.m4.5.6.3.1.cmml">,</mo><mi id="S6.SS1.p5.4.m4.3.3" xref="S6.SS1.p5.4.m4.3.3.cmml">D</mi><mo stretchy="false" id="S6.SS1.p5.4.m4.5.6.3.2.4" xref="S6.SS1.p5.4.m4.5.6.3.1.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS1.p5.4.m4.5b"><apply id="S6.SS1.p5.4.m4.5.6.cmml" xref="S6.SS1.p5.4.m4.5.6"><in id="S6.SS1.p5.4.m4.5.6.1.cmml" xref="S6.SS1.p5.4.m4.5.6.1"></in><list id="S6.SS1.p5.4.m4.5.6.2.1.cmml" xref="S6.SS1.p5.4.m4.5.6.2.2"><ci id="S6.SS1.p5.4.m4.4.4.cmml" xref="S6.SS1.p5.4.m4.4.4">𝑘</ci><ci id="S6.SS1.p5.4.m4.5.5.cmml" xref="S6.SS1.p5.4.m4.5.5">𝑙</ci></list><set id="S6.SS1.p5.4.m4.5.6.3.1.cmml" xref="S6.SS1.p5.4.m4.5.6.3.2"><ci id="S6.SS1.p5.4.m4.1.1.cmml" xref="S6.SS1.p5.4.m4.1.1">𝑀</ci><ci id="S6.SS1.p5.4.m4.2.2.cmml" xref="S6.SS1.p5.4.m4.2.2">𝐼</ci><ci id="S6.SS1.p5.4.m4.3.3.cmml" xref="S6.SS1.p5.4.m4.3.3">𝐷</ci></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p5.4.m4.5c">k,l\in\{M,I,D\}</annotation></semantics></math>, and <math id="S6.SS1.p5.5.m5.1" class="ltx_Math" alttext="\alpha_{kl}" display="inline"><semantics id="S6.SS1.p5.5.m5.1a"><msub id="S6.SS1.p5.5.m5.1.1" xref="S6.SS1.p5.5.m5.1.1.cmml"><mi id="S6.SS1.p5.5.m5.1.1.2" xref="S6.SS1.p5.5.m5.1.1.2.cmml">α</mi><mrow id="S6.SS1.p5.5.m5.1.1.3" xref="S6.SS1.p5.5.m5.1.1.3.cmml"><mi id="S6.SS1.p5.5.m5.1.1.3.2" xref="S6.SS1.p5.5.m5.1.1.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.SS1.p5.5.m5.1.1.3.1" xref="S6.SS1.p5.5.m5.1.1.3.1.cmml">​</mo><mi id="S6.SS1.p5.5.m5.1.1.3.3" xref="S6.SS1.p5.5.m5.1.1.3.3.cmml">l</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS1.p5.5.m5.1b"><apply id="S6.SS1.p5.5.m5.1.1.cmml" xref="S6.SS1.p5.5.m5.1.1"><csymbol cd="ambiguous" id="S6.SS1.p5.5.m5.1.1.1.cmml" xref="S6.SS1.p5.5.m5.1.1">subscript</csymbol><ci id="S6.SS1.p5.5.m5.1.1.2.cmml" xref="S6.SS1.p5.5.m5.1.1.2">𝛼</ci><apply id="S6.SS1.p5.5.m5.1.1.3.cmml" xref="S6.SS1.p5.5.m5.1.1.3"><times id="S6.SS1.p5.5.m5.1.1.3.1.cmml" xref="S6.SS1.p5.5.m5.1.1.3.1"></times><ci id="S6.SS1.p5.5.m5.1.1.3.2.cmml" xref="S6.SS1.p5.5.m5.1.1.3.2">𝑘</ci><ci id="S6.SS1.p5.5.m5.1.1.3.3.cmml" xref="S6.SS1.p5.5.m5.1.1.3.3">𝑙</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p5.5.m5.1c">\alpha_{kl}</annotation></semantics></math> represents the pseudo counts of transition between <math id="S6.SS1.p5.6.m6.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S6.SS1.p5.6.m6.1a"><mi id="S6.SS1.p5.6.m6.1.1" xref="S6.SS1.p5.6.m6.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S6.SS1.p5.6.m6.1b"><ci id="S6.SS1.p5.6.m6.1.1.cmml" xref="S6.SS1.p5.6.m6.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p5.6.m6.1c">k</annotation></semantics></math> and <math id="S6.SS1.p5.7.m7.1" class="ltx_Math" alttext="l" display="inline"><semantics id="S6.SS1.p5.7.m7.1a"><mi id="S6.SS1.p5.7.m7.1.1" xref="S6.SS1.p5.7.m7.1.1.cmml">l</mi><annotation-xml encoding="MathML-Content" id="S6.SS1.p5.7.m7.1b"><ci id="S6.SS1.p5.7.m7.1.1.cmml" xref="S6.SS1.p5.7.m7.1.1">𝑙</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS1.p5.7.m7.1c">l</annotation></semantics></math>.</p>
</div>
<div id="S6.SS1.p6" class="ltx_para">
<table id="S6.E2" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S6.E2.m1.1" class="ltx_Math" alttext="\\
t_{kl}=\dfrac{c_{kl}+\alpha_{kl}}{\sum_{l}c_{kl}+\alpha_{kl}}" display="block"><semantics id="S6.E2.m1.1a"><mrow id="S6.E2.m1.1.1" xref="S6.E2.m1.1.1.cmml"><msub id="S6.E2.m1.1.1.2" xref="S6.E2.m1.1.1.2.cmml"><mi id="S6.E2.m1.1.1.2.2" xref="S6.E2.m1.1.1.2.2.cmml">t</mi><mrow id="S6.E2.m1.1.1.2.3" xref="S6.E2.m1.1.1.2.3.cmml"><mi id="S6.E2.m1.1.1.2.3.2" xref="S6.E2.m1.1.1.2.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.E2.m1.1.1.2.3.1" xref="S6.E2.m1.1.1.2.3.1.cmml">​</mo><mi id="S6.E2.m1.1.1.2.3.3" xref="S6.E2.m1.1.1.2.3.3.cmml">l</mi></mrow></msub><mo id="S6.E2.m1.1.1.1" xref="S6.E2.m1.1.1.1.cmml">=</mo><mfrac id="S6.E2.m1.1.1.3" xref="S6.E2.m1.1.1.3.cmml"><mrow id="S6.E2.m1.1.1.3.2" xref="S6.E2.m1.1.1.3.2.cmml"><msub id="S6.E2.m1.1.1.3.2.2" xref="S6.E2.m1.1.1.3.2.2.cmml"><mi id="S6.E2.m1.1.1.3.2.2.2" xref="S6.E2.m1.1.1.3.2.2.2.cmml">c</mi><mrow id="S6.E2.m1.1.1.3.2.2.3" xref="S6.E2.m1.1.1.3.2.2.3.cmml"><mi id="S6.E2.m1.1.1.3.2.2.3.2" xref="S6.E2.m1.1.1.3.2.2.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.E2.m1.1.1.3.2.2.3.1" xref="S6.E2.m1.1.1.3.2.2.3.1.cmml">​</mo><mi id="S6.E2.m1.1.1.3.2.2.3.3" xref="S6.E2.m1.1.1.3.2.2.3.3.cmml">l</mi></mrow></msub><mo id="S6.E2.m1.1.1.3.2.1" xref="S6.E2.m1.1.1.3.2.1.cmml">+</mo><msub id="S6.E2.m1.1.1.3.2.3" xref="S6.E2.m1.1.1.3.2.3.cmml"><mi id="S6.E2.m1.1.1.3.2.3.2" xref="S6.E2.m1.1.1.3.2.3.2.cmml">α</mi><mrow id="S6.E2.m1.1.1.3.2.3.3" xref="S6.E2.m1.1.1.3.2.3.3.cmml"><mi id="S6.E2.m1.1.1.3.2.3.3.2" xref="S6.E2.m1.1.1.3.2.3.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.E2.m1.1.1.3.2.3.3.1" xref="S6.E2.m1.1.1.3.2.3.3.1.cmml">​</mo><mi id="S6.E2.m1.1.1.3.2.3.3.3" xref="S6.E2.m1.1.1.3.2.3.3.3.cmml">l</mi></mrow></msub></mrow><mrow id="S6.E2.m1.1.1.3.3" xref="S6.E2.m1.1.1.3.3.cmml"><mrow id="S6.E2.m1.1.1.3.3.2" xref="S6.E2.m1.1.1.3.3.2.cmml"><msub id="S6.E2.m1.1.1.3.3.2.1" xref="S6.E2.m1.1.1.3.3.2.1.cmml"><mo id="S6.E2.m1.1.1.3.3.2.1.2" xref="S6.E2.m1.1.1.3.3.2.1.2.cmml">∑</mo><mi id="S6.E2.m1.1.1.3.3.2.1.3" xref="S6.E2.m1.1.1.3.3.2.1.3.cmml">l</mi></msub><msub id="S6.E2.m1.1.1.3.3.2.2" xref="S6.E2.m1.1.1.3.3.2.2.cmml"><mi id="S6.E2.m1.1.1.3.3.2.2.2" xref="S6.E2.m1.1.1.3.3.2.2.2.cmml">c</mi><mrow id="S6.E2.m1.1.1.3.3.2.2.3" xref="S6.E2.m1.1.1.3.3.2.2.3.cmml"><mi id="S6.E2.m1.1.1.3.3.2.2.3.2" xref="S6.E2.m1.1.1.3.3.2.2.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.E2.m1.1.1.3.3.2.2.3.1" xref="S6.E2.m1.1.1.3.3.2.2.3.1.cmml">​</mo><mi id="S6.E2.m1.1.1.3.3.2.2.3.3" xref="S6.E2.m1.1.1.3.3.2.2.3.3.cmml">l</mi></mrow></msub></mrow><mo id="S6.E2.m1.1.1.3.3.1" xref="S6.E2.m1.1.1.3.3.1.cmml">+</mo><msub id="S6.E2.m1.1.1.3.3.3" xref="S6.E2.m1.1.1.3.3.3.cmml"><mi id="S6.E2.m1.1.1.3.3.3.2" xref="S6.E2.m1.1.1.3.3.3.2.cmml">α</mi><mrow id="S6.E2.m1.1.1.3.3.3.3" xref="S6.E2.m1.1.1.3.3.3.3.cmml"><mi id="S6.E2.m1.1.1.3.3.3.3.2" xref="S6.E2.m1.1.1.3.3.3.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.E2.m1.1.1.3.3.3.3.1" xref="S6.E2.m1.1.1.3.3.3.3.1.cmml">​</mo><mi id="S6.E2.m1.1.1.3.3.3.3.3" xref="S6.E2.m1.1.1.3.3.3.3.3.cmml">l</mi></mrow></msub></mrow></mfrac></mrow><annotation-xml encoding="MathML-Content" id="S6.E2.m1.1b"><apply id="S6.E2.m1.1.1.cmml" xref="S6.E2.m1.1.1"><eq id="S6.E2.m1.1.1.1.cmml" xref="S6.E2.m1.1.1.1"></eq><apply id="S6.E2.m1.1.1.2.cmml" xref="S6.E2.m1.1.1.2"><csymbol cd="ambiguous" id="S6.E2.m1.1.1.2.1.cmml" xref="S6.E2.m1.1.1.2">subscript</csymbol><ci id="S6.E2.m1.1.1.2.2.cmml" xref="S6.E2.m1.1.1.2.2">𝑡</ci><apply id="S6.E2.m1.1.1.2.3.cmml" xref="S6.E2.m1.1.1.2.3"><times id="S6.E2.m1.1.1.2.3.1.cmml" xref="S6.E2.m1.1.1.2.3.1"></times><ci id="S6.E2.m1.1.1.2.3.2.cmml" xref="S6.E2.m1.1.1.2.3.2">𝑘</ci><ci id="S6.E2.m1.1.1.2.3.3.cmml" xref="S6.E2.m1.1.1.2.3.3">𝑙</ci></apply></apply><apply id="S6.E2.m1.1.1.3.cmml" xref="S6.E2.m1.1.1.3"><divide id="S6.E2.m1.1.1.3.1.cmml" xref="S6.E2.m1.1.1.3"></divide><apply id="S6.E2.m1.1.1.3.2.cmml" xref="S6.E2.m1.1.1.3.2"><plus id="S6.E2.m1.1.1.3.2.1.cmml" xref="S6.E2.m1.1.1.3.2.1"></plus><apply id="S6.E2.m1.1.1.3.2.2.cmml" xref="S6.E2.m1.1.1.3.2.2"><csymbol cd="ambiguous" id="S6.E2.m1.1.1.3.2.2.1.cmml" xref="S6.E2.m1.1.1.3.2.2">subscript</csymbol><ci id="S6.E2.m1.1.1.3.2.2.2.cmml" xref="S6.E2.m1.1.1.3.2.2.2">𝑐</ci><apply id="S6.E2.m1.1.1.3.2.2.3.cmml" xref="S6.E2.m1.1.1.3.2.2.3"><times id="S6.E2.m1.1.1.3.2.2.3.1.cmml" xref="S6.E2.m1.1.1.3.2.2.3.1"></times><ci id="S6.E2.m1.1.1.3.2.2.3.2.cmml" xref="S6.E2.m1.1.1.3.2.2.3.2">𝑘</ci><ci id="S6.E2.m1.1.1.3.2.2.3.3.cmml" xref="S6.E2.m1.1.1.3.2.2.3.3">𝑙</ci></apply></apply><apply id="S6.E2.m1.1.1.3.2.3.cmml" xref="S6.E2.m1.1.1.3.2.3"><csymbol cd="ambiguous" id="S6.E2.m1.1.1.3.2.3.1.cmml" xref="S6.E2.m1.1.1.3.2.3">subscript</csymbol><ci id="S6.E2.m1.1.1.3.2.3.2.cmml" xref="S6.E2.m1.1.1.3.2.3.2">𝛼</ci><apply id="S6.E2.m1.1.1.3.2.3.3.cmml" xref="S6.E2.m1.1.1.3.2.3.3"><times id="S6.E2.m1.1.1.3.2.3.3.1.cmml" xref="S6.E2.m1.1.1.3.2.3.3.1"></times><ci id="S6.E2.m1.1.1.3.2.3.3.2.cmml" xref="S6.E2.m1.1.1.3.2.3.3.2">𝑘</ci><ci id="S6.E2.m1.1.1.3.2.3.3.3.cmml" xref="S6.E2.m1.1.1.3.2.3.3.3">𝑙</ci></apply></apply></apply><apply id="S6.E2.m1.1.1.3.3.cmml" xref="S6.E2.m1.1.1.3.3"><plus id="S6.E2.m1.1.1.3.3.1.cmml" xref="S6.E2.m1.1.1.3.3.1"></plus><apply id="S6.E2.m1.1.1.3.3.2.cmml" xref="S6.E2.m1.1.1.3.3.2"><apply id="S6.E2.m1.1.1.3.3.2.1.cmml" xref="S6.E2.m1.1.1.3.3.2.1"><csymbol cd="ambiguous" id="S6.E2.m1.1.1.3.3.2.1.1.cmml" xref="S6.E2.m1.1.1.3.3.2.1">subscript</csymbol><sum id="S6.E2.m1.1.1.3.3.2.1.2.cmml" xref="S6.E2.m1.1.1.3.3.2.1.2"></sum><ci id="S6.E2.m1.1.1.3.3.2.1.3.cmml" xref="S6.E2.m1.1.1.3.3.2.1.3">𝑙</ci></apply><apply id="S6.E2.m1.1.1.3.3.2.2.cmml" xref="S6.E2.m1.1.1.3.3.2.2"><csymbol cd="ambiguous" id="S6.E2.m1.1.1.3.3.2.2.1.cmml" xref="S6.E2.m1.1.1.3.3.2.2">subscript</csymbol><ci id="S6.E2.m1.1.1.3.3.2.2.2.cmml" xref="S6.E2.m1.1.1.3.3.2.2.2">𝑐</ci><apply id="S6.E2.m1.1.1.3.3.2.2.3.cmml" xref="S6.E2.m1.1.1.3.3.2.2.3"><times id="S6.E2.m1.1.1.3.3.2.2.3.1.cmml" xref="S6.E2.m1.1.1.3.3.2.2.3.1"></times><ci id="S6.E2.m1.1.1.3.3.2.2.3.2.cmml" xref="S6.E2.m1.1.1.3.3.2.2.3.2">𝑘</ci><ci id="S6.E2.m1.1.1.3.3.2.2.3.3.cmml" xref="S6.E2.m1.1.1.3.3.2.2.3.3">𝑙</ci></apply></apply></apply><apply id="S6.E2.m1.1.1.3.3.3.cmml" xref="S6.E2.m1.1.1.3.3.3"><csymbol cd="ambiguous" id="S6.E2.m1.1.1.3.3.3.1.cmml" xref="S6.E2.m1.1.1.3.3.3">subscript</csymbol><ci id="S6.E2.m1.1.1.3.3.3.2.cmml" xref="S6.E2.m1.1.1.3.3.3.2">𝛼</ci><apply id="S6.E2.m1.1.1.3.3.3.3.cmml" xref="S6.E2.m1.1.1.3.3.3.3"><times id="S6.E2.m1.1.1.3.3.3.3.1.cmml" xref="S6.E2.m1.1.1.3.3.3.3.1"></times><ci id="S6.E2.m1.1.1.3.3.3.3.2.cmml" xref="S6.E2.m1.1.1.3.3.3.3.2">𝑘</ci><ci id="S6.E2.m1.1.1.3.3.3.3.3.cmml" xref="S6.E2.m1.1.1.3.3.3.3.3">𝑙</ci></apply></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.E2.m1.1c">\\
t_{kl}=\dfrac{c_{kl}+\alpha_{kl}}{\sum_{l}c_{kl}+\alpha_{kl}}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(2)</span></td>
</tr></tbody>
</table>
</div>
</section>
<section id="S6.SS2" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">6.2 </span>Sequence Weighting</h3>

<div id="S6.SS2.p1" class="ltx_para">
<p id="S6.SS2.p1.1" class="ltx_p">One problem in representing families of sequences is that often sets of very similar sequences may be over-represented in the training sequences, introducing bias. Therefore, <em id="S6.SS2.p1.1.1" class="ltx_emph ltx_font_italic">sequence-weighting</em> methods were introduced to compensate for over-representation among multiply aligned sequences. In general, very similar sequence receives lower weights and divergent sequence higher weight. Sequence weighting was applied to the construction of position-specifics score matrix (PSSM) <cite class="ltx_cite ltx_citemacro_citep">(Gribskov <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib21" title="" class="ltx_ref">1987</a>)</cite>, and is fundamental to the performance of profile HMMs. In the latter case, the default sequence weighting method used by HMMER package is a high quality algorithm based on phylogenetic trees <cite class="ltx_cite ltx_citemacro_citep">(Gerstein <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib17" title="" class="ltx_ref">1994</a>)</cite>.</p>
</div>
<div id="S6.SS2.p2" class="ltx_para">
<p id="S6.SS2.p2.11" class="ltx_p">Let <math id="S6.SS2.p2.1.m1.1" class="ltx_Math" alttext="A" display="inline"><semantics id="S6.SS2.p2.1.m1.1a"><mi id="S6.SS2.p2.1.m1.1.1" xref="S6.SS2.p2.1.m1.1.1.cmml">A</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p2.1.m1.1b"><ci id="S6.SS2.p2.1.m1.1.1.cmml" xref="S6.SS2.p2.1.m1.1.1">𝐴</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p2.1.m1.1c">A</annotation></semantics></math> be an generic alignment used to train a pHMM. Suppose, <math id="S6.SS2.p2.2.m2.1" class="ltx_Math" alttext="A" display="inline"><semantics id="S6.SS2.p2.2.m2.1a"><mi id="S6.SS2.p2.2.m2.1.1" xref="S6.SS2.p2.2.m2.1.1.cmml">A</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p2.2.m2.1b"><ci id="S6.SS2.p2.2.m2.1.1.cmml" xref="S6.SS2.p2.2.m2.1.1">𝐴</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p2.2.m2.1c">A</annotation></semantics></math> with <math id="S6.SS2.p2.3.m3.1" class="ltx_Math" alttext="N" display="inline"><semantics id="S6.SS2.p2.3.m3.1a"><mi id="S6.SS2.p2.3.m3.1.1" xref="S6.SS2.p2.3.m3.1.1.cmml">N</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p2.3.m3.1b"><ci id="S6.SS2.p2.3.m3.1.1.cmml" xref="S6.SS2.p2.3.m3.1.1">𝑁</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p2.3.m3.1c">N</annotation></semantics></math> sequences and length <math id="S6.SS2.p2.4.m4.1" class="ltx_Math" alttext="L" display="inline"><semantics id="S6.SS2.p2.4.m4.1a"><mi id="S6.SS2.p2.4.m4.1.1" xref="S6.SS2.p2.4.m4.1.1.cmml">L</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p2.4.m4.1b"><ci id="S6.SS2.p2.4.m4.1.1.cmml" xref="S6.SS2.p2.4.m4.1.1">𝐿</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p2.4.m4.1c">L</annotation></semantics></math>. Then, we can represent <math id="S6.SS2.p2.5.m5.1" class="ltx_Math" alttext="A" display="inline"><semantics id="S6.SS2.p2.5.m5.1a"><mi id="S6.SS2.p2.5.m5.1.1" xref="S6.SS2.p2.5.m5.1.1.cmml">A</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p2.5.m5.1b"><ci id="S6.SS2.p2.5.m5.1.1.cmml" xref="S6.SS2.p2.5.m5.1.1">𝐴</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p2.5.m5.1c">A</annotation></semantics></math> alignment weights as a matrix <math id="S6.SS2.p2.6.m6.1" class="ltx_Math" alttext="W" display="inline"><semantics id="S6.SS2.p2.6.m6.1a"><mi id="S6.SS2.p2.6.m6.1.1" xref="S6.SS2.p2.6.m6.1.1.cmml">W</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p2.6.m6.1b"><ci id="S6.SS2.p2.6.m6.1.1.cmml" xref="S6.SS2.p2.6.m6.1.1">𝑊</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p2.6.m6.1c">W</annotation></semantics></math>, such that <math id="S6.SS2.p2.7.m7.1" class="ltx_Math" alttext="w_{ij}" display="inline"><semantics id="S6.SS2.p2.7.m7.1a"><msub id="S6.SS2.p2.7.m7.1.1" xref="S6.SS2.p2.7.m7.1.1.cmml"><mi id="S6.SS2.p2.7.m7.1.1.2" xref="S6.SS2.p2.7.m7.1.1.2.cmml">w</mi><mrow id="S6.SS2.p2.7.m7.1.1.3" xref="S6.SS2.p2.7.m7.1.1.3.cmml"><mi id="S6.SS2.p2.7.m7.1.1.3.2" xref="S6.SS2.p2.7.m7.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p2.7.m7.1.1.3.1" xref="S6.SS2.p2.7.m7.1.1.3.1.cmml">​</mo><mi id="S6.SS2.p2.7.m7.1.1.3.3" xref="S6.SS2.p2.7.m7.1.1.3.3.cmml">j</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS2.p2.7.m7.1b"><apply id="S6.SS2.p2.7.m7.1.1.cmml" xref="S6.SS2.p2.7.m7.1.1"><csymbol cd="ambiguous" id="S6.SS2.p2.7.m7.1.1.1.cmml" xref="S6.SS2.p2.7.m7.1.1">subscript</csymbol><ci id="S6.SS2.p2.7.m7.1.1.2.cmml" xref="S6.SS2.p2.7.m7.1.1.2">𝑤</ci><apply id="S6.SS2.p2.7.m7.1.1.3.cmml" xref="S6.SS2.p2.7.m7.1.1.3"><times id="S6.SS2.p2.7.m7.1.1.3.1.cmml" xref="S6.SS2.p2.7.m7.1.1.3.1"></times><ci id="S6.SS2.p2.7.m7.1.1.3.2.cmml" xref="S6.SS2.p2.7.m7.1.1.3.2">𝑖</ci><ci id="S6.SS2.p2.7.m7.1.1.3.3.cmml" xref="S6.SS2.p2.7.m7.1.1.3.3">𝑗</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p2.7.m7.1c">w_{ij}</annotation></semantics></math> represents the weight of an amino-acid of protein <math id="S6.SS2.p2.8.m8.1" class="ltx_Math" alttext="I" display="inline"><semantics id="S6.SS2.p2.8.m8.1a"><mi id="S6.SS2.p2.8.m8.1.1" xref="S6.SS2.p2.8.m8.1.1.cmml">I</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p2.8.m8.1b"><ci id="S6.SS2.p2.8.m8.1.1.cmml" xref="S6.SS2.p2.8.m8.1.1">𝐼</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p2.8.m8.1c">I</annotation></semantics></math> in the <math id="S6.SS2.p2.9.m9.1" class="ltx_Math" alttext="j^{th}" display="inline"><semantics id="S6.SS2.p2.9.m9.1a"><msup id="S6.SS2.p2.9.m9.1.1" xref="S6.SS2.p2.9.m9.1.1.cmml"><mi id="S6.SS2.p2.9.m9.1.1.2" xref="S6.SS2.p2.9.m9.1.1.2.cmml">j</mi><mrow id="S6.SS2.p2.9.m9.1.1.3" xref="S6.SS2.p2.9.m9.1.1.3.cmml"><mi id="S6.SS2.p2.9.m9.1.1.3.2" xref="S6.SS2.p2.9.m9.1.1.3.2.cmml">t</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p2.9.m9.1.1.3.1" xref="S6.SS2.p2.9.m9.1.1.3.1.cmml">​</mo><mi id="S6.SS2.p2.9.m9.1.1.3.3" xref="S6.SS2.p2.9.m9.1.1.3.3.cmml">h</mi></mrow></msup><annotation-xml encoding="MathML-Content" id="S6.SS2.p2.9.m9.1b"><apply id="S6.SS2.p2.9.m9.1.1.cmml" xref="S6.SS2.p2.9.m9.1.1"><csymbol cd="ambiguous" id="S6.SS2.p2.9.m9.1.1.1.cmml" xref="S6.SS2.p2.9.m9.1.1">superscript</csymbol><ci id="S6.SS2.p2.9.m9.1.1.2.cmml" xref="S6.SS2.p2.9.m9.1.1.2">𝑗</ci><apply id="S6.SS2.p2.9.m9.1.1.3.cmml" xref="S6.SS2.p2.9.m9.1.1.3"><times id="S6.SS2.p2.9.m9.1.1.3.1.cmml" xref="S6.SS2.p2.9.m9.1.1.3.1"></times><ci id="S6.SS2.p2.9.m9.1.1.3.2.cmml" xref="S6.SS2.p2.9.m9.1.1.3.2">𝑡</ci><ci id="S6.SS2.p2.9.m9.1.1.3.3.cmml" xref="S6.SS2.p2.9.m9.1.1.3.3">ℎ</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p2.9.m9.1c">j^{th}</annotation></semantics></math> alignment position, as shown in the equation <a href="#S6.E3" title="In 6.2 Sequence Weighting ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>. Basically, a sequence-weighting method for pHMMs attributes equal weights to all residues in the protein, that is, <math id="S6.SS2.p2.10.m10.1" class="ltx_Math" alttext="w_{ij}=w_{ik}" display="inline"><semantics id="S6.SS2.p2.10.m10.1a"><mrow id="S6.SS2.p2.10.m10.1.1" xref="S6.SS2.p2.10.m10.1.1.cmml"><msub id="S6.SS2.p2.10.m10.1.1.2" xref="S6.SS2.p2.10.m10.1.1.2.cmml"><mi id="S6.SS2.p2.10.m10.1.1.2.2" xref="S6.SS2.p2.10.m10.1.1.2.2.cmml">w</mi><mrow id="S6.SS2.p2.10.m10.1.1.2.3" xref="S6.SS2.p2.10.m10.1.1.2.3.cmml"><mi id="S6.SS2.p2.10.m10.1.1.2.3.2" xref="S6.SS2.p2.10.m10.1.1.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p2.10.m10.1.1.2.3.1" xref="S6.SS2.p2.10.m10.1.1.2.3.1.cmml">​</mo><mi id="S6.SS2.p2.10.m10.1.1.2.3.3" xref="S6.SS2.p2.10.m10.1.1.2.3.3.cmml">j</mi></mrow></msub><mo id="S6.SS2.p2.10.m10.1.1.1" xref="S6.SS2.p2.10.m10.1.1.1.cmml">=</mo><msub id="S6.SS2.p2.10.m10.1.1.3" xref="S6.SS2.p2.10.m10.1.1.3.cmml"><mi id="S6.SS2.p2.10.m10.1.1.3.2" xref="S6.SS2.p2.10.m10.1.1.3.2.cmml">w</mi><mrow id="S6.SS2.p2.10.m10.1.1.3.3" xref="S6.SS2.p2.10.m10.1.1.3.3.cmml"><mi id="S6.SS2.p2.10.m10.1.1.3.3.2" xref="S6.SS2.p2.10.m10.1.1.3.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p2.10.m10.1.1.3.3.1" xref="S6.SS2.p2.10.m10.1.1.3.3.1.cmml">​</mo><mi id="S6.SS2.p2.10.m10.1.1.3.3.3" xref="S6.SS2.p2.10.m10.1.1.3.3.3.cmml">k</mi></mrow></msub></mrow><annotation-xml encoding="MathML-Content" id="S6.SS2.p2.10.m10.1b"><apply id="S6.SS2.p2.10.m10.1.1.cmml" xref="S6.SS2.p2.10.m10.1.1"><eq id="S6.SS2.p2.10.m10.1.1.1.cmml" xref="S6.SS2.p2.10.m10.1.1.1"></eq><apply id="S6.SS2.p2.10.m10.1.1.2.cmml" xref="S6.SS2.p2.10.m10.1.1.2"><csymbol cd="ambiguous" id="S6.SS2.p2.10.m10.1.1.2.1.cmml" xref="S6.SS2.p2.10.m10.1.1.2">subscript</csymbol><ci id="S6.SS2.p2.10.m10.1.1.2.2.cmml" xref="S6.SS2.p2.10.m10.1.1.2.2">𝑤</ci><apply id="S6.SS2.p2.10.m10.1.1.2.3.cmml" xref="S6.SS2.p2.10.m10.1.1.2.3"><times id="S6.SS2.p2.10.m10.1.1.2.3.1.cmml" xref="S6.SS2.p2.10.m10.1.1.2.3.1"></times><ci id="S6.SS2.p2.10.m10.1.1.2.3.2.cmml" xref="S6.SS2.p2.10.m10.1.1.2.3.2">𝑖</ci><ci id="S6.SS2.p2.10.m10.1.1.2.3.3.cmml" xref="S6.SS2.p2.10.m10.1.1.2.3.3">𝑗</ci></apply></apply><apply id="S6.SS2.p2.10.m10.1.1.3.cmml" xref="S6.SS2.p2.10.m10.1.1.3"><csymbol cd="ambiguous" id="S6.SS2.p2.10.m10.1.1.3.1.cmml" xref="S6.SS2.p2.10.m10.1.1.3">subscript</csymbol><ci id="S6.SS2.p2.10.m10.1.1.3.2.cmml" xref="S6.SS2.p2.10.m10.1.1.3.2">𝑤</ci><apply id="S6.SS2.p2.10.m10.1.1.3.3.cmml" xref="S6.SS2.p2.10.m10.1.1.3.3"><times id="S6.SS2.p2.10.m10.1.1.3.3.1.cmml" xref="S6.SS2.p2.10.m10.1.1.3.3.1"></times><ci id="S6.SS2.p2.10.m10.1.1.3.3.2.cmml" xref="S6.SS2.p2.10.m10.1.1.3.3.2">𝑖</ci><ci id="S6.SS2.p2.10.m10.1.1.3.3.3.cmml" xref="S6.SS2.p2.10.m10.1.1.3.3.3">𝑘</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p2.10.m10.1c">w_{ij}=w_{ik}</annotation></semantics></math> for <math id="S6.SS2.p2.11.m11.2" class="ltx_Math" alttext="\forall j,k\leq L" display="inline"><semantics id="S6.SS2.p2.11.m11.2a"><mrow id="S6.SS2.p2.11.m11.2.2" xref="S6.SS2.p2.11.m11.2.2.cmml"><mrow id="S6.SS2.p2.11.m11.2.2.1.1" xref="S6.SS2.p2.11.m11.2.2.1.2.cmml"><mrow id="S6.SS2.p2.11.m11.2.2.1.1.1" xref="S6.SS2.p2.11.m11.2.2.1.1.1.cmml"><mo rspace="0.167em" id="S6.SS2.p2.11.m11.2.2.1.1.1.1" xref="S6.SS2.p2.11.m11.2.2.1.1.1.1.cmml">∀</mo><mi id="S6.SS2.p2.11.m11.2.2.1.1.1.2" xref="S6.SS2.p2.11.m11.2.2.1.1.1.2.cmml">j</mi></mrow><mo id="S6.SS2.p2.11.m11.2.2.1.1.2" xref="S6.SS2.p2.11.m11.2.2.1.2.cmml">,</mo><mi id="S6.SS2.p2.11.m11.1.1" xref="S6.SS2.p2.11.m11.1.1.cmml">k</mi></mrow><mo id="S6.SS2.p2.11.m11.2.2.2" xref="S6.SS2.p2.11.m11.2.2.2.cmml">≤</mo><mi id="S6.SS2.p2.11.m11.2.2.3" xref="S6.SS2.p2.11.m11.2.2.3.cmml">L</mi></mrow><annotation-xml encoding="MathML-Content" id="S6.SS2.p2.11.m11.2b"><apply id="S6.SS2.p2.11.m11.2.2.cmml" xref="S6.SS2.p2.11.m11.2.2"><leq id="S6.SS2.p2.11.m11.2.2.2.cmml" xref="S6.SS2.p2.11.m11.2.2.2"></leq><list id="S6.SS2.p2.11.m11.2.2.1.2.cmml" xref="S6.SS2.p2.11.m11.2.2.1.1"><apply id="S6.SS2.p2.11.m11.2.2.1.1.1.cmml" xref="S6.SS2.p2.11.m11.2.2.1.1.1"><csymbol cd="latexml" id="S6.SS2.p2.11.m11.2.2.1.1.1.1.cmml" xref="S6.SS2.p2.11.m11.2.2.1.1.1.1">for-all</csymbol><ci id="S6.SS2.p2.11.m11.2.2.1.1.1.2.cmml" xref="S6.SS2.p2.11.m11.2.2.1.1.1.2">𝑗</ci></apply><ci id="S6.SS2.p2.11.m11.1.1.cmml" xref="S6.SS2.p2.11.m11.1.1">𝑘</ci></list><ci id="S6.SS2.p2.11.m11.2.2.3.cmml" xref="S6.SS2.p2.11.m11.2.2.3">𝐿</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p2.11.m11.2c">\forall j,k\leq L</annotation></semantics></math>.

<br class="ltx_break"></p>
</div>
<div id="S6.SS2.p3" class="ltx_para">
<table id="S6.E3" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S6.E3.m1.1" class="ltx_Math" alttext="\mathbf{W=}\left(\begin{array}[]{ccc}w_{11}&amp;\ldots&amp;w_{1L}\\
\vdots&amp;\vdots&amp;\vdots\\
w_{N1}&amp;\ldots&amp;w_{NL}\\
\end{array}\right)" display="block"><semantics id="S6.E3.m1.1a"><mrow id="S6.E3.m1.1.2" xref="S6.E3.m1.1.2.cmml"><mi id="S6.E3.m1.1.2.2" xref="S6.E3.m1.1.2.2.cmml">𝐖</mi><mo id="S6.E3.m1.1.2.1" xref="S6.E3.m1.1.2.1.cmml">=</mo><mrow id="S6.E3.m1.1.2.3.2" xref="S6.E3.m1.1.1.cmml"><mo id="S6.E3.m1.1.2.3.2.1" xref="S6.E3.m1.1.1.cmml">(</mo><mtable columnspacing="5pt" displaystyle="true" rowspacing="0pt" id="S6.E3.m1.1.1" xref="S6.E3.m1.1.1.cmml"><mtr id="S6.E3.m1.1.1a" xref="S6.E3.m1.1.1.cmml"><mtd id="S6.E3.m1.1.1b" xref="S6.E3.m1.1.1.cmml"><msub id="S6.E3.m1.1.1.1.1.1" xref="S6.E3.m1.1.1.1.1.1.cmml"><mi id="S6.E3.m1.1.1.1.1.1.2" xref="S6.E3.m1.1.1.1.1.1.2.cmml">w</mi><mn id="S6.E3.m1.1.1.1.1.1.3" xref="S6.E3.m1.1.1.1.1.1.3.cmml">11</mn></msub></mtd><mtd id="S6.E3.m1.1.1c" xref="S6.E3.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E3.m1.1.1.1.2.1" xref="S6.E3.m1.1.1.1.2.1.cmml">…</mi></mtd><mtd id="S6.E3.m1.1.1d" xref="S6.E3.m1.1.1.cmml"><msub id="S6.E3.m1.1.1.1.3.1" xref="S6.E3.m1.1.1.1.3.1.cmml"><mi id="S6.E3.m1.1.1.1.3.1.2" xref="S6.E3.m1.1.1.1.3.1.2.cmml">w</mi><mrow id="S6.E3.m1.1.1.1.3.1.3" xref="S6.E3.m1.1.1.1.3.1.3.cmml"><mn id="S6.E3.m1.1.1.1.3.1.3.2" xref="S6.E3.m1.1.1.1.3.1.3.2.cmml">1</mn><mo lspace="0em" rspace="0em" id="S6.E3.m1.1.1.1.3.1.3.1" xref="S6.E3.m1.1.1.1.3.1.3.1.cmml">​</mo><mi id="S6.E3.m1.1.1.1.3.1.3.3" xref="S6.E3.m1.1.1.1.3.1.3.3.cmml">L</mi></mrow></msub></mtd></mtr><mtr id="S6.E3.m1.1.1e" xref="S6.E3.m1.1.1.cmml"><mtd id="S6.E3.m1.1.1f" xref="S6.E3.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E3.m1.1.1.2.1.1" xref="S6.E3.m1.1.1.2.1.1.cmml">⋮</mi></mtd><mtd id="S6.E3.m1.1.1g" xref="S6.E3.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E3.m1.1.1.2.2.1" xref="S6.E3.m1.1.1.2.2.1.cmml">⋮</mi></mtd><mtd id="S6.E3.m1.1.1h" xref="S6.E3.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E3.m1.1.1.2.3.1" xref="S6.E3.m1.1.1.2.3.1.cmml">⋮</mi></mtd></mtr><mtr id="S6.E3.m1.1.1i" xref="S6.E3.m1.1.1.cmml"><mtd id="S6.E3.m1.1.1j" xref="S6.E3.m1.1.1.cmml"><msub id="S6.E3.m1.1.1.3.1.1" xref="S6.E3.m1.1.1.3.1.1.cmml"><mi id="S6.E3.m1.1.1.3.1.1.2" xref="S6.E3.m1.1.1.3.1.1.2.cmml">w</mi><mrow id="S6.E3.m1.1.1.3.1.1.3" xref="S6.E3.m1.1.1.3.1.1.3.cmml"><mi id="S6.E3.m1.1.1.3.1.1.3.2" xref="S6.E3.m1.1.1.3.1.1.3.2.cmml">N</mi><mo lspace="0em" rspace="0em" id="S6.E3.m1.1.1.3.1.1.3.1" xref="S6.E3.m1.1.1.3.1.1.3.1.cmml">​</mo><mn id="S6.E3.m1.1.1.3.1.1.3.3" xref="S6.E3.m1.1.1.3.1.1.3.3.cmml">1</mn></mrow></msub></mtd><mtd id="S6.E3.m1.1.1k" xref="S6.E3.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E3.m1.1.1.3.2.1" xref="S6.E3.m1.1.1.3.2.1.cmml">…</mi></mtd><mtd id="S6.E3.m1.1.1l" xref="S6.E3.m1.1.1.cmml"><msub id="S6.E3.m1.1.1.3.3.1" xref="S6.E3.m1.1.1.3.3.1.cmml"><mi id="S6.E3.m1.1.1.3.3.1.2" xref="S6.E3.m1.1.1.3.3.1.2.cmml">w</mi><mrow id="S6.E3.m1.1.1.3.3.1.3" xref="S6.E3.m1.1.1.3.3.1.3.cmml"><mi id="S6.E3.m1.1.1.3.3.1.3.2" xref="S6.E3.m1.1.1.3.3.1.3.2.cmml">N</mi><mo lspace="0em" rspace="0em" id="S6.E3.m1.1.1.3.3.1.3.1" xref="S6.E3.m1.1.1.3.3.1.3.1.cmml">​</mo><mi id="S6.E3.m1.1.1.3.3.1.3.3" xref="S6.E3.m1.1.1.3.3.1.3.3.cmml">L</mi></mrow></msub></mtd></mtr></mtable><mo id="S6.E3.m1.1.2.3.2.2" xref="S6.E3.m1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.E3.m1.1b"><apply id="S6.E3.m1.1.2.cmml" xref="S6.E3.m1.1.2"><eq id="S6.E3.m1.1.2.1.cmml" xref="S6.E3.m1.1.2.1"></eq><ci id="S6.E3.m1.1.2.2.cmml" xref="S6.E3.m1.1.2.2">𝐖</ci><matrix id="S6.E3.m1.1.1.cmml" xref="S6.E3.m1.1.2.3.2"><matrixrow id="S6.E3.m1.1.1a.cmml" xref="S6.E3.m1.1.2.3.2"><apply id="S6.E3.m1.1.1.1.1.1.cmml" xref="S6.E3.m1.1.1.1.1.1"><csymbol cd="ambiguous" id="S6.E3.m1.1.1.1.1.1.1.cmml" xref="S6.E3.m1.1.1.1.1.1">subscript</csymbol><ci id="S6.E3.m1.1.1.1.1.1.2.cmml" xref="S6.E3.m1.1.1.1.1.1.2">𝑤</ci><cn type="integer" id="S6.E3.m1.1.1.1.1.1.3.cmml" xref="S6.E3.m1.1.1.1.1.1.3">11</cn></apply><ci id="S6.E3.m1.1.1.1.2.1.cmml" xref="S6.E3.m1.1.1.1.2.1">…</ci><apply id="S6.E3.m1.1.1.1.3.1.cmml" xref="S6.E3.m1.1.1.1.3.1"><csymbol cd="ambiguous" id="S6.E3.m1.1.1.1.3.1.1.cmml" xref="S6.E3.m1.1.1.1.3.1">subscript</csymbol><ci id="S6.E3.m1.1.1.1.3.1.2.cmml" xref="S6.E3.m1.1.1.1.3.1.2">𝑤</ci><apply id="S6.E3.m1.1.1.1.3.1.3.cmml" xref="S6.E3.m1.1.1.1.3.1.3"><times id="S6.E3.m1.1.1.1.3.1.3.1.cmml" xref="S6.E3.m1.1.1.1.3.1.3.1"></times><cn type="integer" id="S6.E3.m1.1.1.1.3.1.3.2.cmml" xref="S6.E3.m1.1.1.1.3.1.3.2">1</cn><ci id="S6.E3.m1.1.1.1.3.1.3.3.cmml" xref="S6.E3.m1.1.1.1.3.1.3.3">𝐿</ci></apply></apply></matrixrow><matrixrow id="S6.E3.m1.1.1b.cmml" xref="S6.E3.m1.1.2.3.2"><ci id="S6.E3.m1.1.1.2.1.1.cmml" xref="S6.E3.m1.1.1.2.1.1">⋮</ci><ci id="S6.E3.m1.1.1.2.2.1.cmml" xref="S6.E3.m1.1.1.2.2.1">⋮</ci><ci id="S6.E3.m1.1.1.2.3.1.cmml" xref="S6.E3.m1.1.1.2.3.1">⋮</ci></matrixrow><matrixrow id="S6.E3.m1.1.1c.cmml" xref="S6.E3.m1.1.2.3.2"><apply id="S6.E3.m1.1.1.3.1.1.cmml" xref="S6.E3.m1.1.1.3.1.1"><csymbol cd="ambiguous" id="S6.E3.m1.1.1.3.1.1.1.cmml" xref="S6.E3.m1.1.1.3.1.1">subscript</csymbol><ci id="S6.E3.m1.1.1.3.1.1.2.cmml" xref="S6.E3.m1.1.1.3.1.1.2">𝑤</ci><apply id="S6.E3.m1.1.1.3.1.1.3.cmml" xref="S6.E3.m1.1.1.3.1.1.3"><times id="S6.E3.m1.1.1.3.1.1.3.1.cmml" xref="S6.E3.m1.1.1.3.1.1.3.1"></times><ci id="S6.E3.m1.1.1.3.1.1.3.2.cmml" xref="S6.E3.m1.1.1.3.1.1.3.2">𝑁</ci><cn type="integer" id="S6.E3.m1.1.1.3.1.1.3.3.cmml" xref="S6.E3.m1.1.1.3.1.1.3.3">1</cn></apply></apply><ci id="S6.E3.m1.1.1.3.2.1.cmml" xref="S6.E3.m1.1.1.3.2.1">…</ci><apply id="S6.E3.m1.1.1.3.3.1.cmml" xref="S6.E3.m1.1.1.3.3.1"><csymbol cd="ambiguous" id="S6.E3.m1.1.1.3.3.1.1.cmml" xref="S6.E3.m1.1.1.3.3.1">subscript</csymbol><ci id="S6.E3.m1.1.1.3.3.1.2.cmml" xref="S6.E3.m1.1.1.3.3.1.2">𝑤</ci><apply id="S6.E3.m1.1.1.3.3.1.3.cmml" xref="S6.E3.m1.1.1.3.3.1.3"><times id="S6.E3.m1.1.1.3.3.1.3.1.cmml" xref="S6.E3.m1.1.1.3.3.1.3.1"></times><ci id="S6.E3.m1.1.1.3.3.1.3.2.cmml" xref="S6.E3.m1.1.1.3.3.1.3.2">𝑁</ci><ci id="S6.E3.m1.1.1.3.3.1.3.3.cmml" xref="S6.E3.m1.1.1.3.3.1.3.3">𝐿</ci></apply></apply></matrixrow></matrix></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.E3.m1.1c">\mathbf{W=}\left(\begin{array}[]{ccc}w_{11}&amp;\ldots&amp;w_{1L}\\
\vdots&amp;\vdots&amp;\vdots\\
w_{N1}&amp;\ldots&amp;w_{NL}\\
\end{array}\right)</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(3)</span></td>
</tr></tbody>
</table>
<br class="ltx_break">
</div>
<div id="S6.SS2.p4" class="ltx_para">
<p id="S6.SS2.p4.1" class="ltx_p">In the spirit of PSSMs, we propose to reinforce residues that correspond to preserved regions in the protein. Our motivation is that when homologue proteins are structurally aligned, spatial overlapping of an atom set occurs. This set is called the <em id="S6.SS2.p4.1.1" class="ltx_emph ltx_font_italic">invariant core</em> or <em id="S6.SS2.p4.1.2" class="ltx_emph ltx_font_italic">core structure</em>, and can be used to characterize homologue proteins. We argue that the residues in the core structure should carry more weight rather than the residues outside the core. Thus, we propose sequence-weighting method that gives different weight to each residue in the same protein, based on structural relevance. We will represent such “structural” weights by a matrix <math id="S6.SS2.p4.1.m1.1" class="ltx_Math" alttext="M_{s}" display="inline"><semantics id="S6.SS2.p4.1.m1.1a"><msub id="S6.SS2.p4.1.m1.1.1" xref="S6.SS2.p4.1.m1.1.1.cmml"><mi id="S6.SS2.p4.1.m1.1.1.2" xref="S6.SS2.p4.1.m1.1.1.2.cmml">M</mi><mi id="S6.SS2.p4.1.m1.1.1.3" xref="S6.SS2.p4.1.m1.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="S6.SS2.p4.1.m1.1b"><apply id="S6.SS2.p4.1.m1.1.1.cmml" xref="S6.SS2.p4.1.m1.1.1"><csymbol cd="ambiguous" id="S6.SS2.p4.1.m1.1.1.1.cmml" xref="S6.SS2.p4.1.m1.1.1">subscript</csymbol><ci id="S6.SS2.p4.1.m1.1.1.2.cmml" xref="S6.SS2.p4.1.m1.1.1.2">𝑀</ci><ci id="S6.SS2.p4.1.m1.1.1.3.cmml" xref="S6.SS2.p4.1.m1.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p4.1.m1.1c">M_{s}</annotation></semantics></math>, where each residue of the same protein has a different weight.

<br class="ltx_break"></p>
</div>
<div id="S6.SS2.p5" class="ltx_para">
<table id="S6.E4" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S6.E4.m1.1" class="ltx_Math" alttext="\mathbf{M_{s}=}\left(\begin{array}[]{ccc}m_{11}&amp;\ldots&amp;m_{1L}\\
\vdots&amp;\vdots&amp;\vdots\\
m_{N1}&amp;\ldots&amp;w_{NL}\\
\end{array}\right)" display="block"><semantics id="S6.E4.m1.1a"><mrow id="S6.E4.m1.1.2" xref="S6.E4.m1.1.2.cmml"><msub id="S6.E4.m1.1.2.2" xref="S6.E4.m1.1.2.2.cmml"><mi id="S6.E4.m1.1.2.2.2" xref="S6.E4.m1.1.2.2.2.cmml">𝐌</mi><mi id="S6.E4.m1.1.2.2.3" xref="S6.E4.m1.1.2.2.3.cmml">𝐬</mi></msub><mo id="S6.E4.m1.1.2.1" xref="S6.E4.m1.1.2.1.cmml">=</mo><mrow id="S6.E4.m1.1.2.3.2" xref="S6.E4.m1.1.1.cmml"><mo id="S6.E4.m1.1.2.3.2.1" xref="S6.E4.m1.1.1.cmml">(</mo><mtable columnspacing="5pt" displaystyle="true" rowspacing="0pt" id="S6.E4.m1.1.1" xref="S6.E4.m1.1.1.cmml"><mtr id="S6.E4.m1.1.1a" xref="S6.E4.m1.1.1.cmml"><mtd id="S6.E4.m1.1.1b" xref="S6.E4.m1.1.1.cmml"><msub id="S6.E4.m1.1.1.1.1.1" xref="S6.E4.m1.1.1.1.1.1.cmml"><mi id="S6.E4.m1.1.1.1.1.1.2" xref="S6.E4.m1.1.1.1.1.1.2.cmml">m</mi><mn id="S6.E4.m1.1.1.1.1.1.3" xref="S6.E4.m1.1.1.1.1.1.3.cmml">11</mn></msub></mtd><mtd id="S6.E4.m1.1.1c" xref="S6.E4.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E4.m1.1.1.1.2.1" xref="S6.E4.m1.1.1.1.2.1.cmml">…</mi></mtd><mtd id="S6.E4.m1.1.1d" xref="S6.E4.m1.1.1.cmml"><msub id="S6.E4.m1.1.1.1.3.1" xref="S6.E4.m1.1.1.1.3.1.cmml"><mi id="S6.E4.m1.1.1.1.3.1.2" xref="S6.E4.m1.1.1.1.3.1.2.cmml">m</mi><mrow id="S6.E4.m1.1.1.1.3.1.3" xref="S6.E4.m1.1.1.1.3.1.3.cmml"><mn id="S6.E4.m1.1.1.1.3.1.3.2" xref="S6.E4.m1.1.1.1.3.1.3.2.cmml">1</mn><mo lspace="0em" rspace="0em" id="S6.E4.m1.1.1.1.3.1.3.1" xref="S6.E4.m1.1.1.1.3.1.3.1.cmml">​</mo><mi id="S6.E4.m1.1.1.1.3.1.3.3" xref="S6.E4.m1.1.1.1.3.1.3.3.cmml">L</mi></mrow></msub></mtd></mtr><mtr id="S6.E4.m1.1.1e" xref="S6.E4.m1.1.1.cmml"><mtd id="S6.E4.m1.1.1f" xref="S6.E4.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E4.m1.1.1.2.1.1" xref="S6.E4.m1.1.1.2.1.1.cmml">⋮</mi></mtd><mtd id="S6.E4.m1.1.1g" xref="S6.E4.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E4.m1.1.1.2.2.1" xref="S6.E4.m1.1.1.2.2.1.cmml">⋮</mi></mtd><mtd id="S6.E4.m1.1.1h" xref="S6.E4.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E4.m1.1.1.2.3.1" xref="S6.E4.m1.1.1.2.3.1.cmml">⋮</mi></mtd></mtr><mtr id="S6.E4.m1.1.1i" xref="S6.E4.m1.1.1.cmml"><mtd id="S6.E4.m1.1.1j" xref="S6.E4.m1.1.1.cmml"><msub id="S6.E4.m1.1.1.3.1.1" xref="S6.E4.m1.1.1.3.1.1.cmml"><mi id="S6.E4.m1.1.1.3.1.1.2" xref="S6.E4.m1.1.1.3.1.1.2.cmml">m</mi><mrow id="S6.E4.m1.1.1.3.1.1.3" xref="S6.E4.m1.1.1.3.1.1.3.cmml"><mi id="S6.E4.m1.1.1.3.1.1.3.2" xref="S6.E4.m1.1.1.3.1.1.3.2.cmml">N</mi><mo lspace="0em" rspace="0em" id="S6.E4.m1.1.1.3.1.1.3.1" xref="S6.E4.m1.1.1.3.1.1.3.1.cmml">​</mo><mn id="S6.E4.m1.1.1.3.1.1.3.3" xref="S6.E4.m1.1.1.3.1.1.3.3.cmml">1</mn></mrow></msub></mtd><mtd id="S6.E4.m1.1.1k" xref="S6.E4.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E4.m1.1.1.3.2.1" xref="S6.E4.m1.1.1.3.2.1.cmml">…</mi></mtd><mtd id="S6.E4.m1.1.1l" xref="S6.E4.m1.1.1.cmml"><msub id="S6.E4.m1.1.1.3.3.1" xref="S6.E4.m1.1.1.3.3.1.cmml"><mi id="S6.E4.m1.1.1.3.3.1.2" xref="S6.E4.m1.1.1.3.3.1.2.cmml">w</mi><mrow id="S6.E4.m1.1.1.3.3.1.3" xref="S6.E4.m1.1.1.3.3.1.3.cmml"><mi id="S6.E4.m1.1.1.3.3.1.3.2" xref="S6.E4.m1.1.1.3.3.1.3.2.cmml">N</mi><mo lspace="0em" rspace="0em" id="S6.E4.m1.1.1.3.3.1.3.1" xref="S6.E4.m1.1.1.3.3.1.3.1.cmml">​</mo><mi id="S6.E4.m1.1.1.3.3.1.3.3" xref="S6.E4.m1.1.1.3.3.1.3.3.cmml">L</mi></mrow></msub></mtd></mtr></mtable><mo id="S6.E4.m1.1.2.3.2.2" xref="S6.E4.m1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.E4.m1.1b"><apply id="S6.E4.m1.1.2.cmml" xref="S6.E4.m1.1.2"><eq id="S6.E4.m1.1.2.1.cmml" xref="S6.E4.m1.1.2.1"></eq><apply id="S6.E4.m1.1.2.2.cmml" xref="S6.E4.m1.1.2.2"><csymbol cd="ambiguous" id="S6.E4.m1.1.2.2.1.cmml" xref="S6.E4.m1.1.2.2">subscript</csymbol><ci id="S6.E4.m1.1.2.2.2.cmml" xref="S6.E4.m1.1.2.2.2">𝐌</ci><ci id="S6.E4.m1.1.2.2.3.cmml" xref="S6.E4.m1.1.2.2.3">𝐬</ci></apply><matrix id="S6.E4.m1.1.1.cmml" xref="S6.E4.m1.1.2.3.2"><matrixrow id="S6.E4.m1.1.1a.cmml" xref="S6.E4.m1.1.2.3.2"><apply id="S6.E4.m1.1.1.1.1.1.cmml" xref="S6.E4.m1.1.1.1.1.1"><csymbol cd="ambiguous" id="S6.E4.m1.1.1.1.1.1.1.cmml" xref="S6.E4.m1.1.1.1.1.1">subscript</csymbol><ci id="S6.E4.m1.1.1.1.1.1.2.cmml" xref="S6.E4.m1.1.1.1.1.1.2">𝑚</ci><cn type="integer" id="S6.E4.m1.1.1.1.1.1.3.cmml" xref="S6.E4.m1.1.1.1.1.1.3">11</cn></apply><ci id="S6.E4.m1.1.1.1.2.1.cmml" xref="S6.E4.m1.1.1.1.2.1">…</ci><apply id="S6.E4.m1.1.1.1.3.1.cmml" xref="S6.E4.m1.1.1.1.3.1"><csymbol cd="ambiguous" id="S6.E4.m1.1.1.1.3.1.1.cmml" xref="S6.E4.m1.1.1.1.3.1">subscript</csymbol><ci id="S6.E4.m1.1.1.1.3.1.2.cmml" xref="S6.E4.m1.1.1.1.3.1.2">𝑚</ci><apply id="S6.E4.m1.1.1.1.3.1.3.cmml" xref="S6.E4.m1.1.1.1.3.1.3"><times id="S6.E4.m1.1.1.1.3.1.3.1.cmml" xref="S6.E4.m1.1.1.1.3.1.3.1"></times><cn type="integer" id="S6.E4.m1.1.1.1.3.1.3.2.cmml" xref="S6.E4.m1.1.1.1.3.1.3.2">1</cn><ci id="S6.E4.m1.1.1.1.3.1.3.3.cmml" xref="S6.E4.m1.1.1.1.3.1.3.3">𝐿</ci></apply></apply></matrixrow><matrixrow id="S6.E4.m1.1.1b.cmml" xref="S6.E4.m1.1.2.3.2"><ci id="S6.E4.m1.1.1.2.1.1.cmml" xref="S6.E4.m1.1.1.2.1.1">⋮</ci><ci id="S6.E4.m1.1.1.2.2.1.cmml" xref="S6.E4.m1.1.1.2.2.1">⋮</ci><ci id="S6.E4.m1.1.1.2.3.1.cmml" xref="S6.E4.m1.1.1.2.3.1">⋮</ci></matrixrow><matrixrow id="S6.E4.m1.1.1c.cmml" xref="S6.E4.m1.1.2.3.2"><apply id="S6.E4.m1.1.1.3.1.1.cmml" xref="S6.E4.m1.1.1.3.1.1"><csymbol cd="ambiguous" id="S6.E4.m1.1.1.3.1.1.1.cmml" xref="S6.E4.m1.1.1.3.1.1">subscript</csymbol><ci id="S6.E4.m1.1.1.3.1.1.2.cmml" xref="S6.E4.m1.1.1.3.1.1.2">𝑚</ci><apply id="S6.E4.m1.1.1.3.1.1.3.cmml" xref="S6.E4.m1.1.1.3.1.1.3"><times id="S6.E4.m1.1.1.3.1.1.3.1.cmml" xref="S6.E4.m1.1.1.3.1.1.3.1"></times><ci id="S6.E4.m1.1.1.3.1.1.3.2.cmml" xref="S6.E4.m1.1.1.3.1.1.3.2">𝑁</ci><cn type="integer" id="S6.E4.m1.1.1.3.1.1.3.3.cmml" xref="S6.E4.m1.1.1.3.1.1.3.3">1</cn></apply></apply><ci id="S6.E4.m1.1.1.3.2.1.cmml" xref="S6.E4.m1.1.1.3.2.1">…</ci><apply id="S6.E4.m1.1.1.3.3.1.cmml" xref="S6.E4.m1.1.1.3.3.1"><csymbol cd="ambiguous" id="S6.E4.m1.1.1.3.3.1.1.cmml" xref="S6.E4.m1.1.1.3.3.1">subscript</csymbol><ci id="S6.E4.m1.1.1.3.3.1.2.cmml" xref="S6.E4.m1.1.1.3.3.1.2">𝑤</ci><apply id="S6.E4.m1.1.1.3.3.1.3.cmml" xref="S6.E4.m1.1.1.3.3.1.3"><times id="S6.E4.m1.1.1.3.3.1.3.1.cmml" xref="S6.E4.m1.1.1.3.3.1.3.1"></times><ci id="S6.E4.m1.1.1.3.3.1.3.2.cmml" xref="S6.E4.m1.1.1.3.3.1.3.2">𝑁</ci><ci id="S6.E4.m1.1.1.3.3.1.3.3.cmml" xref="S6.E4.m1.1.1.3.3.1.3.3">𝐿</ci></apply></apply></matrixrow></matrix></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.E4.m1.1c">\mathbf{M_{s}=}\left(\begin{array}[]{ccc}m_{11}&amp;\ldots&amp;m_{1L}\\
\vdots&amp;\vdots&amp;\vdots\\
m_{N1}&amp;\ldots&amp;w_{NL}\\
\end{array}\right)</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(4)</span></td>
</tr></tbody>
</table>
<br class="ltx_break">
</div>
<div id="S6.SS2.p6" class="ltx_para">
<p id="S6.SS2.p6.2" class="ltx_p">As mentioned before, the default sequence-weighting method used by HMMER package is a high quality algorithm. Therefore, we combine both the default HMMER’s <math id="S6.SS2.p6.1.m1.1" class="ltx_Math" alttext="M" display="inline"><semantics id="S6.SS2.p6.1.m1.1a"><mi id="S6.SS2.p6.1.m1.1.1" xref="S6.SS2.p6.1.m1.1.1.cmml">M</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p6.1.m1.1b"><ci id="S6.SS2.p6.1.m1.1.1.cmml" xref="S6.SS2.p6.1.m1.1.1">𝑀</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p6.1.m1.1c">M</annotation></semantics></math> matrix in (<a href="#S6.E3" title="In 6.2 Sequence Weighting ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>) and the of <math id="S6.SS2.p6.2.m2.1" class="ltx_Math" alttext="M_{s}" display="inline"><semantics id="S6.SS2.p6.2.m2.1a"><msub id="S6.SS2.p6.2.m2.1.1" xref="S6.SS2.p6.2.m2.1.1.cmml"><mi id="S6.SS2.p6.2.m2.1.1.2" xref="S6.SS2.p6.2.m2.1.1.2.cmml">M</mi><mi id="S6.SS2.p6.2.m2.1.1.3" xref="S6.SS2.p6.2.m2.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="S6.SS2.p6.2.m2.1b"><apply id="S6.SS2.p6.2.m2.1.1.cmml" xref="S6.SS2.p6.2.m2.1.1"><csymbol cd="ambiguous" id="S6.SS2.p6.2.m2.1.1.1.cmml" xref="S6.SS2.p6.2.m2.1.1">subscript</csymbol><ci id="S6.SS2.p6.2.m2.1.1.2.cmml" xref="S6.SS2.p6.2.m2.1.1.2">𝑀</ci><ci id="S6.SS2.p6.2.m2.1.1.3.cmml" xref="S6.SS2.p6.2.m2.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p6.2.m2.1c">M_{s}</annotation></semantics></math> structural matrix in (<a href="#S6.E4" title="In 6.2 Sequence Weighting ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a>), as shown in (<a href="#S6.E5" title="In 6.2 Sequence Weighting ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>).

<br class="ltx_break"></p>
</div>
<div id="S6.SS2.p7" class="ltx_para">
<table id="S6.E5" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S6.E5.m1.1" class="ltx_Math" alttext="\mathbf{M^{{}^{\prime}}_{s}=MM_{s}^{T}}=\left(\begin{array}[]{ccc}w_{11}m_{11}&amp;\ldots&amp;w_{1L}m_{1L}\\
\vdots&amp;\vdots&amp;\vdots\\
w_{N1}m_{N1}&amp;\ldots&amp;w_{NL}m_{NL}\\
\end{array}\right)" display="block"><semantics id="S6.E5.m1.1a"><mrow id="S6.E5.m1.1.2" xref="S6.E5.m1.1.2.cmml"><msubsup id="S6.E5.m1.1.2.2" xref="S6.E5.m1.1.2.2.cmml"><mi id="S6.E5.m1.1.2.2.2.2" xref="S6.E5.m1.1.2.2.2.2.cmml">𝐌</mi><mi id="S6.E5.m1.1.2.2.3" xref="S6.E5.m1.1.2.2.3.cmml">𝐬</mi><msup id="S6.E5.m1.1.2.2.2.3" xref="S6.E5.m1.1.2.2.2.3.cmml"><mi id="S6.E5.m1.1.2.2.2.3a" xref="S6.E5.m1.1.2.2.2.3.cmml"></mi><mo id="S6.E5.m1.1.2.2.2.3.1" xref="S6.E5.m1.1.2.2.2.3.1.cmml">′</mo></msup></msubsup><mo id="S6.E5.m1.1.2.3" xref="S6.E5.m1.1.2.3.cmml">=</mo><msubsup id="S6.E5.m1.1.2.4" xref="S6.E5.m1.1.2.4.cmml"><mi id="S6.E5.m1.1.2.4.2.2" xref="S6.E5.m1.1.2.4.2.2.cmml">𝐌𝐌</mi><mi id="S6.E5.m1.1.2.4.2.3" xref="S6.E5.m1.1.2.4.2.3.cmml">𝐬</mi><mi id="S6.E5.m1.1.2.4.3" xref="S6.E5.m1.1.2.4.3.cmml">𝐓</mi></msubsup><mo id="S6.E5.m1.1.2.5" xref="S6.E5.m1.1.2.5.cmml">=</mo><mrow id="S6.E5.m1.1.2.6.2" xref="S6.E5.m1.1.1.cmml"><mo id="S6.E5.m1.1.2.6.2.1" xref="S6.E5.m1.1.1.cmml">(</mo><mtable columnspacing="5pt" displaystyle="true" rowspacing="0pt" id="S6.E5.m1.1.1" xref="S6.E5.m1.1.1.cmml"><mtr id="S6.E5.m1.1.1a" xref="S6.E5.m1.1.1.cmml"><mtd id="S6.E5.m1.1.1b" xref="S6.E5.m1.1.1.cmml"><mrow id="S6.E5.m1.1.1.1.1.1" xref="S6.E5.m1.1.1.1.1.1.cmml"><msub id="S6.E5.m1.1.1.1.1.1.2" xref="S6.E5.m1.1.1.1.1.1.2.cmml"><mi id="S6.E5.m1.1.1.1.1.1.2.2" xref="S6.E5.m1.1.1.1.1.1.2.2.cmml">w</mi><mn id="S6.E5.m1.1.1.1.1.1.2.3" xref="S6.E5.m1.1.1.1.1.1.2.3.cmml">11</mn></msub><mo lspace="0em" rspace="0em" id="S6.E5.m1.1.1.1.1.1.1" xref="S6.E5.m1.1.1.1.1.1.1.cmml">​</mo><msub id="S6.E5.m1.1.1.1.1.1.3" xref="S6.E5.m1.1.1.1.1.1.3.cmml"><mi id="S6.E5.m1.1.1.1.1.1.3.2" xref="S6.E5.m1.1.1.1.1.1.3.2.cmml">m</mi><mn id="S6.E5.m1.1.1.1.1.1.3.3" xref="S6.E5.m1.1.1.1.1.1.3.3.cmml">11</mn></msub></mrow></mtd><mtd id="S6.E5.m1.1.1c" xref="S6.E5.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E5.m1.1.1.1.2.1" xref="S6.E5.m1.1.1.1.2.1.cmml">…</mi></mtd><mtd id="S6.E5.m1.1.1d" xref="S6.E5.m1.1.1.cmml"><mrow id="S6.E5.m1.1.1.1.3.1" xref="S6.E5.m1.1.1.1.3.1.cmml"><msub id="S6.E5.m1.1.1.1.3.1.2" xref="S6.E5.m1.1.1.1.3.1.2.cmml"><mi id="S6.E5.m1.1.1.1.3.1.2.2" xref="S6.E5.m1.1.1.1.3.1.2.2.cmml">w</mi><mrow id="S6.E5.m1.1.1.1.3.1.2.3" xref="S6.E5.m1.1.1.1.3.1.2.3.cmml"><mn id="S6.E5.m1.1.1.1.3.1.2.3.2" xref="S6.E5.m1.1.1.1.3.1.2.3.2.cmml">1</mn><mo lspace="0em" rspace="0em" id="S6.E5.m1.1.1.1.3.1.2.3.1" xref="S6.E5.m1.1.1.1.3.1.2.3.1.cmml">​</mo><mi id="S6.E5.m1.1.1.1.3.1.2.3.3" xref="S6.E5.m1.1.1.1.3.1.2.3.3.cmml">L</mi></mrow></msub><mo lspace="0em" rspace="0em" id="S6.E5.m1.1.1.1.3.1.1" xref="S6.E5.m1.1.1.1.3.1.1.cmml">​</mo><msub id="S6.E5.m1.1.1.1.3.1.3" xref="S6.E5.m1.1.1.1.3.1.3.cmml"><mi id="S6.E5.m1.1.1.1.3.1.3.2" xref="S6.E5.m1.1.1.1.3.1.3.2.cmml">m</mi><mrow id="S6.E5.m1.1.1.1.3.1.3.3" xref="S6.E5.m1.1.1.1.3.1.3.3.cmml"><mn id="S6.E5.m1.1.1.1.3.1.3.3.2" xref="S6.E5.m1.1.1.1.3.1.3.3.2.cmml">1</mn><mo lspace="0em" rspace="0em" id="S6.E5.m1.1.1.1.3.1.3.3.1" xref="S6.E5.m1.1.1.1.3.1.3.3.1.cmml">​</mo><mi id="S6.E5.m1.1.1.1.3.1.3.3.3" xref="S6.E5.m1.1.1.1.3.1.3.3.3.cmml">L</mi></mrow></msub></mrow></mtd></mtr><mtr id="S6.E5.m1.1.1e" xref="S6.E5.m1.1.1.cmml"><mtd id="S6.E5.m1.1.1f" xref="S6.E5.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E5.m1.1.1.2.1.1" xref="S6.E5.m1.1.1.2.1.1.cmml">⋮</mi></mtd><mtd id="S6.E5.m1.1.1g" xref="S6.E5.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E5.m1.1.1.2.2.1" xref="S6.E5.m1.1.1.2.2.1.cmml">⋮</mi></mtd><mtd id="S6.E5.m1.1.1h" xref="S6.E5.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E5.m1.1.1.2.3.1" xref="S6.E5.m1.1.1.2.3.1.cmml">⋮</mi></mtd></mtr><mtr id="S6.E5.m1.1.1i" xref="S6.E5.m1.1.1.cmml"><mtd id="S6.E5.m1.1.1j" xref="S6.E5.m1.1.1.cmml"><mrow id="S6.E5.m1.1.1.3.1.1" xref="S6.E5.m1.1.1.3.1.1.cmml"><msub id="S6.E5.m1.1.1.3.1.1.2" xref="S6.E5.m1.1.1.3.1.1.2.cmml"><mi id="S6.E5.m1.1.1.3.1.1.2.2" xref="S6.E5.m1.1.1.3.1.1.2.2.cmml">w</mi><mrow id="S6.E5.m1.1.1.3.1.1.2.3" xref="S6.E5.m1.1.1.3.1.1.2.3.cmml"><mi id="S6.E5.m1.1.1.3.1.1.2.3.2" xref="S6.E5.m1.1.1.3.1.1.2.3.2.cmml">N</mi><mo lspace="0em" rspace="0em" id="S6.E5.m1.1.1.3.1.1.2.3.1" xref="S6.E5.m1.1.1.3.1.1.2.3.1.cmml">​</mo><mn id="S6.E5.m1.1.1.3.1.1.2.3.3" xref="S6.E5.m1.1.1.3.1.1.2.3.3.cmml">1</mn></mrow></msub><mo lspace="0em" rspace="0em" id="S6.E5.m1.1.1.3.1.1.1" xref="S6.E5.m1.1.1.3.1.1.1.cmml">​</mo><msub id="S6.E5.m1.1.1.3.1.1.3" xref="S6.E5.m1.1.1.3.1.1.3.cmml"><mi id="S6.E5.m1.1.1.3.1.1.3.2" xref="S6.E5.m1.1.1.3.1.1.3.2.cmml">m</mi><mrow id="S6.E5.m1.1.1.3.1.1.3.3" xref="S6.E5.m1.1.1.3.1.1.3.3.cmml"><mi id="S6.E5.m1.1.1.3.1.1.3.3.2" xref="S6.E5.m1.1.1.3.1.1.3.3.2.cmml">N</mi><mo lspace="0em" rspace="0em" id="S6.E5.m1.1.1.3.1.1.3.3.1" xref="S6.E5.m1.1.1.3.1.1.3.3.1.cmml">​</mo><mn id="S6.E5.m1.1.1.3.1.1.3.3.3" xref="S6.E5.m1.1.1.3.1.1.3.3.3.cmml">1</mn></mrow></msub></mrow></mtd><mtd id="S6.E5.m1.1.1k" xref="S6.E5.m1.1.1.cmml"><mi mathvariant="normal" id="S6.E5.m1.1.1.3.2.1" xref="S6.E5.m1.1.1.3.2.1.cmml">…</mi></mtd><mtd id="S6.E5.m1.1.1l" xref="S6.E5.m1.1.1.cmml"><mrow id="S6.E5.m1.1.1.3.3.1" xref="S6.E5.m1.1.1.3.3.1.cmml"><msub id="S6.E5.m1.1.1.3.3.1.2" xref="S6.E5.m1.1.1.3.3.1.2.cmml"><mi id="S6.E5.m1.1.1.3.3.1.2.2" xref="S6.E5.m1.1.1.3.3.1.2.2.cmml">w</mi><mrow id="S6.E5.m1.1.1.3.3.1.2.3" xref="S6.E5.m1.1.1.3.3.1.2.3.cmml"><mi id="S6.E5.m1.1.1.3.3.1.2.3.2" xref="S6.E5.m1.1.1.3.3.1.2.3.2.cmml">N</mi><mo lspace="0em" rspace="0em" id="S6.E5.m1.1.1.3.3.1.2.3.1" xref="S6.E5.m1.1.1.3.3.1.2.3.1.cmml">​</mo><mi id="S6.E5.m1.1.1.3.3.1.2.3.3" xref="S6.E5.m1.1.1.3.3.1.2.3.3.cmml">L</mi></mrow></msub><mo lspace="0em" rspace="0em" id="S6.E5.m1.1.1.3.3.1.1" xref="S6.E5.m1.1.1.3.3.1.1.cmml">​</mo><msub id="S6.E5.m1.1.1.3.3.1.3" xref="S6.E5.m1.1.1.3.3.1.3.cmml"><mi id="S6.E5.m1.1.1.3.3.1.3.2" xref="S6.E5.m1.1.1.3.3.1.3.2.cmml">m</mi><mrow id="S6.E5.m1.1.1.3.3.1.3.3" xref="S6.E5.m1.1.1.3.3.1.3.3.cmml"><mi id="S6.E5.m1.1.1.3.3.1.3.3.2" xref="S6.E5.m1.1.1.3.3.1.3.3.2.cmml">N</mi><mo lspace="0em" rspace="0em" id="S6.E5.m1.1.1.3.3.1.3.3.1" xref="S6.E5.m1.1.1.3.3.1.3.3.1.cmml">​</mo><mi id="S6.E5.m1.1.1.3.3.1.3.3.3" xref="S6.E5.m1.1.1.3.3.1.3.3.3.cmml">L</mi></mrow></msub></mrow></mtd></mtr></mtable><mo id="S6.E5.m1.1.2.6.2.2" xref="S6.E5.m1.1.1.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.E5.m1.1b"><apply id="S6.E5.m1.1.2.cmml" xref="S6.E5.m1.1.2"><and id="S6.E5.m1.1.2a.cmml" xref="S6.E5.m1.1.2"></and><apply id="S6.E5.m1.1.2b.cmml" xref="S6.E5.m1.1.2"><eq id="S6.E5.m1.1.2.3.cmml" xref="S6.E5.m1.1.2.3"></eq><apply id="S6.E5.m1.1.2.2.cmml" xref="S6.E5.m1.1.2.2"><csymbol cd="ambiguous" id="S6.E5.m1.1.2.2.1.cmml" xref="S6.E5.m1.1.2.2">subscript</csymbol><apply id="S6.E5.m1.1.2.2.2.cmml" xref="S6.E5.m1.1.2.2"><csymbol cd="ambiguous" id="S6.E5.m1.1.2.2.2.1.cmml" xref="S6.E5.m1.1.2.2">superscript</csymbol><ci id="S6.E5.m1.1.2.2.2.2.cmml" xref="S6.E5.m1.1.2.2.2.2">𝐌</ci><apply id="S6.E5.m1.1.2.2.2.3.cmml" xref="S6.E5.m1.1.2.2.2.3"><ci id="S6.E5.m1.1.2.2.2.3.1.cmml" xref="S6.E5.m1.1.2.2.2.3.1">′</ci></apply></apply><ci id="S6.E5.m1.1.2.2.3.cmml" xref="S6.E5.m1.1.2.2.3">𝐬</ci></apply><apply id="S6.E5.m1.1.2.4.cmml" xref="S6.E5.m1.1.2.4"><csymbol cd="ambiguous" id="S6.E5.m1.1.2.4.1.cmml" xref="S6.E5.m1.1.2.4">superscript</csymbol><apply id="S6.E5.m1.1.2.4.2.cmml" xref="S6.E5.m1.1.2.4"><csymbol cd="ambiguous" id="S6.E5.m1.1.2.4.2.1.cmml" xref="S6.E5.m1.1.2.4">subscript</csymbol><ci id="S6.E5.m1.1.2.4.2.2.cmml" xref="S6.E5.m1.1.2.4.2.2">𝐌𝐌</ci><ci id="S6.E5.m1.1.2.4.2.3.cmml" xref="S6.E5.m1.1.2.4.2.3">𝐬</ci></apply><ci id="S6.E5.m1.1.2.4.3.cmml" xref="S6.E5.m1.1.2.4.3">𝐓</ci></apply></apply><apply id="S6.E5.m1.1.2c.cmml" xref="S6.E5.m1.1.2"><eq id="S6.E5.m1.1.2.5.cmml" xref="S6.E5.m1.1.2.5"></eq><share href="#S6.E5.m1.1.2.4.cmml" id="S6.E5.m1.1.2d.cmml" xref="S6.E5.m1.1.2"></share><matrix id="S6.E5.m1.1.1.cmml" xref="S6.E5.m1.1.2.6.2"><matrixrow id="S6.E5.m1.1.1a.cmml" xref="S6.E5.m1.1.2.6.2"><apply id="S6.E5.m1.1.1.1.1.1.cmml" xref="S6.E5.m1.1.1.1.1.1"><times id="S6.E5.m1.1.1.1.1.1.1.cmml" xref="S6.E5.m1.1.1.1.1.1.1"></times><apply id="S6.E5.m1.1.1.1.1.1.2.cmml" xref="S6.E5.m1.1.1.1.1.1.2"><csymbol cd="ambiguous" id="S6.E5.m1.1.1.1.1.1.2.1.cmml" xref="S6.E5.m1.1.1.1.1.1.2">subscript</csymbol><ci id="S6.E5.m1.1.1.1.1.1.2.2.cmml" xref="S6.E5.m1.1.1.1.1.1.2.2">𝑤</ci><cn type="integer" id="S6.E5.m1.1.1.1.1.1.2.3.cmml" xref="S6.E5.m1.1.1.1.1.1.2.3">11</cn></apply><apply id="S6.E5.m1.1.1.1.1.1.3.cmml" xref="S6.E5.m1.1.1.1.1.1.3"><csymbol cd="ambiguous" id="S6.E5.m1.1.1.1.1.1.3.1.cmml" xref="S6.E5.m1.1.1.1.1.1.3">subscript</csymbol><ci id="S6.E5.m1.1.1.1.1.1.3.2.cmml" xref="S6.E5.m1.1.1.1.1.1.3.2">𝑚</ci><cn type="integer" id="S6.E5.m1.1.1.1.1.1.3.3.cmml" xref="S6.E5.m1.1.1.1.1.1.3.3">11</cn></apply></apply><ci id="S6.E5.m1.1.1.1.2.1.cmml" xref="S6.E5.m1.1.1.1.2.1">…</ci><apply id="S6.E5.m1.1.1.1.3.1.cmml" xref="S6.E5.m1.1.1.1.3.1"><times id="S6.E5.m1.1.1.1.3.1.1.cmml" xref="S6.E5.m1.1.1.1.3.1.1"></times><apply id="S6.E5.m1.1.1.1.3.1.2.cmml" xref="S6.E5.m1.1.1.1.3.1.2"><csymbol cd="ambiguous" id="S6.E5.m1.1.1.1.3.1.2.1.cmml" xref="S6.E5.m1.1.1.1.3.1.2">subscript</csymbol><ci id="S6.E5.m1.1.1.1.3.1.2.2.cmml" xref="S6.E5.m1.1.1.1.3.1.2.2">𝑤</ci><apply id="S6.E5.m1.1.1.1.3.1.2.3.cmml" xref="S6.E5.m1.1.1.1.3.1.2.3"><times id="S6.E5.m1.1.1.1.3.1.2.3.1.cmml" xref="S6.E5.m1.1.1.1.3.1.2.3.1"></times><cn type="integer" id="S6.E5.m1.1.1.1.3.1.2.3.2.cmml" xref="S6.E5.m1.1.1.1.3.1.2.3.2">1</cn><ci id="S6.E5.m1.1.1.1.3.1.2.3.3.cmml" xref="S6.E5.m1.1.1.1.3.1.2.3.3">𝐿</ci></apply></apply><apply id="S6.E5.m1.1.1.1.3.1.3.cmml" xref="S6.E5.m1.1.1.1.3.1.3"><csymbol cd="ambiguous" id="S6.E5.m1.1.1.1.3.1.3.1.cmml" xref="S6.E5.m1.1.1.1.3.1.3">subscript</csymbol><ci id="S6.E5.m1.1.1.1.3.1.3.2.cmml" xref="S6.E5.m1.1.1.1.3.1.3.2">𝑚</ci><apply id="S6.E5.m1.1.1.1.3.1.3.3.cmml" xref="S6.E5.m1.1.1.1.3.1.3.3"><times id="S6.E5.m1.1.1.1.3.1.3.3.1.cmml" xref="S6.E5.m1.1.1.1.3.1.3.3.1"></times><cn type="integer" id="S6.E5.m1.1.1.1.3.1.3.3.2.cmml" xref="S6.E5.m1.1.1.1.3.1.3.3.2">1</cn><ci id="S6.E5.m1.1.1.1.3.1.3.3.3.cmml" xref="S6.E5.m1.1.1.1.3.1.3.3.3">𝐿</ci></apply></apply></apply></matrixrow><matrixrow id="S6.E5.m1.1.1b.cmml" xref="S6.E5.m1.1.2.6.2"><ci id="S6.E5.m1.1.1.2.1.1.cmml" xref="S6.E5.m1.1.1.2.1.1">⋮</ci><ci id="S6.E5.m1.1.1.2.2.1.cmml" xref="S6.E5.m1.1.1.2.2.1">⋮</ci><ci id="S6.E5.m1.1.1.2.3.1.cmml" xref="S6.E5.m1.1.1.2.3.1">⋮</ci></matrixrow><matrixrow id="S6.E5.m1.1.1c.cmml" xref="S6.E5.m1.1.2.6.2"><apply id="S6.E5.m1.1.1.3.1.1.cmml" xref="S6.E5.m1.1.1.3.1.1"><times id="S6.E5.m1.1.1.3.1.1.1.cmml" xref="S6.E5.m1.1.1.3.1.1.1"></times><apply id="S6.E5.m1.1.1.3.1.1.2.cmml" xref="S6.E5.m1.1.1.3.1.1.2"><csymbol cd="ambiguous" id="S6.E5.m1.1.1.3.1.1.2.1.cmml" xref="S6.E5.m1.1.1.3.1.1.2">subscript</csymbol><ci id="S6.E5.m1.1.1.3.1.1.2.2.cmml" xref="S6.E5.m1.1.1.3.1.1.2.2">𝑤</ci><apply id="S6.E5.m1.1.1.3.1.1.2.3.cmml" xref="S6.E5.m1.1.1.3.1.1.2.3"><times id="S6.E5.m1.1.1.3.1.1.2.3.1.cmml" xref="S6.E5.m1.1.1.3.1.1.2.3.1"></times><ci id="S6.E5.m1.1.1.3.1.1.2.3.2.cmml" xref="S6.E5.m1.1.1.3.1.1.2.3.2">𝑁</ci><cn type="integer" id="S6.E5.m1.1.1.3.1.1.2.3.3.cmml" xref="S6.E5.m1.1.1.3.1.1.2.3.3">1</cn></apply></apply><apply id="S6.E5.m1.1.1.3.1.1.3.cmml" xref="S6.E5.m1.1.1.3.1.1.3"><csymbol cd="ambiguous" id="S6.E5.m1.1.1.3.1.1.3.1.cmml" xref="S6.E5.m1.1.1.3.1.1.3">subscript</csymbol><ci id="S6.E5.m1.1.1.3.1.1.3.2.cmml" xref="S6.E5.m1.1.1.3.1.1.3.2">𝑚</ci><apply id="S6.E5.m1.1.1.3.1.1.3.3.cmml" xref="S6.E5.m1.1.1.3.1.1.3.3"><times id="S6.E5.m1.1.1.3.1.1.3.3.1.cmml" xref="S6.E5.m1.1.1.3.1.1.3.3.1"></times><ci id="S6.E5.m1.1.1.3.1.1.3.3.2.cmml" xref="S6.E5.m1.1.1.3.1.1.3.3.2">𝑁</ci><cn type="integer" id="S6.E5.m1.1.1.3.1.1.3.3.3.cmml" xref="S6.E5.m1.1.1.3.1.1.3.3.3">1</cn></apply></apply></apply><ci id="S6.E5.m1.1.1.3.2.1.cmml" xref="S6.E5.m1.1.1.3.2.1">…</ci><apply id="S6.E5.m1.1.1.3.3.1.cmml" xref="S6.E5.m1.1.1.3.3.1"><times id="S6.E5.m1.1.1.3.3.1.1.cmml" xref="S6.E5.m1.1.1.3.3.1.1"></times><apply id="S6.E5.m1.1.1.3.3.1.2.cmml" xref="S6.E5.m1.1.1.3.3.1.2"><csymbol cd="ambiguous" id="S6.E5.m1.1.1.3.3.1.2.1.cmml" xref="S6.E5.m1.1.1.3.3.1.2">subscript</csymbol><ci id="S6.E5.m1.1.1.3.3.1.2.2.cmml" xref="S6.E5.m1.1.1.3.3.1.2.2">𝑤</ci><apply id="S6.E5.m1.1.1.3.3.1.2.3.cmml" xref="S6.E5.m1.1.1.3.3.1.2.3"><times id="S6.E5.m1.1.1.3.3.1.2.3.1.cmml" xref="S6.E5.m1.1.1.3.3.1.2.3.1"></times><ci id="S6.E5.m1.1.1.3.3.1.2.3.2.cmml" xref="S6.E5.m1.1.1.3.3.1.2.3.2">𝑁</ci><ci id="S6.E5.m1.1.1.3.3.1.2.3.3.cmml" xref="S6.E5.m1.1.1.3.3.1.2.3.3">𝐿</ci></apply></apply><apply id="S6.E5.m1.1.1.3.3.1.3.cmml" xref="S6.E5.m1.1.1.3.3.1.3"><csymbol cd="ambiguous" id="S6.E5.m1.1.1.3.3.1.3.1.cmml" xref="S6.E5.m1.1.1.3.3.1.3">subscript</csymbol><ci id="S6.E5.m1.1.1.3.3.1.3.2.cmml" xref="S6.E5.m1.1.1.3.3.1.3.2">𝑚</ci><apply id="S6.E5.m1.1.1.3.3.1.3.3.cmml" xref="S6.E5.m1.1.1.3.3.1.3.3"><times id="S6.E5.m1.1.1.3.3.1.3.3.1.cmml" xref="S6.E5.m1.1.1.3.3.1.3.3.1"></times><ci id="S6.E5.m1.1.1.3.3.1.3.3.2.cmml" xref="S6.E5.m1.1.1.3.3.1.3.3.2">𝑁</ci><ci id="S6.E5.m1.1.1.3.3.1.3.3.3.cmml" xref="S6.E5.m1.1.1.3.3.1.3.3.3">𝐿</ci></apply></apply></apply></matrixrow></matrix></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.E5.m1.1c">\mathbf{M^{{}^{\prime}}_{s}=MM_{s}^{T}}=\left(\begin{array}[]{ccc}w_{11}m_{11}&amp;\ldots&amp;w_{1L}m_{1L}\\
\vdots&amp;\vdots&amp;\vdots\\
w_{N1}m_{N1}&amp;\ldots&amp;w_{NL}m_{NL}\\
\end{array}\right)</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(5)</span></td>
</tr></tbody>
</table>
<br class="ltx_break">
</div>
<div id="S6.SS2.p8" class="ltx_para">
<p id="S6.SS2.p8.4" class="ltx_p">However, introducing weights affect the computation of the observed frequencies. More precisely, the observed frequency <math id="S6.SS2.p8.1.m1.1" class="ltx_Math" alttext="c_{j}(\sigma)" display="inline"><semantics id="S6.SS2.p8.1.m1.1a"><mrow id="S6.SS2.p8.1.m1.1.2" xref="S6.SS2.p8.1.m1.1.2.cmml"><msub id="S6.SS2.p8.1.m1.1.2.2" xref="S6.SS2.p8.1.m1.1.2.2.cmml"><mi id="S6.SS2.p8.1.m1.1.2.2.2" xref="S6.SS2.p8.1.m1.1.2.2.2.cmml">c</mi><mi id="S6.SS2.p8.1.m1.1.2.2.3" xref="S6.SS2.p8.1.m1.1.2.2.3.cmml">j</mi></msub><mo lspace="0em" rspace="0em" id="S6.SS2.p8.1.m1.1.2.1" xref="S6.SS2.p8.1.m1.1.2.1.cmml">​</mo><mrow id="S6.SS2.p8.1.m1.1.2.3.2" xref="S6.SS2.p8.1.m1.1.2.cmml"><mo stretchy="false" id="S6.SS2.p8.1.m1.1.2.3.2.1" xref="S6.SS2.p8.1.m1.1.2.cmml">(</mo><mi id="S6.SS2.p8.1.m1.1.1" xref="S6.SS2.p8.1.m1.1.1.cmml">σ</mi><mo stretchy="false" id="S6.SS2.p8.1.m1.1.2.3.2.2" xref="S6.SS2.p8.1.m1.1.2.cmml">)</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS2.p8.1.m1.1b"><apply id="S6.SS2.p8.1.m1.1.2.cmml" xref="S6.SS2.p8.1.m1.1.2"><times id="S6.SS2.p8.1.m1.1.2.1.cmml" xref="S6.SS2.p8.1.m1.1.2.1"></times><apply id="S6.SS2.p8.1.m1.1.2.2.cmml" xref="S6.SS2.p8.1.m1.1.2.2"><csymbol cd="ambiguous" id="S6.SS2.p8.1.m1.1.2.2.1.cmml" xref="S6.SS2.p8.1.m1.1.2.2">subscript</csymbol><ci id="S6.SS2.p8.1.m1.1.2.2.2.cmml" xref="S6.SS2.p8.1.m1.1.2.2.2">𝑐</ci><ci id="S6.SS2.p8.1.m1.1.2.2.3.cmml" xref="S6.SS2.p8.1.m1.1.2.2.3">𝑗</ci></apply><ci id="S6.SS2.p8.1.m1.1.1.cmml" xref="S6.SS2.p8.1.m1.1.1">𝜎</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p8.1.m1.1c">c_{j}(\sigma)</annotation></semantics></math> shown in <a href="#S6.E1" title="In 6.1 Profile HMMs ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a> is now found through the equation <a href="#S6.E6" title="In 6.2 Sequence Weighting ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">6</span></a>, where <math id="S6.SS2.p8.2.m2.1" class="ltx_Math" alttext="s_{ij}=w_{ij}m_{ij}" display="inline"><semantics id="S6.SS2.p8.2.m2.1a"><mrow id="S6.SS2.p8.2.m2.1.1" xref="S6.SS2.p8.2.m2.1.1.cmml"><msub id="S6.SS2.p8.2.m2.1.1.2" xref="S6.SS2.p8.2.m2.1.1.2.cmml"><mi id="S6.SS2.p8.2.m2.1.1.2.2" xref="S6.SS2.p8.2.m2.1.1.2.2.cmml">s</mi><mrow id="S6.SS2.p8.2.m2.1.1.2.3" xref="S6.SS2.p8.2.m2.1.1.2.3.cmml"><mi id="S6.SS2.p8.2.m2.1.1.2.3.2" xref="S6.SS2.p8.2.m2.1.1.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p8.2.m2.1.1.2.3.1" xref="S6.SS2.p8.2.m2.1.1.2.3.1.cmml">​</mo><mi id="S6.SS2.p8.2.m2.1.1.2.3.3" xref="S6.SS2.p8.2.m2.1.1.2.3.3.cmml">j</mi></mrow></msub><mo id="S6.SS2.p8.2.m2.1.1.1" xref="S6.SS2.p8.2.m2.1.1.1.cmml">=</mo><mrow id="S6.SS2.p8.2.m2.1.1.3" xref="S6.SS2.p8.2.m2.1.1.3.cmml"><msub id="S6.SS2.p8.2.m2.1.1.3.2" xref="S6.SS2.p8.2.m2.1.1.3.2.cmml"><mi id="S6.SS2.p8.2.m2.1.1.3.2.2" xref="S6.SS2.p8.2.m2.1.1.3.2.2.cmml">w</mi><mrow id="S6.SS2.p8.2.m2.1.1.3.2.3" xref="S6.SS2.p8.2.m2.1.1.3.2.3.cmml"><mi id="S6.SS2.p8.2.m2.1.1.3.2.3.2" xref="S6.SS2.p8.2.m2.1.1.3.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p8.2.m2.1.1.3.2.3.1" xref="S6.SS2.p8.2.m2.1.1.3.2.3.1.cmml">​</mo><mi id="S6.SS2.p8.2.m2.1.1.3.2.3.3" xref="S6.SS2.p8.2.m2.1.1.3.2.3.3.cmml">j</mi></mrow></msub><mo lspace="0em" rspace="0em" id="S6.SS2.p8.2.m2.1.1.3.1" xref="S6.SS2.p8.2.m2.1.1.3.1.cmml">​</mo><msub id="S6.SS2.p8.2.m2.1.1.3.3" xref="S6.SS2.p8.2.m2.1.1.3.3.cmml"><mi id="S6.SS2.p8.2.m2.1.1.3.3.2" xref="S6.SS2.p8.2.m2.1.1.3.3.2.cmml">m</mi><mrow id="S6.SS2.p8.2.m2.1.1.3.3.3" xref="S6.SS2.p8.2.m2.1.1.3.3.3.cmml"><mi id="S6.SS2.p8.2.m2.1.1.3.3.3.2" xref="S6.SS2.p8.2.m2.1.1.3.3.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p8.2.m2.1.1.3.3.3.1" xref="S6.SS2.p8.2.m2.1.1.3.3.3.1.cmml">​</mo><mi id="S6.SS2.p8.2.m2.1.1.3.3.3.3" xref="S6.SS2.p8.2.m2.1.1.3.3.3.3.cmml">j</mi></mrow></msub></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS2.p8.2.m2.1b"><apply id="S6.SS2.p8.2.m2.1.1.cmml" xref="S6.SS2.p8.2.m2.1.1"><eq id="S6.SS2.p8.2.m2.1.1.1.cmml" xref="S6.SS2.p8.2.m2.1.1.1"></eq><apply id="S6.SS2.p8.2.m2.1.1.2.cmml" xref="S6.SS2.p8.2.m2.1.1.2"><csymbol cd="ambiguous" id="S6.SS2.p8.2.m2.1.1.2.1.cmml" xref="S6.SS2.p8.2.m2.1.1.2">subscript</csymbol><ci id="S6.SS2.p8.2.m2.1.1.2.2.cmml" xref="S6.SS2.p8.2.m2.1.1.2.2">𝑠</ci><apply id="S6.SS2.p8.2.m2.1.1.2.3.cmml" xref="S6.SS2.p8.2.m2.1.1.2.3"><times id="S6.SS2.p8.2.m2.1.1.2.3.1.cmml" xref="S6.SS2.p8.2.m2.1.1.2.3.1"></times><ci id="S6.SS2.p8.2.m2.1.1.2.3.2.cmml" xref="S6.SS2.p8.2.m2.1.1.2.3.2">𝑖</ci><ci id="S6.SS2.p8.2.m2.1.1.2.3.3.cmml" xref="S6.SS2.p8.2.m2.1.1.2.3.3">𝑗</ci></apply></apply><apply id="S6.SS2.p8.2.m2.1.1.3.cmml" xref="S6.SS2.p8.2.m2.1.1.3"><times id="S6.SS2.p8.2.m2.1.1.3.1.cmml" xref="S6.SS2.p8.2.m2.1.1.3.1"></times><apply id="S6.SS2.p8.2.m2.1.1.3.2.cmml" xref="S6.SS2.p8.2.m2.1.1.3.2"><csymbol cd="ambiguous" id="S6.SS2.p8.2.m2.1.1.3.2.1.cmml" xref="S6.SS2.p8.2.m2.1.1.3.2">subscript</csymbol><ci id="S6.SS2.p8.2.m2.1.1.3.2.2.cmml" xref="S6.SS2.p8.2.m2.1.1.3.2.2">𝑤</ci><apply id="S6.SS2.p8.2.m2.1.1.3.2.3.cmml" xref="S6.SS2.p8.2.m2.1.1.3.2.3"><times id="S6.SS2.p8.2.m2.1.1.3.2.3.1.cmml" xref="S6.SS2.p8.2.m2.1.1.3.2.3.1"></times><ci id="S6.SS2.p8.2.m2.1.1.3.2.3.2.cmml" xref="S6.SS2.p8.2.m2.1.1.3.2.3.2">𝑖</ci><ci id="S6.SS2.p8.2.m2.1.1.3.2.3.3.cmml" xref="S6.SS2.p8.2.m2.1.1.3.2.3.3">𝑗</ci></apply></apply><apply id="S6.SS2.p8.2.m2.1.1.3.3.cmml" xref="S6.SS2.p8.2.m2.1.1.3.3"><csymbol cd="ambiguous" id="S6.SS2.p8.2.m2.1.1.3.3.1.cmml" xref="S6.SS2.p8.2.m2.1.1.3.3">subscript</csymbol><ci id="S6.SS2.p8.2.m2.1.1.3.3.2.cmml" xref="S6.SS2.p8.2.m2.1.1.3.3.2">𝑚</ci><apply id="S6.SS2.p8.2.m2.1.1.3.3.3.cmml" xref="S6.SS2.p8.2.m2.1.1.3.3.3"><times id="S6.SS2.p8.2.m2.1.1.3.3.3.1.cmml" xref="S6.SS2.p8.2.m2.1.1.3.3.3.1"></times><ci id="S6.SS2.p8.2.m2.1.1.3.3.3.2.cmml" xref="S6.SS2.p8.2.m2.1.1.3.3.3.2">𝑖</ci><ci id="S6.SS2.p8.2.m2.1.1.3.3.3.3.cmml" xref="S6.SS2.p8.2.m2.1.1.3.3.3.3">𝑗</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p8.2.m2.1c">s_{ij}=w_{ij}m_{ij}</annotation></semantics></math> is structural weight of residue <math id="S6.SS2.p8.3.m3.1" class="ltx_Math" alttext="\sigma" display="inline"><semantics id="S6.SS2.p8.3.m3.1a"><mi id="S6.SS2.p8.3.m3.1.1" xref="S6.SS2.p8.3.m3.1.1.cmml">σ</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p8.3.m3.1b"><ci id="S6.SS2.p8.3.m3.1.1.cmml" xref="S6.SS2.p8.3.m3.1.1">𝜎</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p8.3.m3.1c">\sigma</annotation></semantics></math>, according to <math id="S6.SS2.p8.4.m4.1" class="ltx_Math" alttext="M^{{}^{\prime}}_{s}" display="inline"><semantics id="S6.SS2.p8.4.m4.1a"><msubsup id="S6.SS2.p8.4.m4.1.1" xref="S6.SS2.p8.4.m4.1.1.cmml"><mi id="S6.SS2.p8.4.m4.1.1.2.2" xref="S6.SS2.p8.4.m4.1.1.2.2.cmml">M</mi><mi id="S6.SS2.p8.4.m4.1.1.3" xref="S6.SS2.p8.4.m4.1.1.3.cmml">s</mi><msup id="S6.SS2.p8.4.m4.1.1.2.3" xref="S6.SS2.p8.4.m4.1.1.2.3.cmml"><mi id="S6.SS2.p8.4.m4.1.1.2.3a" xref="S6.SS2.p8.4.m4.1.1.2.3.cmml"></mi><mo id="S6.SS2.p8.4.m4.1.1.2.3.1" xref="S6.SS2.p8.4.m4.1.1.2.3.1.cmml">′</mo></msup></msubsup><annotation-xml encoding="MathML-Content" id="S6.SS2.p8.4.m4.1b"><apply id="S6.SS2.p8.4.m4.1.1.cmml" xref="S6.SS2.p8.4.m4.1.1"><csymbol cd="ambiguous" id="S6.SS2.p8.4.m4.1.1.1.cmml" xref="S6.SS2.p8.4.m4.1.1">subscript</csymbol><apply id="S6.SS2.p8.4.m4.1.1.2.cmml" xref="S6.SS2.p8.4.m4.1.1"><csymbol cd="ambiguous" id="S6.SS2.p8.4.m4.1.1.2.1.cmml" xref="S6.SS2.p8.4.m4.1.1">superscript</csymbol><ci id="S6.SS2.p8.4.m4.1.1.2.2.cmml" xref="S6.SS2.p8.4.m4.1.1.2.2">𝑀</ci><apply id="S6.SS2.p8.4.m4.1.1.2.3.cmml" xref="S6.SS2.p8.4.m4.1.1.2.3"><ci id="S6.SS2.p8.4.m4.1.1.2.3.1.cmml" xref="S6.SS2.p8.4.m4.1.1.2.3.1">′</ci></apply></apply><ci id="S6.SS2.p8.4.m4.1.1.3.cmml" xref="S6.SS2.p8.4.m4.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p8.4.m4.1c">M^{{}^{\prime}}_{s}</annotation></semantics></math> matrix.</p>
</div>
<div id="S6.SS2.p9" class="ltx_para">
<table id="S6.E6" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S6.E6.m1.5" class="ltx_Math" alttext="\\
c_{j}(\sigma)=\sum_{i}^{N}f(\sigma)\therefore f(\sigma)=\left\{\begin{array}[]{cc}s_{ij},&amp;\textrm{if }\sigma\textrm{ is the amino-acid}\textrm{ in position ij}\\
\\
0,&amp;\textrm{otherwise}\end{array}\right." display="block"><semantics id="S6.E6.m1.5a"><mrow id="S6.E6.m1.5.6" xref="S6.E6.m1.5.6.cmml"><mrow id="S6.E6.m1.5.6.2" xref="S6.E6.m1.5.6.2.cmml"><mrow id="S6.E6.m1.5.6.2.2" xref="S6.E6.m1.5.6.2.2.cmml"><msub id="S6.E6.m1.5.6.2.2.2" xref="S6.E6.m1.5.6.2.2.2.cmml"><mi id="S6.E6.m1.5.6.2.2.2.2" xref="S6.E6.m1.5.6.2.2.2.2.cmml">c</mi><mi id="S6.E6.m1.5.6.2.2.2.3" xref="S6.E6.m1.5.6.2.2.2.3.cmml">j</mi></msub><mo lspace="0em" rspace="0em" id="S6.E6.m1.5.6.2.2.1" xref="S6.E6.m1.5.6.2.2.1.cmml">​</mo><mrow id="S6.E6.m1.5.6.2.2.3.2" xref="S6.E6.m1.5.6.2.2.cmml"><mo stretchy="false" id="S6.E6.m1.5.6.2.2.3.2.1" xref="S6.E6.m1.5.6.2.2.cmml">(</mo><mi id="S6.E6.m1.3.3" xref="S6.E6.m1.3.3.cmml">σ</mi><mo stretchy="false" id="S6.E6.m1.5.6.2.2.3.2.2" xref="S6.E6.m1.5.6.2.2.cmml">)</mo></mrow></mrow><mo rspace="0.111em" id="S6.E6.m1.5.6.2.1" xref="S6.E6.m1.5.6.2.1.cmml">=</mo><mrow id="S6.E6.m1.5.6.2.3" xref="S6.E6.m1.5.6.2.3.cmml"><munderover id="S6.E6.m1.5.6.2.3.1" xref="S6.E6.m1.5.6.2.3.1.cmml"><mo movablelimits="false" id="S6.E6.m1.5.6.2.3.1.2.2" xref="S6.E6.m1.5.6.2.3.1.2.2.cmml">∑</mo><mi id="S6.E6.m1.5.6.2.3.1.2.3" xref="S6.E6.m1.5.6.2.3.1.2.3.cmml">i</mi><mi id="S6.E6.m1.5.6.2.3.1.3" xref="S6.E6.m1.5.6.2.3.1.3.cmml">N</mi></munderover><mrow id="S6.E6.m1.5.6.2.3.2" xref="S6.E6.m1.5.6.2.3.2.cmml"><mi id="S6.E6.m1.5.6.2.3.2.2" xref="S6.E6.m1.5.6.2.3.2.2.cmml">f</mi><mo lspace="0em" rspace="0em" id="S6.E6.m1.5.6.2.3.2.1" xref="S6.E6.m1.5.6.2.3.2.1.cmml">​</mo><mrow id="S6.E6.m1.5.6.2.3.2.3.2" xref="S6.E6.m1.5.6.2.3.2.cmml"><mo stretchy="false" id="S6.E6.m1.5.6.2.3.2.3.2.1" xref="S6.E6.m1.5.6.2.3.2.cmml">(</mo><mi id="S6.E6.m1.4.4" xref="S6.E6.m1.4.4.cmml">σ</mi><mo stretchy="false" id="S6.E6.m1.5.6.2.3.2.3.2.2" xref="S6.E6.m1.5.6.2.3.2.cmml">)</mo></mrow></mrow></mrow></mrow><mo id="S6.E6.m1.5.6.1" xref="S6.E6.m1.5.6.1.cmml">∴</mo><mrow id="S6.E6.m1.5.6.3" xref="S6.E6.m1.5.6.3.cmml"><mrow id="S6.E6.m1.5.6.3.2" xref="S6.E6.m1.5.6.3.2.cmml"><mi id="S6.E6.m1.5.6.3.2.2" xref="S6.E6.m1.5.6.3.2.2.cmml">f</mi><mo lspace="0em" rspace="0em" id="S6.E6.m1.5.6.3.2.1" xref="S6.E6.m1.5.6.3.2.1.cmml">​</mo><mrow id="S6.E6.m1.5.6.3.2.3.2" xref="S6.E6.m1.5.6.3.2.cmml"><mo stretchy="false" id="S6.E6.m1.5.6.3.2.3.2.1" xref="S6.E6.m1.5.6.3.2.cmml">(</mo><mi id="S6.E6.m1.5.5" xref="S6.E6.m1.5.5.cmml">σ</mi><mo stretchy="false" id="S6.E6.m1.5.6.3.2.3.2.2" xref="S6.E6.m1.5.6.3.2.cmml">)</mo></mrow></mrow><mo id="S6.E6.m1.5.6.3.1" xref="S6.E6.m1.5.6.3.1.cmml">=</mo><mrow id="S6.E6.m1.5.6.3.3.2" xref="S6.E6.m1.5.6.3.3.1.cmml"><mo id="S6.E6.m1.5.6.3.3.2.1" xref="S6.E6.m1.5.6.3.3.1.1.cmml">{</mo><mtable columnspacing="5pt" displaystyle="true" rowspacing="0pt" id="S6.E6.m1.2.2" xref="S6.E6.m1.2.2.cmml"><mtr id="S6.E6.m1.2.2a" xref="S6.E6.m1.2.2.cmml"><mtd id="S6.E6.m1.2.2b" xref="S6.E6.m1.2.2.cmml"><mrow id="S6.E6.m1.1.1.1.1.1.1" xref="S6.E6.m1.1.1.1.1.1.1.1.cmml"><msub id="S6.E6.m1.1.1.1.1.1.1.1" xref="S6.E6.m1.1.1.1.1.1.1.1.cmml"><mi id="S6.E6.m1.1.1.1.1.1.1.1.2" xref="S6.E6.m1.1.1.1.1.1.1.1.2.cmml">s</mi><mrow id="S6.E6.m1.1.1.1.1.1.1.1.3" xref="S6.E6.m1.1.1.1.1.1.1.1.3.cmml"><mi id="S6.E6.m1.1.1.1.1.1.1.1.3.2" xref="S6.E6.m1.1.1.1.1.1.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.E6.m1.1.1.1.1.1.1.1.3.1" xref="S6.E6.m1.1.1.1.1.1.1.1.3.1.cmml">​</mo><mi id="S6.E6.m1.1.1.1.1.1.1.1.3.3" xref="S6.E6.m1.1.1.1.1.1.1.1.3.3.cmml">j</mi></mrow></msub><mo id="S6.E6.m1.1.1.1.1.1.1.2" xref="S6.E6.m1.1.1.1.1.1.1.1.cmml">,</mo></mrow></mtd><mtd id="S6.E6.m1.2.2c" xref="S6.E6.m1.2.2.cmml"><mrow id="S6.E6.m1.1.1.1.2.1" xref="S6.E6.m1.1.1.1.2.1.cmml"><mtext id="S6.E6.m1.1.1.1.2.1.2" xref="S6.E6.m1.1.1.1.2.1.2a.cmml">if </mtext><mo lspace="0em" rspace="0em" id="S6.E6.m1.1.1.1.2.1.1" xref="S6.E6.m1.1.1.1.2.1.1.cmml">​</mo><mi id="S6.E6.m1.1.1.1.2.1.3" xref="S6.E6.m1.1.1.1.2.1.3.cmml">σ</mi><mo lspace="0em" rspace="0em" id="S6.E6.m1.1.1.1.2.1.1a" xref="S6.E6.m1.1.1.1.2.1.1.cmml">​</mo><mrow id="S6.E6.m1.1.1.1.2.1.4" xref="S6.E6.m1.1.1.1.2.1.4c.cmml"><mtext id="S6.E6.m1.1.1.1.2.1.4a" xref="S6.E6.m1.1.1.1.2.1.4c.cmml"> is the amino-acid</mtext><mtext id="S6.E6.m1.1.1.1.2.1.4b" xref="S6.E6.m1.1.1.1.2.1.4c.cmml"> in position ij</mtext></mrow></mrow></mtd></mtr><mtr id="S6.E6.m1.2.2d" xref="S6.E6.m1.2.2.cmml"><mtd id="S6.E6.m1.2.2e" xref="S6.E6.m1.2.2.cmml"></mtd><mtd id="S6.E6.m1.2.2f" xref="S6.E6.m1.2.2.cmml"></mtd></mtr><mtr id="S6.E6.m1.2.2g" xref="S6.E6.m1.2.2.cmml"><mtd id="S6.E6.m1.2.2h" xref="S6.E6.m1.2.2.cmml"><mrow id="S6.E6.m1.2.2.2.1.1.3" xref="S6.E6.m1.2.2.cmml"><mn id="S6.E6.m1.2.2.2.1.1.1" xref="S6.E6.m1.2.2.2.1.1.1.cmml">0</mn><mo id="S6.E6.m1.2.2.2.1.1.3.1" xref="S6.E6.m1.2.2.cmml">,</mo></mrow></mtd><mtd id="S6.E6.m1.2.2i" xref="S6.E6.m1.2.2.cmml"><mtext id="S6.E6.m1.2.2.2.2.1" xref="S6.E6.m1.2.2.2.2.1a.cmml">otherwise</mtext></mtd></mtr></mtable><mi id="S6.E6.m1.5.6.3.3.2.2" xref="S6.E6.m1.5.6.3.3.1.1.cmml"></mi></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.E6.m1.5b"><apply id="S6.E6.m1.5.6.cmml" xref="S6.E6.m1.5.6"><csymbol cd="latexml" id="S6.E6.m1.5.6.1.cmml" xref="S6.E6.m1.5.6.1">therefore</csymbol><apply id="S6.E6.m1.5.6.2.cmml" xref="S6.E6.m1.5.6.2"><eq id="S6.E6.m1.5.6.2.1.cmml" xref="S6.E6.m1.5.6.2.1"></eq><apply id="S6.E6.m1.5.6.2.2.cmml" xref="S6.E6.m1.5.6.2.2"><times id="S6.E6.m1.5.6.2.2.1.cmml" xref="S6.E6.m1.5.6.2.2.1"></times><apply id="S6.E6.m1.5.6.2.2.2.cmml" xref="S6.E6.m1.5.6.2.2.2"><csymbol cd="ambiguous" id="S6.E6.m1.5.6.2.2.2.1.cmml" xref="S6.E6.m1.5.6.2.2.2">subscript</csymbol><ci id="S6.E6.m1.5.6.2.2.2.2.cmml" xref="S6.E6.m1.5.6.2.2.2.2">𝑐</ci><ci id="S6.E6.m1.5.6.2.2.2.3.cmml" xref="S6.E6.m1.5.6.2.2.2.3">𝑗</ci></apply><ci id="S6.E6.m1.3.3.cmml" xref="S6.E6.m1.3.3">𝜎</ci></apply><apply id="S6.E6.m1.5.6.2.3.cmml" xref="S6.E6.m1.5.6.2.3"><apply id="S6.E6.m1.5.6.2.3.1.cmml" xref="S6.E6.m1.5.6.2.3.1"><csymbol cd="ambiguous" id="S6.E6.m1.5.6.2.3.1.1.cmml" xref="S6.E6.m1.5.6.2.3.1">superscript</csymbol><apply id="S6.E6.m1.5.6.2.3.1.2.cmml" xref="S6.E6.m1.5.6.2.3.1"><csymbol cd="ambiguous" id="S6.E6.m1.5.6.2.3.1.2.1.cmml" xref="S6.E6.m1.5.6.2.3.1">subscript</csymbol><sum id="S6.E6.m1.5.6.2.3.1.2.2.cmml" xref="S6.E6.m1.5.6.2.3.1.2.2"></sum><ci id="S6.E6.m1.5.6.2.3.1.2.3.cmml" xref="S6.E6.m1.5.6.2.3.1.2.3">𝑖</ci></apply><ci id="S6.E6.m1.5.6.2.3.1.3.cmml" xref="S6.E6.m1.5.6.2.3.1.3">𝑁</ci></apply><apply id="S6.E6.m1.5.6.2.3.2.cmml" xref="S6.E6.m1.5.6.2.3.2"><times id="S6.E6.m1.5.6.2.3.2.1.cmml" xref="S6.E6.m1.5.6.2.3.2.1"></times><ci id="S6.E6.m1.5.6.2.3.2.2.cmml" xref="S6.E6.m1.5.6.2.3.2.2">𝑓</ci><ci id="S6.E6.m1.4.4.cmml" xref="S6.E6.m1.4.4">𝜎</ci></apply></apply></apply><apply id="S6.E6.m1.5.6.3.cmml" xref="S6.E6.m1.5.6.3"><eq id="S6.E6.m1.5.6.3.1.cmml" xref="S6.E6.m1.5.6.3.1"></eq><apply id="S6.E6.m1.5.6.3.2.cmml" xref="S6.E6.m1.5.6.3.2"><times id="S6.E6.m1.5.6.3.2.1.cmml" xref="S6.E6.m1.5.6.3.2.1"></times><ci id="S6.E6.m1.5.6.3.2.2.cmml" xref="S6.E6.m1.5.6.3.2.2">𝑓</ci><ci id="S6.E6.m1.5.5.cmml" xref="S6.E6.m1.5.5">𝜎</ci></apply><apply id="S6.E6.m1.5.6.3.3.1.cmml" xref="S6.E6.m1.5.6.3.3.2"><csymbol cd="latexml" id="S6.E6.m1.5.6.3.3.1.1.cmml" xref="S6.E6.m1.5.6.3.3.2.1">cases</csymbol><matrix id="S6.E6.m1.2.2.cmml" xref="S6.E6.m1.2.2"><matrixrow id="S6.E6.m1.2.2a.cmml" xref="S6.E6.m1.2.2"><apply id="S6.E6.m1.1.1.1.1.1.1.1.cmml" xref="S6.E6.m1.1.1.1.1.1.1"><csymbol cd="ambiguous" id="S6.E6.m1.1.1.1.1.1.1.1.1.cmml" xref="S6.E6.m1.1.1.1.1.1.1">subscript</csymbol><ci id="S6.E6.m1.1.1.1.1.1.1.1.2.cmml" xref="S6.E6.m1.1.1.1.1.1.1.1.2">𝑠</ci><apply id="S6.E6.m1.1.1.1.1.1.1.1.3.cmml" xref="S6.E6.m1.1.1.1.1.1.1.1.3"><times id="S6.E6.m1.1.1.1.1.1.1.1.3.1.cmml" xref="S6.E6.m1.1.1.1.1.1.1.1.3.1"></times><ci id="S6.E6.m1.1.1.1.1.1.1.1.3.2.cmml" xref="S6.E6.m1.1.1.1.1.1.1.1.3.2">𝑖</ci><ci id="S6.E6.m1.1.1.1.1.1.1.1.3.3.cmml" xref="S6.E6.m1.1.1.1.1.1.1.1.3.3">𝑗</ci></apply></apply><apply id="S6.E6.m1.1.1.1.2.1.cmml" xref="S6.E6.m1.1.1.1.2.1"><times id="S6.E6.m1.1.1.1.2.1.1.cmml" xref="S6.E6.m1.1.1.1.2.1.1"></times><ci id="S6.E6.m1.1.1.1.2.1.2a.cmml" xref="S6.E6.m1.1.1.1.2.1.2"><mtext id="S6.E6.m1.1.1.1.2.1.2.cmml" xref="S6.E6.m1.1.1.1.2.1.2">if </mtext></ci><ci id="S6.E6.m1.1.1.1.2.1.3.cmml" xref="S6.E6.m1.1.1.1.2.1.3">𝜎</ci><ci id="S6.E6.m1.1.1.1.2.1.4c.cmml" xref="S6.E6.m1.1.1.1.2.1.4"><mrow id="S6.E6.m1.1.1.1.2.1.4.cmml" xref="S6.E6.m1.1.1.1.2.1.4"><mtext id="S6.E6.m1.1.1.1.2.1.4a.cmml" xref="S6.E6.m1.1.1.1.2.1.4"> is the amino-acid</mtext><mtext id="S6.E6.m1.1.1.1.2.1.4b.cmml" xref="S6.E6.m1.1.1.1.2.1.4"> in position ij</mtext></mrow></ci></apply></matrixrow><matrixrow id="S6.E6.m1.2.2b.cmml" xref="S6.E6.m1.2.2"><cerror id="S6.E6.m1.2.2c.cmml" xref="S6.E6.m1.2.2"><csymbol cd="ambiguous" id="S6.E6.m1.2.2d.cmml" xref="S6.E6.m1.2.2">missing-subexpression</csymbol></cerror><cerror id="S6.E6.m1.2.2e.cmml" xref="S6.E6.m1.2.2"><csymbol cd="ambiguous" id="S6.E6.m1.2.2f.cmml" xref="S6.E6.m1.2.2">missing-subexpression</csymbol></cerror></matrixrow><matrixrow id="S6.E6.m1.2.2g.cmml" xref="S6.E6.m1.2.2"><cn type="integer" id="S6.E6.m1.2.2.2.1.1.1.cmml" xref="S6.E6.m1.2.2.2.1.1.1">0</cn><ci id="S6.E6.m1.2.2.2.2.1a.cmml" xref="S6.E6.m1.2.2.2.2.1"><mtext id="S6.E6.m1.2.2.2.2.1.cmml" xref="S6.E6.m1.2.2.2.2.1">otherwise</mtext></ci></matrixrow></matrix></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.E6.m1.5c">\\
c_{j}(\sigma)=\sum_{i}^{N}f(\sigma)\therefore f(\sigma)=\left\{\begin{array}[]{cc}s_{ij},&amp;\textrm{if }\sigma\textrm{ is the amino-acid}\textrm{ in position ij}\\
\\
0,&amp;\textrm{otherwise}\end{array}\right.</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(6)</span></td>
</tr></tbody>
</table>
<br class="ltx_break">
</div>
<div id="S6.SS2.p10" class="ltx_para">
<p id="S6.SS2.p10.12" class="ltx_p">In the same way, we apply the equations <a href="#S6.E7" title="In 6.2 Sequence Weighting ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a> to determine <math id="S6.SS2.p10.1.m1.1" class="ltx_Math" alttext="c_{kl}" display="inline"><semantics id="S6.SS2.p10.1.m1.1a"><msub id="S6.SS2.p10.1.m1.1.1" xref="S6.SS2.p10.1.m1.1.1.cmml"><mi id="S6.SS2.p10.1.m1.1.1.2" xref="S6.SS2.p10.1.m1.1.1.2.cmml">c</mi><mrow id="S6.SS2.p10.1.m1.1.1.3" xref="S6.SS2.p10.1.m1.1.1.3.cmml"><mi id="S6.SS2.p10.1.m1.1.1.3.2" xref="S6.SS2.p10.1.m1.1.1.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p10.1.m1.1.1.3.1" xref="S6.SS2.p10.1.m1.1.1.3.1.cmml">​</mo><mi id="S6.SS2.p10.1.m1.1.1.3.3" xref="S6.SS2.p10.1.m1.1.1.3.3.cmml">l</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.1.m1.1b"><apply id="S6.SS2.p10.1.m1.1.1.cmml" xref="S6.SS2.p10.1.m1.1.1"><csymbol cd="ambiguous" id="S6.SS2.p10.1.m1.1.1.1.cmml" xref="S6.SS2.p10.1.m1.1.1">subscript</csymbol><ci id="S6.SS2.p10.1.m1.1.1.2.cmml" xref="S6.SS2.p10.1.m1.1.1.2">𝑐</ci><apply id="S6.SS2.p10.1.m1.1.1.3.cmml" xref="S6.SS2.p10.1.m1.1.1.3"><times id="S6.SS2.p10.1.m1.1.1.3.1.cmml" xref="S6.SS2.p10.1.m1.1.1.3.1"></times><ci id="S6.SS2.p10.1.m1.1.1.3.2.cmml" xref="S6.SS2.p10.1.m1.1.1.3.2">𝑘</ci><ci id="S6.SS2.p10.1.m1.1.1.3.3.cmml" xref="S6.SS2.p10.1.m1.1.1.3.3">𝑙</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.1.m1.1c">c_{kl}</annotation></semantics></math> shown in <a href="#S6.E2" title="In 6.1 Profile HMMs ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>. If the <math id="S6.SS2.p10.2.m2.1" class="ltx_Math" alttext="k" display="inline"><semantics id="S6.SS2.p10.2.m2.1a"><mi id="S6.SS2.p10.2.m2.1.1" xref="S6.SS2.p10.2.m2.1.1.cmml">k</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.2.m2.1b"><ci id="S6.SS2.p10.2.m2.1.1.cmml" xref="S6.SS2.p10.2.m2.1.1">𝑘</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.2.m2.1c">k</annotation></semantics></math> and <math id="S6.SS2.p10.3.m3.1" class="ltx_Math" alttext="l" display="inline"><semantics id="S6.SS2.p10.3.m3.1a"><mi id="S6.SS2.p10.3.m3.1.1" xref="S6.SS2.p10.3.m3.1.1.cmml">l</mi><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.3.m3.1b"><ci id="S6.SS2.p10.3.m3.1.1.cmml" xref="S6.SS2.p10.3.m3.1.1">𝑙</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.3.m3.1c">l</annotation></semantics></math> states are either M or I states, <math id="S6.SS2.p10.4.m4.1" class="ltx_Math" alttext="c_{kl}" display="inline"><semantics id="S6.SS2.p10.4.m4.1a"><msub id="S6.SS2.p10.4.m4.1.1" xref="S6.SS2.p10.4.m4.1.1.cmml"><mi id="S6.SS2.p10.4.m4.1.1.2" xref="S6.SS2.p10.4.m4.1.1.2.cmml">c</mi><mrow id="S6.SS2.p10.4.m4.1.1.3" xref="S6.SS2.p10.4.m4.1.1.3.cmml"><mi id="S6.SS2.p10.4.m4.1.1.3.2" xref="S6.SS2.p10.4.m4.1.1.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p10.4.m4.1.1.3.1" xref="S6.SS2.p10.4.m4.1.1.3.1.cmml">​</mo><mi id="S6.SS2.p10.4.m4.1.1.3.3" xref="S6.SS2.p10.4.m4.1.1.3.3.cmml">l</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.4.m4.1b"><apply id="S6.SS2.p10.4.m4.1.1.cmml" xref="S6.SS2.p10.4.m4.1.1"><csymbol cd="ambiguous" id="S6.SS2.p10.4.m4.1.1.1.cmml" xref="S6.SS2.p10.4.m4.1.1">subscript</csymbol><ci id="S6.SS2.p10.4.m4.1.1.2.cmml" xref="S6.SS2.p10.4.m4.1.1.2">𝑐</ci><apply id="S6.SS2.p10.4.m4.1.1.3.cmml" xref="S6.SS2.p10.4.m4.1.1.3"><times id="S6.SS2.p10.4.m4.1.1.3.1.cmml" xref="S6.SS2.p10.4.m4.1.1.3.1"></times><ci id="S6.SS2.p10.4.m4.1.1.3.2.cmml" xref="S6.SS2.p10.4.m4.1.1.3.2">𝑘</ci><ci id="S6.SS2.p10.4.m4.1.1.3.3.cmml" xref="S6.SS2.p10.4.m4.1.1.3.3">𝑙</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.4.m4.1c">c_{kl}</annotation></semantics></math> can be calculated through the arithmetic mean of <math id="S6.SS2.p10.5.m5.1" class="ltx_Math" alttext="m_{ik}" display="inline"><semantics id="S6.SS2.p10.5.m5.1a"><msub id="S6.SS2.p10.5.m5.1.1" xref="S6.SS2.p10.5.m5.1.1.cmml"><mi id="S6.SS2.p10.5.m5.1.1.2" xref="S6.SS2.p10.5.m5.1.1.2.cmml">m</mi><mrow id="S6.SS2.p10.5.m5.1.1.3" xref="S6.SS2.p10.5.m5.1.1.3.cmml"><mi id="S6.SS2.p10.5.m5.1.1.3.2" xref="S6.SS2.p10.5.m5.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p10.5.m5.1.1.3.1" xref="S6.SS2.p10.5.m5.1.1.3.1.cmml">​</mo><mi id="S6.SS2.p10.5.m5.1.1.3.3" xref="S6.SS2.p10.5.m5.1.1.3.3.cmml">k</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.5.m5.1b"><apply id="S6.SS2.p10.5.m5.1.1.cmml" xref="S6.SS2.p10.5.m5.1.1"><csymbol cd="ambiguous" id="S6.SS2.p10.5.m5.1.1.1.cmml" xref="S6.SS2.p10.5.m5.1.1">subscript</csymbol><ci id="S6.SS2.p10.5.m5.1.1.2.cmml" xref="S6.SS2.p10.5.m5.1.1.2">𝑚</ci><apply id="S6.SS2.p10.5.m5.1.1.3.cmml" xref="S6.SS2.p10.5.m5.1.1.3"><times id="S6.SS2.p10.5.m5.1.1.3.1.cmml" xref="S6.SS2.p10.5.m5.1.1.3.1"></times><ci id="S6.SS2.p10.5.m5.1.1.3.2.cmml" xref="S6.SS2.p10.5.m5.1.1.3.2">𝑖</ci><ci id="S6.SS2.p10.5.m5.1.1.3.3.cmml" xref="S6.SS2.p10.5.m5.1.1.3.3">𝑘</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.5.m5.1c">m_{ik}</annotation></semantics></math> and <math id="S6.SS2.p10.6.m6.1" class="ltx_Math" alttext="m_{il}" display="inline"><semantics id="S6.SS2.p10.6.m6.1a"><msub id="S6.SS2.p10.6.m6.1.1" xref="S6.SS2.p10.6.m6.1.1.cmml"><mi id="S6.SS2.p10.6.m6.1.1.2" xref="S6.SS2.p10.6.m6.1.1.2.cmml">m</mi><mrow id="S6.SS2.p10.6.m6.1.1.3" xref="S6.SS2.p10.6.m6.1.1.3.cmml"><mi id="S6.SS2.p10.6.m6.1.1.3.2" xref="S6.SS2.p10.6.m6.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p10.6.m6.1.1.3.1" xref="S6.SS2.p10.6.m6.1.1.3.1.cmml">​</mo><mi id="S6.SS2.p10.6.m6.1.1.3.3" xref="S6.SS2.p10.6.m6.1.1.3.3.cmml">l</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.6.m6.1b"><apply id="S6.SS2.p10.6.m6.1.1.cmml" xref="S6.SS2.p10.6.m6.1.1"><csymbol cd="ambiguous" id="S6.SS2.p10.6.m6.1.1.1.cmml" xref="S6.SS2.p10.6.m6.1.1">subscript</csymbol><ci id="S6.SS2.p10.6.m6.1.1.2.cmml" xref="S6.SS2.p10.6.m6.1.1.2">𝑚</ci><apply id="S6.SS2.p10.6.m6.1.1.3.cmml" xref="S6.SS2.p10.6.m6.1.1.3"><times id="S6.SS2.p10.6.m6.1.1.3.1.cmml" xref="S6.SS2.p10.6.m6.1.1.3.1"></times><ci id="S6.SS2.p10.6.m6.1.1.3.2.cmml" xref="S6.SS2.p10.6.m6.1.1.3.2">𝑖</ci><ci id="S6.SS2.p10.6.m6.1.1.3.3.cmml" xref="S6.SS2.p10.6.m6.1.1.3.3">𝑙</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.6.m6.1c">m_{il}</annotation></semantics></math>. If at least one state is a D state, <math id="S6.SS2.p10.7.m7.1" class="ltx_Math" alttext="c_{kl}" display="inline"><semantics id="S6.SS2.p10.7.m7.1a"><msub id="S6.SS2.p10.7.m7.1.1" xref="S6.SS2.p10.7.m7.1.1.cmml"><mi id="S6.SS2.p10.7.m7.1.1.2" xref="S6.SS2.p10.7.m7.1.1.2.cmml">c</mi><mrow id="S6.SS2.p10.7.m7.1.1.3" xref="S6.SS2.p10.7.m7.1.1.3.cmml"><mi id="S6.SS2.p10.7.m7.1.1.3.2" xref="S6.SS2.p10.7.m7.1.1.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p10.7.m7.1.1.3.1" xref="S6.SS2.p10.7.m7.1.1.3.1.cmml">​</mo><mi id="S6.SS2.p10.7.m7.1.1.3.3" xref="S6.SS2.p10.7.m7.1.1.3.3.cmml">l</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.7.m7.1b"><apply id="S6.SS2.p10.7.m7.1.1.cmml" xref="S6.SS2.p10.7.m7.1.1"><csymbol cd="ambiguous" id="S6.SS2.p10.7.m7.1.1.1.cmml" xref="S6.SS2.p10.7.m7.1.1">subscript</csymbol><ci id="S6.SS2.p10.7.m7.1.1.2.cmml" xref="S6.SS2.p10.7.m7.1.1.2">𝑐</ci><apply id="S6.SS2.p10.7.m7.1.1.3.cmml" xref="S6.SS2.p10.7.m7.1.1.3"><times id="S6.SS2.p10.7.m7.1.1.3.1.cmml" xref="S6.SS2.p10.7.m7.1.1.3.1"></times><ci id="S6.SS2.p10.7.m7.1.1.3.2.cmml" xref="S6.SS2.p10.7.m7.1.1.3.2">𝑘</ci><ci id="S6.SS2.p10.7.m7.1.1.3.3.cmml" xref="S6.SS2.p10.7.m7.1.1.3.3">𝑙</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.7.m7.1c">c_{kl}</annotation></semantics></math> is either <math id="S6.SS2.p10.8.m8.1" class="ltx_Math" alttext="m_{ik}" display="inline"><semantics id="S6.SS2.p10.8.m8.1a"><msub id="S6.SS2.p10.8.m8.1.1" xref="S6.SS2.p10.8.m8.1.1.cmml"><mi id="S6.SS2.p10.8.m8.1.1.2" xref="S6.SS2.p10.8.m8.1.1.2.cmml">m</mi><mrow id="S6.SS2.p10.8.m8.1.1.3" xref="S6.SS2.p10.8.m8.1.1.3.cmml"><mi id="S6.SS2.p10.8.m8.1.1.3.2" xref="S6.SS2.p10.8.m8.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p10.8.m8.1.1.3.1" xref="S6.SS2.p10.8.m8.1.1.3.1.cmml">​</mo><mi id="S6.SS2.p10.8.m8.1.1.3.3" xref="S6.SS2.p10.8.m8.1.1.3.3.cmml">k</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.8.m8.1b"><apply id="S6.SS2.p10.8.m8.1.1.cmml" xref="S6.SS2.p10.8.m8.1.1"><csymbol cd="ambiguous" id="S6.SS2.p10.8.m8.1.1.1.cmml" xref="S6.SS2.p10.8.m8.1.1">subscript</csymbol><ci id="S6.SS2.p10.8.m8.1.1.2.cmml" xref="S6.SS2.p10.8.m8.1.1.2">𝑚</ci><apply id="S6.SS2.p10.8.m8.1.1.3.cmml" xref="S6.SS2.p10.8.m8.1.1.3"><times id="S6.SS2.p10.8.m8.1.1.3.1.cmml" xref="S6.SS2.p10.8.m8.1.1.3.1"></times><ci id="S6.SS2.p10.8.m8.1.1.3.2.cmml" xref="S6.SS2.p10.8.m8.1.1.3.2">𝑖</ci><ci id="S6.SS2.p10.8.m8.1.1.3.3.cmml" xref="S6.SS2.p10.8.m8.1.1.3.3">𝑘</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.8.m8.1c">m_{ik}</annotation></semantics></math>, if <math id="S6.SS2.p10.9.m9.1" class="ltx_Math" alttext="l\in\{D\}" display="inline"><semantics id="S6.SS2.p10.9.m9.1a"><mrow id="S6.SS2.p10.9.m9.1.2" xref="S6.SS2.p10.9.m9.1.2.cmml"><mi id="S6.SS2.p10.9.m9.1.2.2" xref="S6.SS2.p10.9.m9.1.2.2.cmml">l</mi><mo id="S6.SS2.p10.9.m9.1.2.1" xref="S6.SS2.p10.9.m9.1.2.1.cmml">∈</mo><mrow id="S6.SS2.p10.9.m9.1.2.3.2" xref="S6.SS2.p10.9.m9.1.2.3.1.cmml"><mo stretchy="false" id="S6.SS2.p10.9.m9.1.2.3.2.1" xref="S6.SS2.p10.9.m9.1.2.3.1.cmml">{</mo><mi id="S6.SS2.p10.9.m9.1.1" xref="S6.SS2.p10.9.m9.1.1.cmml">D</mi><mo stretchy="false" id="S6.SS2.p10.9.m9.1.2.3.2.2" xref="S6.SS2.p10.9.m9.1.2.3.1.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.9.m9.1b"><apply id="S6.SS2.p10.9.m9.1.2.cmml" xref="S6.SS2.p10.9.m9.1.2"><in id="S6.SS2.p10.9.m9.1.2.1.cmml" xref="S6.SS2.p10.9.m9.1.2.1"></in><ci id="S6.SS2.p10.9.m9.1.2.2.cmml" xref="S6.SS2.p10.9.m9.1.2.2">𝑙</ci><set id="S6.SS2.p10.9.m9.1.2.3.1.cmml" xref="S6.SS2.p10.9.m9.1.2.3.2"><ci id="S6.SS2.p10.9.m9.1.1.cmml" xref="S6.SS2.p10.9.m9.1.1">𝐷</ci></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.9.m9.1c">l\in\{D\}</annotation></semantics></math>, or <math id="S6.SS2.p10.10.m10.1" class="ltx_Math" alttext="m_{il}" display="inline"><semantics id="S6.SS2.p10.10.m10.1a"><msub id="S6.SS2.p10.10.m10.1.1" xref="S6.SS2.p10.10.m10.1.1.cmml"><mi id="S6.SS2.p10.10.m10.1.1.2" xref="S6.SS2.p10.10.m10.1.1.2.cmml">m</mi><mrow id="S6.SS2.p10.10.m10.1.1.3" xref="S6.SS2.p10.10.m10.1.1.3.cmml"><mi id="S6.SS2.p10.10.m10.1.1.3.2" xref="S6.SS2.p10.10.m10.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p10.10.m10.1.1.3.1" xref="S6.SS2.p10.10.m10.1.1.3.1.cmml">​</mo><mi id="S6.SS2.p10.10.m10.1.1.3.3" xref="S6.SS2.p10.10.m10.1.1.3.3.cmml">l</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.10.m10.1b"><apply id="S6.SS2.p10.10.m10.1.1.cmml" xref="S6.SS2.p10.10.m10.1.1"><csymbol cd="ambiguous" id="S6.SS2.p10.10.m10.1.1.1.cmml" xref="S6.SS2.p10.10.m10.1.1">subscript</csymbol><ci id="S6.SS2.p10.10.m10.1.1.2.cmml" xref="S6.SS2.p10.10.m10.1.1.2">𝑚</ci><apply id="S6.SS2.p10.10.m10.1.1.3.cmml" xref="S6.SS2.p10.10.m10.1.1.3"><times id="S6.SS2.p10.10.m10.1.1.3.1.cmml" xref="S6.SS2.p10.10.m10.1.1.3.1"></times><ci id="S6.SS2.p10.10.m10.1.1.3.2.cmml" xref="S6.SS2.p10.10.m10.1.1.3.2">𝑖</ci><ci id="S6.SS2.p10.10.m10.1.1.3.3.cmml" xref="S6.SS2.p10.10.m10.1.1.3.3">𝑙</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.10.m10.1c">m_{il}</annotation></semantics></math>, if <math id="S6.SS2.p10.11.m11.1" class="ltx_Math" alttext="k\in\{D\}" display="inline"><semantics id="S6.SS2.p10.11.m11.1a"><mrow id="S6.SS2.p10.11.m11.1.2" xref="S6.SS2.p10.11.m11.1.2.cmml"><mi id="S6.SS2.p10.11.m11.1.2.2" xref="S6.SS2.p10.11.m11.1.2.2.cmml">k</mi><mo id="S6.SS2.p10.11.m11.1.2.1" xref="S6.SS2.p10.11.m11.1.2.1.cmml">∈</mo><mrow id="S6.SS2.p10.11.m11.1.2.3.2" xref="S6.SS2.p10.11.m11.1.2.3.1.cmml"><mo stretchy="false" id="S6.SS2.p10.11.m11.1.2.3.2.1" xref="S6.SS2.p10.11.m11.1.2.3.1.cmml">{</mo><mi id="S6.SS2.p10.11.m11.1.1" xref="S6.SS2.p10.11.m11.1.1.cmml">D</mi><mo stretchy="false" id="S6.SS2.p10.11.m11.1.2.3.2.2" xref="S6.SS2.p10.11.m11.1.2.3.1.cmml">}</mo></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.11.m11.1b"><apply id="S6.SS2.p10.11.m11.1.2.cmml" xref="S6.SS2.p10.11.m11.1.2"><in id="S6.SS2.p10.11.m11.1.2.1.cmml" xref="S6.SS2.p10.11.m11.1.2.1"></in><ci id="S6.SS2.p10.11.m11.1.2.2.cmml" xref="S6.SS2.p10.11.m11.1.2.2">𝑘</ci><set id="S6.SS2.p10.11.m11.1.2.3.1.cmml" xref="S6.SS2.p10.11.m11.1.2.3.2"><ci id="S6.SS2.p10.11.m11.1.1.cmml" xref="S6.SS2.p10.11.m11.1.1">𝐷</ci></set></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.11.m11.1c">k\in\{D\}</annotation></semantics></math>. Last, if both are D states, <math id="S6.SS2.p10.12.m12.1" class="ltx_Math" alttext="c_{kl}" display="inline"><semantics id="S6.SS2.p10.12.m12.1a"><msub id="S6.SS2.p10.12.m12.1.1" xref="S6.SS2.p10.12.m12.1.1.cmml"><mi id="S6.SS2.p10.12.m12.1.1.2" xref="S6.SS2.p10.12.m12.1.1.2.cmml">c</mi><mrow id="S6.SS2.p10.12.m12.1.1.3" xref="S6.SS2.p10.12.m12.1.1.3.cmml"><mi id="S6.SS2.p10.12.m12.1.1.3.2" xref="S6.SS2.p10.12.m12.1.1.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.SS2.p10.12.m12.1.1.3.1" xref="S6.SS2.p10.12.m12.1.1.3.1.cmml">​</mo><mi id="S6.SS2.p10.12.m12.1.1.3.3" xref="S6.SS2.p10.12.m12.1.1.3.3.cmml">l</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS2.p10.12.m12.1b"><apply id="S6.SS2.p10.12.m12.1.1.cmml" xref="S6.SS2.p10.12.m12.1.1"><csymbol cd="ambiguous" id="S6.SS2.p10.12.m12.1.1.1.cmml" xref="S6.SS2.p10.12.m12.1.1">subscript</csymbol><ci id="S6.SS2.p10.12.m12.1.1.2.cmml" xref="S6.SS2.p10.12.m12.1.1.2">𝑐</ci><apply id="S6.SS2.p10.12.m12.1.1.3.cmml" xref="S6.SS2.p10.12.m12.1.1.3"><times id="S6.SS2.p10.12.m12.1.1.3.1.cmml" xref="S6.SS2.p10.12.m12.1.1.3.1"></times><ci id="S6.SS2.p10.12.m12.1.1.3.2.cmml" xref="S6.SS2.p10.12.m12.1.1.3.2">𝑘</ci><ci id="S6.SS2.p10.12.m12.1.1.3.3.cmml" xref="S6.SS2.p10.12.m12.1.1.3.3">𝑙</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS2.p10.12.m12.1c">c_{kl}</annotation></semantics></math> is 1.</p>
</div>
<div id="S6.SS2.p11" class="ltx_para">
<table id="S6.E7" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S6.E7.m1.11" class="ltx_Math" alttext="\\
c_{kl}=\sum_{i}^{N}f_{kl}\therefore\\
f_{kl}=\left\{\begin{array}[]{cc}\dfrac{s_{ik}+s_{il}}{2},&amp;\textrm{se k,l}\in{\{M,I\}}\\
\\
s_{ik},&amp;\textrm{se l}\in{\{D\}}\ e\ \textrm{k}\notin{\{D\}}\\
\\
s_{il},&amp;\textrm{se k}\in{\{D\}}\ e\ \textrm{l}\notin{\{D\}}\\
\\
1,&amp;\textrm{se k,l}\in{\{D\}}\end{array}\right." display="block"><semantics id="S6.E7.m1.11a"><mrow id="S6.E7.m1.11.12" xref="S6.E7.m1.11.12.cmml"><mrow id="S6.E7.m1.11.12.2" xref="S6.E7.m1.11.12.2.cmml"><msub id="S6.E7.m1.11.12.2.2" xref="S6.E7.m1.11.12.2.2.cmml"><mi id="S6.E7.m1.11.12.2.2.2" xref="S6.E7.m1.11.12.2.2.2.cmml">c</mi><mrow id="S6.E7.m1.11.12.2.2.3" xref="S6.E7.m1.11.12.2.2.3.cmml"><mi id="S6.E7.m1.11.12.2.2.3.2" xref="S6.E7.m1.11.12.2.2.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.E7.m1.11.12.2.2.3.1" xref="S6.E7.m1.11.12.2.2.3.1.cmml">​</mo><mi id="S6.E7.m1.11.12.2.2.3.3" xref="S6.E7.m1.11.12.2.2.3.3.cmml">l</mi></mrow></msub><mo rspace="0.111em" id="S6.E7.m1.11.12.2.1" xref="S6.E7.m1.11.12.2.1.cmml">=</mo><mrow id="S6.E7.m1.11.12.2.3" xref="S6.E7.m1.11.12.2.3.cmml"><munderover id="S6.E7.m1.11.12.2.3.1" xref="S6.E7.m1.11.12.2.3.1.cmml"><mo movablelimits="false" id="S6.E7.m1.11.12.2.3.1.2.2" xref="S6.E7.m1.11.12.2.3.1.2.2.cmml">∑</mo><mi id="S6.E7.m1.11.12.2.3.1.2.3" xref="S6.E7.m1.11.12.2.3.1.2.3.cmml">i</mi><mi id="S6.E7.m1.11.12.2.3.1.3" xref="S6.E7.m1.11.12.2.3.1.3.cmml">N</mi></munderover><msub id="S6.E7.m1.11.12.2.3.2" xref="S6.E7.m1.11.12.2.3.2.cmml"><mi id="S6.E7.m1.11.12.2.3.2.2" xref="S6.E7.m1.11.12.2.3.2.2.cmml">f</mi><mrow id="S6.E7.m1.11.12.2.3.2.3" xref="S6.E7.m1.11.12.2.3.2.3.cmml"><mi id="S6.E7.m1.11.12.2.3.2.3.2" xref="S6.E7.m1.11.12.2.3.2.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.E7.m1.11.12.2.3.2.3.1" xref="S6.E7.m1.11.12.2.3.2.3.1.cmml">​</mo><mi id="S6.E7.m1.11.12.2.3.2.3.3" xref="S6.E7.m1.11.12.2.3.2.3.3.cmml">l</mi></mrow></msub></mrow></mrow><mo id="S6.E7.m1.11.12.1" xref="S6.E7.m1.11.12.1.cmml">∴</mo><mrow id="S6.E7.m1.11.12.3" xref="S6.E7.m1.11.12.3.cmml"><msub id="S6.E7.m1.11.12.3.2" xref="S6.E7.m1.11.12.3.2.cmml"><mi id="S6.E7.m1.11.12.3.2.2" xref="S6.E7.m1.11.12.3.2.2.cmml">f</mi><mrow id="S6.E7.m1.11.12.3.2.3" xref="S6.E7.m1.11.12.3.2.3.cmml"><mi id="S6.E7.m1.11.12.3.2.3.2" xref="S6.E7.m1.11.12.3.2.3.2.cmml">k</mi><mo lspace="0em" rspace="0em" id="S6.E7.m1.11.12.3.2.3.1" xref="S6.E7.m1.11.12.3.2.3.1.cmml">​</mo><mi id="S6.E7.m1.11.12.3.2.3.3" xref="S6.E7.m1.11.12.3.2.3.3.cmml">l</mi></mrow></msub><mo id="S6.E7.m1.11.12.3.1" xref="S6.E7.m1.11.12.3.1.cmml">=</mo><mrow id="S6.E7.m1.11.12.3.3.2" xref="S6.E7.m1.11.12.3.3.1.cmml"><mo id="S6.E7.m1.11.12.3.3.2.1" xref="S6.E7.m1.11.12.3.3.1.1.cmml">{</mo><mtable columnspacing="5pt" displaystyle="true" rowspacing="0pt" id="S6.E7.m1.11.11" xref="S6.E7.m1.11.11.cmml"><mtr id="S6.E7.m1.11.11a" xref="S6.E7.m1.11.11.cmml"><mtd id="S6.E7.m1.11.11b" xref="S6.E7.m1.11.11.cmml"><mrow id="S6.E7.m1.1.1.1.1.1.3" xref="S6.E7.m1.1.1.1.1.1.1.cmml"><mfrac id="S6.E7.m1.1.1.1.1.1.1" xref="S6.E7.m1.1.1.1.1.1.1.cmml"><mrow id="S6.E7.m1.1.1.1.1.1.1.2" xref="S6.E7.m1.1.1.1.1.1.1.2.cmml"><msub id="S6.E7.m1.1.1.1.1.1.1.2.2" xref="S6.E7.m1.1.1.1.1.1.1.2.2.cmml"><mi id="S6.E7.m1.1.1.1.1.1.1.2.2.2" xref="S6.E7.m1.1.1.1.1.1.1.2.2.2.cmml">s</mi><mrow id="S6.E7.m1.1.1.1.1.1.1.2.2.3" xref="S6.E7.m1.1.1.1.1.1.1.2.2.3.cmml"><mi id="S6.E7.m1.1.1.1.1.1.1.2.2.3.2" xref="S6.E7.m1.1.1.1.1.1.1.2.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.E7.m1.1.1.1.1.1.1.2.2.3.1" xref="S6.E7.m1.1.1.1.1.1.1.2.2.3.1.cmml">​</mo><mi id="S6.E7.m1.1.1.1.1.1.1.2.2.3.3" xref="S6.E7.m1.1.1.1.1.1.1.2.2.3.3.cmml">k</mi></mrow></msub><mo id="S6.E7.m1.1.1.1.1.1.1.2.1" xref="S6.E7.m1.1.1.1.1.1.1.2.1.cmml">+</mo><msub id="S6.E7.m1.1.1.1.1.1.1.2.3" xref="S6.E7.m1.1.1.1.1.1.1.2.3.cmml"><mi id="S6.E7.m1.1.1.1.1.1.1.2.3.2" xref="S6.E7.m1.1.1.1.1.1.1.2.3.2.cmml">s</mi><mrow id="S6.E7.m1.1.1.1.1.1.1.2.3.3" xref="S6.E7.m1.1.1.1.1.1.1.2.3.3.cmml"><mi id="S6.E7.m1.1.1.1.1.1.1.2.3.3.2" xref="S6.E7.m1.1.1.1.1.1.1.2.3.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.E7.m1.1.1.1.1.1.1.2.3.3.1" xref="S6.E7.m1.1.1.1.1.1.1.2.3.3.1.cmml">​</mo><mi id="S6.E7.m1.1.1.1.1.1.1.2.3.3.3" xref="S6.E7.m1.1.1.1.1.1.1.2.3.3.3.cmml">l</mi></mrow></msub></mrow><mn id="S6.E7.m1.1.1.1.1.1.1.3" xref="S6.E7.m1.1.1.1.1.1.1.3.cmml">2</mn></mfrac><mo id="S6.E7.m1.1.1.1.1.1.3.1" xref="S6.E7.m1.1.1.1.1.1.1.cmml">,</mo></mrow></mtd><mtd id="S6.E7.m1.11.11c" xref="S6.E7.m1.11.11.cmml"><mrow id="S6.E7.m1.3.3.3.3.2" xref="S6.E7.m1.3.3.3.3.2.cmml"><mtext id="S6.E7.m1.3.3.3.3.2.4" xref="S6.E7.m1.3.3.3.3.2.4a.cmml">se k,l</mtext><mo id="S6.E7.m1.3.3.3.3.2.3" xref="S6.E7.m1.3.3.3.3.2.3.cmml">∈</mo><mrow id="S6.E7.m1.3.3.3.3.2.5.2" xref="S6.E7.m1.3.3.3.3.2.5.1.cmml"><mo stretchy="false" id="S6.E7.m1.3.3.3.3.2.5.2.1" xref="S6.E7.m1.3.3.3.3.2.5.1.cmml">{</mo><mi id="S6.E7.m1.2.2.2.2.1.1" xref="S6.E7.m1.2.2.2.2.1.1.cmml">M</mi><mo id="S6.E7.m1.3.3.3.3.2.5.2.2" xref="S6.E7.m1.3.3.3.3.2.5.1.cmml">,</mo><mi id="S6.E7.m1.3.3.3.3.2.2" xref="S6.E7.m1.3.3.3.3.2.2.cmml">I</mi><mo stretchy="false" id="S6.E7.m1.3.3.3.3.2.5.2.3" xref="S6.E7.m1.3.3.3.3.2.5.1.cmml">}</mo></mrow></mrow></mtd></mtr><mtr id="S6.E7.m1.11.11d" xref="S6.E7.m1.11.11.cmml"><mtd id="S6.E7.m1.11.11e" xref="S6.E7.m1.11.11.cmml"></mtd><mtd id="S6.E7.m1.11.11f" xref="S6.E7.m1.11.11.cmml"></mtd></mtr><mtr id="S6.E7.m1.11.11g" xref="S6.E7.m1.11.11.cmml"><mtd id="S6.E7.m1.11.11h" xref="S6.E7.m1.11.11.cmml"><mrow id="S6.E7.m1.4.4.4.1.1.1" xref="S6.E7.m1.4.4.4.1.1.1.1.cmml"><msub id="S6.E7.m1.4.4.4.1.1.1.1" xref="S6.E7.m1.4.4.4.1.1.1.1.cmml"><mi id="S6.E7.m1.4.4.4.1.1.1.1.2" xref="S6.E7.m1.4.4.4.1.1.1.1.2.cmml">s</mi><mrow id="S6.E7.m1.4.4.4.1.1.1.1.3" xref="S6.E7.m1.4.4.4.1.1.1.1.3.cmml"><mi id="S6.E7.m1.4.4.4.1.1.1.1.3.2" xref="S6.E7.m1.4.4.4.1.1.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.E7.m1.4.4.4.1.1.1.1.3.1" xref="S6.E7.m1.4.4.4.1.1.1.1.3.1.cmml">​</mo><mi id="S6.E7.m1.4.4.4.1.1.1.1.3.3" xref="S6.E7.m1.4.4.4.1.1.1.1.3.3.cmml">k</mi></mrow></msub><mo id="S6.E7.m1.4.4.4.1.1.1.2" xref="S6.E7.m1.4.4.4.1.1.1.1.cmml">,</mo></mrow></mtd><mtd id="S6.E7.m1.11.11i" xref="S6.E7.m1.11.11.cmml"><mrow id="S6.E7.m1.6.6.6.3.2" xref="S6.E7.m1.6.6.6.3.2.cmml"><mtext id="S6.E7.m1.6.6.6.3.2.4" xref="S6.E7.m1.6.6.6.3.2.4a.cmml">se l</mtext><mo id="S6.E7.m1.6.6.6.3.2.5" xref="S6.E7.m1.6.6.6.3.2.5.cmml">∈</mo><mrow id="S6.E7.m1.6.6.6.3.2.6" xref="S6.E7.m1.6.6.6.3.2.6.cmml"><mrow id="S6.E7.m1.6.6.6.3.2.6.2.2" xref="S6.E7.m1.6.6.6.3.2.6.2.1.cmml"><mo stretchy="false" id="S6.E7.m1.6.6.6.3.2.6.2.2.1" xref="S6.E7.m1.6.6.6.3.2.6.2.1.cmml">{</mo><mi id="S6.E7.m1.5.5.5.2.1.1" xref="S6.E7.m1.5.5.5.2.1.1.cmml">D</mi><mo stretchy="false" id="S6.E7.m1.6.6.6.3.2.6.2.2.2" xref="S6.E7.m1.6.6.6.3.2.6.2.1.cmml">}</mo></mrow><mo lspace="0.500em" rspace="0em" id="S6.E7.m1.6.6.6.3.2.6.1" xref="S6.E7.m1.6.6.6.3.2.6.1.cmml">​</mo><mi id="S6.E7.m1.6.6.6.3.2.6.3" xref="S6.E7.m1.6.6.6.3.2.6.3.cmml">e</mi><mo lspace="0.500em" rspace="0em" id="S6.E7.m1.6.6.6.3.2.6.1a" xref="S6.E7.m1.6.6.6.3.2.6.1.cmml">​</mo><mtext id="S6.E7.m1.6.6.6.3.2.6.4" xref="S6.E7.m1.6.6.6.3.2.6.4a.cmml">k</mtext></mrow><mo id="S6.E7.m1.6.6.6.3.2.7" xref="S6.E7.m1.6.6.6.3.2.7.cmml">∉</mo><mrow id="S6.E7.m1.6.6.6.3.2.8.2" xref="S6.E7.m1.6.6.6.3.2.8.1.cmml"><mo stretchy="false" id="S6.E7.m1.6.6.6.3.2.8.2.1" xref="S6.E7.m1.6.6.6.3.2.8.1.cmml">{</mo><mi id="S6.E7.m1.6.6.6.3.2.2" xref="S6.E7.m1.6.6.6.3.2.2.cmml">D</mi><mo stretchy="false" id="S6.E7.m1.6.6.6.3.2.8.2.2" xref="S6.E7.m1.6.6.6.3.2.8.1.cmml">}</mo></mrow></mrow></mtd></mtr><mtr id="S6.E7.m1.11.11j" xref="S6.E7.m1.11.11.cmml"><mtd id="S6.E7.m1.11.11k" xref="S6.E7.m1.11.11.cmml"></mtd><mtd id="S6.E7.m1.11.11l" xref="S6.E7.m1.11.11.cmml"></mtd></mtr><mtr id="S6.E7.m1.11.11m" xref="S6.E7.m1.11.11.cmml"><mtd id="S6.E7.m1.11.11n" xref="S6.E7.m1.11.11.cmml"><mrow id="S6.E7.m1.7.7.7.1.1.1" xref="S6.E7.m1.7.7.7.1.1.1.1.cmml"><msub id="S6.E7.m1.7.7.7.1.1.1.1" xref="S6.E7.m1.7.7.7.1.1.1.1.cmml"><mi id="S6.E7.m1.7.7.7.1.1.1.1.2" xref="S6.E7.m1.7.7.7.1.1.1.1.2.cmml">s</mi><mrow id="S6.E7.m1.7.7.7.1.1.1.1.3" xref="S6.E7.m1.7.7.7.1.1.1.1.3.cmml"><mi id="S6.E7.m1.7.7.7.1.1.1.1.3.2" xref="S6.E7.m1.7.7.7.1.1.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.E7.m1.7.7.7.1.1.1.1.3.1" xref="S6.E7.m1.7.7.7.1.1.1.1.3.1.cmml">​</mo><mi id="S6.E7.m1.7.7.7.1.1.1.1.3.3" xref="S6.E7.m1.7.7.7.1.1.1.1.3.3.cmml">l</mi></mrow></msub><mo id="S6.E7.m1.7.7.7.1.1.1.2" xref="S6.E7.m1.7.7.7.1.1.1.1.cmml">,</mo></mrow></mtd><mtd id="S6.E7.m1.11.11o" xref="S6.E7.m1.11.11.cmml"><mrow id="S6.E7.m1.9.9.9.3.2" xref="S6.E7.m1.9.9.9.3.2.cmml"><mtext id="S6.E7.m1.9.9.9.3.2.4" xref="S6.E7.m1.9.9.9.3.2.4a.cmml">se k</mtext><mo id="S6.E7.m1.9.9.9.3.2.5" xref="S6.E7.m1.9.9.9.3.2.5.cmml">∈</mo><mrow id="S6.E7.m1.9.9.9.3.2.6" xref="S6.E7.m1.9.9.9.3.2.6.cmml"><mrow id="S6.E7.m1.9.9.9.3.2.6.2.2" xref="S6.E7.m1.9.9.9.3.2.6.2.1.cmml"><mo stretchy="false" id="S6.E7.m1.9.9.9.3.2.6.2.2.1" xref="S6.E7.m1.9.9.9.3.2.6.2.1.cmml">{</mo><mi id="S6.E7.m1.8.8.8.2.1.1" xref="S6.E7.m1.8.8.8.2.1.1.cmml">D</mi><mo stretchy="false" id="S6.E7.m1.9.9.9.3.2.6.2.2.2" xref="S6.E7.m1.9.9.9.3.2.6.2.1.cmml">}</mo></mrow><mo lspace="0.500em" rspace="0em" id="S6.E7.m1.9.9.9.3.2.6.1" xref="S6.E7.m1.9.9.9.3.2.6.1.cmml">​</mo><mi id="S6.E7.m1.9.9.9.3.2.6.3" xref="S6.E7.m1.9.9.9.3.2.6.3.cmml">e</mi><mo lspace="0.500em" rspace="0em" id="S6.E7.m1.9.9.9.3.2.6.1a" xref="S6.E7.m1.9.9.9.3.2.6.1.cmml">​</mo><mtext id="S6.E7.m1.9.9.9.3.2.6.4" xref="S6.E7.m1.9.9.9.3.2.6.4a.cmml">l</mtext></mrow><mo id="S6.E7.m1.9.9.9.3.2.7" xref="S6.E7.m1.9.9.9.3.2.7.cmml">∉</mo><mrow id="S6.E7.m1.9.9.9.3.2.8.2" xref="S6.E7.m1.9.9.9.3.2.8.1.cmml"><mo stretchy="false" id="S6.E7.m1.9.9.9.3.2.8.2.1" xref="S6.E7.m1.9.9.9.3.2.8.1.cmml">{</mo><mi id="S6.E7.m1.9.9.9.3.2.2" xref="S6.E7.m1.9.9.9.3.2.2.cmml">D</mi><mo stretchy="false" id="S6.E7.m1.9.9.9.3.2.8.2.2" xref="S6.E7.m1.9.9.9.3.2.8.1.cmml">}</mo></mrow></mrow></mtd></mtr><mtr id="S6.E7.m1.11.11p" xref="S6.E7.m1.11.11.cmml"><mtd id="S6.E7.m1.11.11q" xref="S6.E7.m1.11.11.cmml"></mtd><mtd id="S6.E7.m1.11.11r" xref="S6.E7.m1.11.11.cmml"></mtd></mtr><mtr id="S6.E7.m1.11.11s" xref="S6.E7.m1.11.11.cmml"><mtd id="S6.E7.m1.11.11t" xref="S6.E7.m1.11.11.cmml"><mrow id="S6.E7.m1.10.10.10.1.1.3" xref="S6.E7.m1.11.11.cmml"><mn id="S6.E7.m1.10.10.10.1.1.1" xref="S6.E7.m1.10.10.10.1.1.1.cmml">1</mn><mo id="S6.E7.m1.10.10.10.1.1.3.1" xref="S6.E7.m1.11.11.cmml">,</mo></mrow></mtd><mtd id="S6.E7.m1.11.11u" xref="S6.E7.m1.11.11.cmml"><mrow id="S6.E7.m1.11.11.11.2.1" xref="S6.E7.m1.11.11.11.2.1.cmml"><mtext id="S6.E7.m1.11.11.11.2.1.3" xref="S6.E7.m1.11.11.11.2.1.3a.cmml">se k,l</mtext><mo id="S6.E7.m1.11.11.11.2.1.2" xref="S6.E7.m1.11.11.11.2.1.2.cmml">∈</mo><mrow id="S6.E7.m1.11.11.11.2.1.4.2" xref="S6.E7.m1.11.11.11.2.1.4.1.cmml"><mo stretchy="false" id="S6.E7.m1.11.11.11.2.1.4.2.1" xref="S6.E7.m1.11.11.11.2.1.4.1.cmml">{</mo><mi id="S6.E7.m1.11.11.11.2.1.1" xref="S6.E7.m1.11.11.11.2.1.1.cmml">D</mi><mo stretchy="false" id="S6.E7.m1.11.11.11.2.1.4.2.2" xref="S6.E7.m1.11.11.11.2.1.4.1.cmml">}</mo></mrow></mrow></mtd></mtr></mtable><mi id="S6.E7.m1.11.12.3.3.2.2" xref="S6.E7.m1.11.12.3.3.1.1.cmml"></mi></mrow></mrow></mrow><annotation-xml encoding="MathML-Content" id="S6.E7.m1.11b"><apply id="S6.E7.m1.11.12.cmml" xref="S6.E7.m1.11.12"><csymbol cd="latexml" id="S6.E7.m1.11.12.1.cmml" xref="S6.E7.m1.11.12.1">therefore</csymbol><apply id="S6.E7.m1.11.12.2.cmml" xref="S6.E7.m1.11.12.2"><eq id="S6.E7.m1.11.12.2.1.cmml" xref="S6.E7.m1.11.12.2.1"></eq><apply id="S6.E7.m1.11.12.2.2.cmml" xref="S6.E7.m1.11.12.2.2"><csymbol cd="ambiguous" id="S6.E7.m1.11.12.2.2.1.cmml" xref="S6.E7.m1.11.12.2.2">subscript</csymbol><ci id="S6.E7.m1.11.12.2.2.2.cmml" xref="S6.E7.m1.11.12.2.2.2">𝑐</ci><apply id="S6.E7.m1.11.12.2.2.3.cmml" xref="S6.E7.m1.11.12.2.2.3"><times id="S6.E7.m1.11.12.2.2.3.1.cmml" xref="S6.E7.m1.11.12.2.2.3.1"></times><ci id="S6.E7.m1.11.12.2.2.3.2.cmml" xref="S6.E7.m1.11.12.2.2.3.2">𝑘</ci><ci id="S6.E7.m1.11.12.2.2.3.3.cmml" xref="S6.E7.m1.11.12.2.2.3.3">𝑙</ci></apply></apply><apply id="S6.E7.m1.11.12.2.3.cmml" xref="S6.E7.m1.11.12.2.3"><apply id="S6.E7.m1.11.12.2.3.1.cmml" xref="S6.E7.m1.11.12.2.3.1"><csymbol cd="ambiguous" id="S6.E7.m1.11.12.2.3.1.1.cmml" xref="S6.E7.m1.11.12.2.3.1">superscript</csymbol><apply id="S6.E7.m1.11.12.2.3.1.2.cmml" xref="S6.E7.m1.11.12.2.3.1"><csymbol cd="ambiguous" id="S6.E7.m1.11.12.2.3.1.2.1.cmml" xref="S6.E7.m1.11.12.2.3.1">subscript</csymbol><sum id="S6.E7.m1.11.12.2.3.1.2.2.cmml" xref="S6.E7.m1.11.12.2.3.1.2.2"></sum><ci id="S6.E7.m1.11.12.2.3.1.2.3.cmml" xref="S6.E7.m1.11.12.2.3.1.2.3">𝑖</ci></apply><ci id="S6.E7.m1.11.12.2.3.1.3.cmml" xref="S6.E7.m1.11.12.2.3.1.3">𝑁</ci></apply><apply id="S6.E7.m1.11.12.2.3.2.cmml" xref="S6.E7.m1.11.12.2.3.2"><csymbol cd="ambiguous" id="S6.E7.m1.11.12.2.3.2.1.cmml" xref="S6.E7.m1.11.12.2.3.2">subscript</csymbol><ci id="S6.E7.m1.11.12.2.3.2.2.cmml" xref="S6.E7.m1.11.12.2.3.2.2">𝑓</ci><apply id="S6.E7.m1.11.12.2.3.2.3.cmml" xref="S6.E7.m1.11.12.2.3.2.3"><times id="S6.E7.m1.11.12.2.3.2.3.1.cmml" xref="S6.E7.m1.11.12.2.3.2.3.1"></times><ci id="S6.E7.m1.11.12.2.3.2.3.2.cmml" xref="S6.E7.m1.11.12.2.3.2.3.2">𝑘</ci><ci id="S6.E7.m1.11.12.2.3.2.3.3.cmml" xref="S6.E7.m1.11.12.2.3.2.3.3">𝑙</ci></apply></apply></apply></apply><apply id="S6.E7.m1.11.12.3.cmml" xref="S6.E7.m1.11.12.3"><eq id="S6.E7.m1.11.12.3.1.cmml" xref="S6.E7.m1.11.12.3.1"></eq><apply id="S6.E7.m1.11.12.3.2.cmml" xref="S6.E7.m1.11.12.3.2"><csymbol cd="ambiguous" id="S6.E7.m1.11.12.3.2.1.cmml" xref="S6.E7.m1.11.12.3.2">subscript</csymbol><ci id="S6.E7.m1.11.12.3.2.2.cmml" xref="S6.E7.m1.11.12.3.2.2">𝑓</ci><apply id="S6.E7.m1.11.12.3.2.3.cmml" xref="S6.E7.m1.11.12.3.2.3"><times id="S6.E7.m1.11.12.3.2.3.1.cmml" xref="S6.E7.m1.11.12.3.2.3.1"></times><ci id="S6.E7.m1.11.12.3.2.3.2.cmml" xref="S6.E7.m1.11.12.3.2.3.2">𝑘</ci><ci id="S6.E7.m1.11.12.3.2.3.3.cmml" xref="S6.E7.m1.11.12.3.2.3.3">𝑙</ci></apply></apply><apply id="S6.E7.m1.11.12.3.3.1.cmml" xref="S6.E7.m1.11.12.3.3.2"><csymbol cd="latexml" id="S6.E7.m1.11.12.3.3.1.1.cmml" xref="S6.E7.m1.11.12.3.3.2.1">cases</csymbol><matrix id="S6.E7.m1.11.11.cmml" xref="S6.E7.m1.11.11"><matrixrow id="S6.E7.m1.11.11a.cmml" xref="S6.E7.m1.11.11"><apply id="S6.E7.m1.1.1.1.1.1.1.cmml" xref="S6.E7.m1.1.1.1.1.1.3"><divide id="S6.E7.m1.1.1.1.1.1.1.1.cmml" xref="S6.E7.m1.1.1.1.1.1.3"></divide><apply id="S6.E7.m1.1.1.1.1.1.1.2.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2"><plus id="S6.E7.m1.1.1.1.1.1.1.2.1.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.1"></plus><apply id="S6.E7.m1.1.1.1.1.1.1.2.2.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.2"><csymbol cd="ambiguous" id="S6.E7.m1.1.1.1.1.1.1.2.2.1.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.2">subscript</csymbol><ci id="S6.E7.m1.1.1.1.1.1.1.2.2.2.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.2.2">𝑠</ci><apply id="S6.E7.m1.1.1.1.1.1.1.2.2.3.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.2.3"><times id="S6.E7.m1.1.1.1.1.1.1.2.2.3.1.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.2.3.1"></times><ci id="S6.E7.m1.1.1.1.1.1.1.2.2.3.2.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.2.3.2">𝑖</ci><ci id="S6.E7.m1.1.1.1.1.1.1.2.2.3.3.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.2.3.3">𝑘</ci></apply></apply><apply id="S6.E7.m1.1.1.1.1.1.1.2.3.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.3"><csymbol cd="ambiguous" id="S6.E7.m1.1.1.1.1.1.1.2.3.1.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.3">subscript</csymbol><ci id="S6.E7.m1.1.1.1.1.1.1.2.3.2.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.3.2">𝑠</ci><apply id="S6.E7.m1.1.1.1.1.1.1.2.3.3.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.3.3"><times id="S6.E7.m1.1.1.1.1.1.1.2.3.3.1.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.3.3.1"></times><ci id="S6.E7.m1.1.1.1.1.1.1.2.3.3.2.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.3.3.2">𝑖</ci><ci id="S6.E7.m1.1.1.1.1.1.1.2.3.3.3.cmml" xref="S6.E7.m1.1.1.1.1.1.1.2.3.3.3">𝑙</ci></apply></apply></apply><cn type="integer" id="S6.E7.m1.1.1.1.1.1.1.3.cmml" xref="S6.E7.m1.1.1.1.1.1.1.3">2</cn></apply><apply id="S6.E7.m1.3.3.3.3.2.cmml" xref="S6.E7.m1.3.3.3.3.2"><in id="S6.E7.m1.3.3.3.3.2.3.cmml" xref="S6.E7.m1.3.3.3.3.2.3"></in><ci id="S6.E7.m1.3.3.3.3.2.4a.cmml" xref="S6.E7.m1.3.3.3.3.2.4"><mtext id="S6.E7.m1.3.3.3.3.2.4.cmml" xref="S6.E7.m1.3.3.3.3.2.4">se k,l</mtext></ci><set id="S6.E7.m1.3.3.3.3.2.5.1.cmml" xref="S6.E7.m1.3.3.3.3.2.5.2"><ci id="S6.E7.m1.2.2.2.2.1.1.cmml" xref="S6.E7.m1.2.2.2.2.1.1">𝑀</ci><ci id="S6.E7.m1.3.3.3.3.2.2.cmml" xref="S6.E7.m1.3.3.3.3.2.2">𝐼</ci></set></apply></matrixrow><matrixrow id="S6.E7.m1.11.11b.cmml" xref="S6.E7.m1.11.11"><cerror id="S6.E7.m1.11.11c.cmml" xref="S6.E7.m1.11.11"><csymbol cd="ambiguous" id="S6.E7.m1.11.11d.cmml" xref="S6.E7.m1.11.11">missing-subexpression</csymbol></cerror><cerror id="S6.E7.m1.11.11e.cmml" xref="S6.E7.m1.11.11"><csymbol cd="ambiguous" id="S6.E7.m1.11.11f.cmml" xref="S6.E7.m1.11.11">missing-subexpression</csymbol></cerror></matrixrow><matrixrow id="S6.E7.m1.11.11g.cmml" xref="S6.E7.m1.11.11"><apply id="S6.E7.m1.4.4.4.1.1.1.1.cmml" xref="S6.E7.m1.4.4.4.1.1.1"><csymbol cd="ambiguous" id="S6.E7.m1.4.4.4.1.1.1.1.1.cmml" xref="S6.E7.m1.4.4.4.1.1.1">subscript</csymbol><ci id="S6.E7.m1.4.4.4.1.1.1.1.2.cmml" xref="S6.E7.m1.4.4.4.1.1.1.1.2">𝑠</ci><apply id="S6.E7.m1.4.4.4.1.1.1.1.3.cmml" xref="S6.E7.m1.4.4.4.1.1.1.1.3"><times id="S6.E7.m1.4.4.4.1.1.1.1.3.1.cmml" xref="S6.E7.m1.4.4.4.1.1.1.1.3.1"></times><ci id="S6.E7.m1.4.4.4.1.1.1.1.3.2.cmml" xref="S6.E7.m1.4.4.4.1.1.1.1.3.2">𝑖</ci><ci id="S6.E7.m1.4.4.4.1.1.1.1.3.3.cmml" xref="S6.E7.m1.4.4.4.1.1.1.1.3.3">𝑘</ci></apply></apply><apply id="S6.E7.m1.6.6.6.3.2.cmml" xref="S6.E7.m1.6.6.6.3.2"><and id="S6.E7.m1.6.6.6.3.2a.cmml" xref="S6.E7.m1.6.6.6.3.2"></and><apply id="S6.E7.m1.6.6.6.3.2b.cmml" xref="S6.E7.m1.6.6.6.3.2"><in id="S6.E7.m1.6.6.6.3.2.5.cmml" xref="S6.E7.m1.6.6.6.3.2.5"></in><ci id="S6.E7.m1.6.6.6.3.2.4a.cmml" xref="S6.E7.m1.6.6.6.3.2.4"><mtext id="S6.E7.m1.6.6.6.3.2.4.cmml" xref="S6.E7.m1.6.6.6.3.2.4">se l</mtext></ci><apply id="S6.E7.m1.6.6.6.3.2.6.cmml" xref="S6.E7.m1.6.6.6.3.2.6"><times id="S6.E7.m1.6.6.6.3.2.6.1.cmml" xref="S6.E7.m1.6.6.6.3.2.6.1"></times><set id="S6.E7.m1.6.6.6.3.2.6.2.1.cmml" xref="S6.E7.m1.6.6.6.3.2.6.2.2"><ci id="S6.E7.m1.5.5.5.2.1.1.cmml" xref="S6.E7.m1.5.5.5.2.1.1">𝐷</ci></set><ci id="S6.E7.m1.6.6.6.3.2.6.3.cmml" xref="S6.E7.m1.6.6.6.3.2.6.3">𝑒</ci><ci id="S6.E7.m1.6.6.6.3.2.6.4a.cmml" xref="S6.E7.m1.6.6.6.3.2.6.4"><mtext id="S6.E7.m1.6.6.6.3.2.6.4.cmml" xref="S6.E7.m1.6.6.6.3.2.6.4">k</mtext></ci></apply></apply><apply id="S6.E7.m1.6.6.6.3.2c.cmml" xref="S6.E7.m1.6.6.6.3.2"><notin id="S6.E7.m1.6.6.6.3.2.7.cmml" xref="S6.E7.m1.6.6.6.3.2.7"></notin><share href="#S6.E7.m1.6.6.6.3.2.6.cmml" id="S6.E7.m1.6.6.6.3.2d.cmml" xref="S6.E7.m1.6.6.6.3.2"></share><set id="S6.E7.m1.6.6.6.3.2.8.1.cmml" xref="S6.E7.m1.6.6.6.3.2.8.2"><ci id="S6.E7.m1.6.6.6.3.2.2.cmml" xref="S6.E7.m1.6.6.6.3.2.2">𝐷</ci></set></apply></apply></matrixrow><matrixrow id="S6.E7.m1.11.11h.cmml" xref="S6.E7.m1.11.11"><cerror id="S6.E7.m1.11.11i.cmml" xref="S6.E7.m1.11.11"><csymbol cd="ambiguous" id="S6.E7.m1.11.11j.cmml" xref="S6.E7.m1.11.11">missing-subexpression</csymbol></cerror><cerror id="S6.E7.m1.11.11k.cmml" xref="S6.E7.m1.11.11"><csymbol cd="ambiguous" id="S6.E7.m1.11.11l.cmml" xref="S6.E7.m1.11.11">missing-subexpression</csymbol></cerror></matrixrow><matrixrow id="S6.E7.m1.11.11m.cmml" xref="S6.E7.m1.11.11"><apply id="S6.E7.m1.7.7.7.1.1.1.1.cmml" xref="S6.E7.m1.7.7.7.1.1.1"><csymbol cd="ambiguous" id="S6.E7.m1.7.7.7.1.1.1.1.1.cmml" xref="S6.E7.m1.7.7.7.1.1.1">subscript</csymbol><ci id="S6.E7.m1.7.7.7.1.1.1.1.2.cmml" xref="S6.E7.m1.7.7.7.1.1.1.1.2">𝑠</ci><apply id="S6.E7.m1.7.7.7.1.1.1.1.3.cmml" xref="S6.E7.m1.7.7.7.1.1.1.1.3"><times id="S6.E7.m1.7.7.7.1.1.1.1.3.1.cmml" xref="S6.E7.m1.7.7.7.1.1.1.1.3.1"></times><ci id="S6.E7.m1.7.7.7.1.1.1.1.3.2.cmml" xref="S6.E7.m1.7.7.7.1.1.1.1.3.2">𝑖</ci><ci id="S6.E7.m1.7.7.7.1.1.1.1.3.3.cmml" xref="S6.E7.m1.7.7.7.1.1.1.1.3.3">𝑙</ci></apply></apply><apply id="S6.E7.m1.9.9.9.3.2.cmml" xref="S6.E7.m1.9.9.9.3.2"><and id="S6.E7.m1.9.9.9.3.2a.cmml" xref="S6.E7.m1.9.9.9.3.2"></and><apply id="S6.E7.m1.9.9.9.3.2b.cmml" xref="S6.E7.m1.9.9.9.3.2"><in id="S6.E7.m1.9.9.9.3.2.5.cmml" xref="S6.E7.m1.9.9.9.3.2.5"></in><ci id="S6.E7.m1.9.9.9.3.2.4a.cmml" xref="S6.E7.m1.9.9.9.3.2.4"><mtext id="S6.E7.m1.9.9.9.3.2.4.cmml" xref="S6.E7.m1.9.9.9.3.2.4">se k</mtext></ci><apply id="S6.E7.m1.9.9.9.3.2.6.cmml" xref="S6.E7.m1.9.9.9.3.2.6"><times id="S6.E7.m1.9.9.9.3.2.6.1.cmml" xref="S6.E7.m1.9.9.9.3.2.6.1"></times><set id="S6.E7.m1.9.9.9.3.2.6.2.1.cmml" xref="S6.E7.m1.9.9.9.3.2.6.2.2"><ci id="S6.E7.m1.8.8.8.2.1.1.cmml" xref="S6.E7.m1.8.8.8.2.1.1">𝐷</ci></set><ci id="S6.E7.m1.9.9.9.3.2.6.3.cmml" xref="S6.E7.m1.9.9.9.3.2.6.3">𝑒</ci><ci id="S6.E7.m1.9.9.9.3.2.6.4a.cmml" xref="S6.E7.m1.9.9.9.3.2.6.4"><mtext id="S6.E7.m1.9.9.9.3.2.6.4.cmml" xref="S6.E7.m1.9.9.9.3.2.6.4">l</mtext></ci></apply></apply><apply id="S6.E7.m1.9.9.9.3.2c.cmml" xref="S6.E7.m1.9.9.9.3.2"><notin id="S6.E7.m1.9.9.9.3.2.7.cmml" xref="S6.E7.m1.9.9.9.3.2.7"></notin><share href="#S6.E7.m1.9.9.9.3.2.6.cmml" id="S6.E7.m1.9.9.9.3.2d.cmml" xref="S6.E7.m1.9.9.9.3.2"></share><set id="S6.E7.m1.9.9.9.3.2.8.1.cmml" xref="S6.E7.m1.9.9.9.3.2.8.2"><ci id="S6.E7.m1.9.9.9.3.2.2.cmml" xref="S6.E7.m1.9.9.9.3.2.2">𝐷</ci></set></apply></apply></matrixrow><matrixrow id="S6.E7.m1.11.11n.cmml" xref="S6.E7.m1.11.11"><cerror id="S6.E7.m1.11.11o.cmml" xref="S6.E7.m1.11.11"><csymbol cd="ambiguous" id="S6.E7.m1.11.11p.cmml" xref="S6.E7.m1.11.11">missing-subexpression</csymbol></cerror><cerror id="S6.E7.m1.11.11q.cmml" xref="S6.E7.m1.11.11"><csymbol cd="ambiguous" id="S6.E7.m1.11.11r.cmml" xref="S6.E7.m1.11.11">missing-subexpression</csymbol></cerror></matrixrow><matrixrow id="S6.E7.m1.11.11s.cmml" xref="S6.E7.m1.11.11"><cn type="integer" id="S6.E7.m1.10.10.10.1.1.1.cmml" xref="S6.E7.m1.10.10.10.1.1.1">1</cn><apply id="S6.E7.m1.11.11.11.2.1.cmml" xref="S6.E7.m1.11.11.11.2.1"><in id="S6.E7.m1.11.11.11.2.1.2.cmml" xref="S6.E7.m1.11.11.11.2.1.2"></in><ci id="S6.E7.m1.11.11.11.2.1.3a.cmml" xref="S6.E7.m1.11.11.11.2.1.3"><mtext id="S6.E7.m1.11.11.11.2.1.3.cmml" xref="S6.E7.m1.11.11.11.2.1.3">se k,l</mtext></ci><set id="S6.E7.m1.11.11.11.2.1.4.1.cmml" xref="S6.E7.m1.11.11.11.2.1.4.2"><ci id="S6.E7.m1.11.11.11.2.1.1.cmml" xref="S6.E7.m1.11.11.11.2.1.1">𝐷</ci></set></apply></matrixrow></matrix></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.E7.m1.11c">\\
c_{kl}=\sum_{i}^{N}f_{kl}\therefore\\
f_{kl}=\left\{\begin{array}[]{cc}\dfrac{s_{ik}+s_{il}}{2},&amp;\textrm{se k,l}\in{\{M,I\}}\\
\\
s_{ik},&amp;\textrm{se l}\in{\{D\}}\ e\ \textrm{k}\notin{\{D\}}\\
\\
s_{il},&amp;\textrm{se k}\in{\{D\}}\ e\ \textrm{l}\notin{\{D\}}\\
\\
1,&amp;\textrm{se k,l}\in{\{D\}}\end{array}\right.</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(7)</span></td>
</tr></tbody>
</table>
<br class="ltx_break">
</div>
</section>
<section id="S6.SS3" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">6.3 </span>The <math id="S6.SS3.1.m1.1" class="ltx_Math" alttext="M_{s}" display="inline"><semantics id="S6.SS3.1.m1.1b"><msub id="S6.SS3.1.m1.1.1" xref="S6.SS3.1.m1.1.1.cmml"><mi id="S6.SS3.1.m1.1.1.2" xref="S6.SS3.1.m1.1.1.2.cmml">M</mi><mi id="S6.SS3.1.m1.1.1.3" xref="S6.SS3.1.m1.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="S6.SS3.1.m1.1c"><apply id="S6.SS3.1.m1.1.1.cmml" xref="S6.SS3.1.m1.1.1"><csymbol cd="ambiguous" id="S6.SS3.1.m1.1.1.1.cmml" xref="S6.SS3.1.m1.1.1">subscript</csymbol><ci id="S6.SS3.1.m1.1.1.2.cmml" xref="S6.SS3.1.m1.1.1.2">𝑀</ci><ci id="S6.SS3.1.m1.1.1.3.cmml" xref="S6.SS3.1.m1.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.1.m1.1d">M_{s}</annotation></semantics></math> structural weight matrices</h3>

<div id="S6.SS3.p1" class="ltx_para">
<p id="S6.SS3.p1.1" class="ltx_p">As explained above, our algorithm considers a number of different sources of structural information.
Next, we approach how this information was obtained and used to built <math id="S6.SS3.p1.1.m1.1" class="ltx_Math" alttext="M_{s}" display="inline"><semantics id="S6.SS3.p1.1.m1.1a"><msub id="S6.SS3.p1.1.m1.1.1" xref="S6.SS3.p1.1.m1.1.1.cmml"><mi id="S6.SS3.p1.1.m1.1.1.2" xref="S6.SS3.p1.1.m1.1.1.2.cmml">M</mi><mi id="S6.SS3.p1.1.m1.1.1.3" xref="S6.SS3.p1.1.m1.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="S6.SS3.p1.1.m1.1b"><apply id="S6.SS3.p1.1.m1.1.1.cmml" xref="S6.SS3.p1.1.m1.1.1"><csymbol cd="ambiguous" id="S6.SS3.p1.1.m1.1.1.1.cmml" xref="S6.SS3.p1.1.m1.1.1">subscript</csymbol><ci id="S6.SS3.p1.1.m1.1.1.2.cmml" xref="S6.SS3.p1.1.m1.1.1.2">𝑀</ci><ci id="S6.SS3.p1.1.m1.1.1.3.cmml" xref="S6.SS3.p1.1.m1.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.p1.1.m1.1c">M_{s}</annotation></semantics></math> matrix.</p>
</div>
<section id="S6.SS3.SSS1" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">6.3.1 </span>Secondary structural elements</h4>

<div id="S6.SS3.SSS1.p1" class="ltx_para">
<p id="S6.SS3.SSS1.p1.1" class="ltx_p">Secondary structure is often conserved among homologue proteins. Indeed, <em id="S6.SS3.SSS1.p1.1.1" class="ltx_emph ltx_font_italic">motifs</em> <cite class="ltx_cite ltx_citemacro_citep">(Branden <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib9" title="" class="ltx_ref">1991</a>)</cite>, consensus sequences in homologue proteins, usually include a combination of well conserved secondary structure elements <cite class="ltx_cite ltx_citemacro_citep">(Chakrabarti <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib12" title="" class="ltx_ref">2004</a>)</cite>.</p>
</div>
<div id="S6.SS3.SSS1.p2" class="ltx_para">
<p id="S6.SS3.SSS1.p2.4" class="ltx_p">In order to build a <math id="S6.SS3.SSS1.p2.1.m1.1" class="ltx_Math" alttext="M_{s}" display="inline"><semantics id="S6.SS3.SSS1.p2.1.m1.1a"><msub id="S6.SS3.SSS1.p2.1.m1.1.1" xref="S6.SS3.SSS1.p2.1.m1.1.1.cmml"><mi id="S6.SS3.SSS1.p2.1.m1.1.1.2" xref="S6.SS3.SSS1.p2.1.m1.1.1.2.cmml">M</mi><mi id="S6.SS3.SSS1.p2.1.m1.1.1.3" xref="S6.SS3.SSS1.p2.1.m1.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS1.p2.1.m1.1b"><apply id="S6.SS3.SSS1.p2.1.m1.1.1.cmml" xref="S6.SS3.SSS1.p2.1.m1.1.1"><csymbol cd="ambiguous" id="S6.SS3.SSS1.p2.1.m1.1.1.1.cmml" xref="S6.SS3.SSS1.p2.1.m1.1.1">subscript</csymbol><ci id="S6.SS3.SSS1.p2.1.m1.1.1.2.cmml" xref="S6.SS3.SSS1.p2.1.m1.1.1.2">𝑀</ci><ci id="S6.SS3.SSS1.p2.1.m1.1.1.3.cmml" xref="S6.SS3.SSS1.p2.1.m1.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS1.p2.1.m1.1c">M_{s}</annotation></semantics></math> matrix based on secondary structure elements we need to identify secondary structure elements in the original sequences. This is possible because we assume we have full structural data for the <em id="S6.SS3.SSS1.p2.4.1" class="ltx_emph ltx_font_italic">training</em> sequences. In this work, we chose to utilize the SSTRUCT program, part of the widely used <span id="S6.SS3.SSS1.p2.4.2" class="ltx_text ltx_font_typewriter">joy</span> package <cite class="ltx_cite ltx_citemacro_citep">(Mizuguchi <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib31" title="" class="ltx_ref">1998</a>)</cite>, to extract secondary elements from the PDB files. SSTRUCT output is a character sequence, such that the characters {L=loop, H=helix, C=sheet} match a secondary structure element against a residue, as shown in figure <a href="#S6.F2" title="Figure 2 ‣ 6.3.1 Secondary structural elements ‣ 6.3 The 𝑀_𝑠 structural weight matrices ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>. Following Deane’s work on the relative frequency of conserved regions <cite class="ltx_cite ltx_citemacro_citep">(Deane <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib13" title="" class="ltx_ref">2003</a>)</cite>, we mapped each SSTRUCT element as follows: <math id="S6.SS3.SSS1.p2.2.m2.1" class="ltx_Math" alttext="L\rightarrow 1" display="inline"><semantics id="S6.SS3.SSS1.p2.2.m2.1a"><mrow id="S6.SS3.SSS1.p2.2.m2.1.1" xref="S6.SS3.SSS1.p2.2.m2.1.1.cmml"><mi id="S6.SS3.SSS1.p2.2.m2.1.1.2" xref="S6.SS3.SSS1.p2.2.m2.1.1.2.cmml">L</mi><mo stretchy="false" id="S6.SS3.SSS1.p2.2.m2.1.1.1" xref="S6.SS3.SSS1.p2.2.m2.1.1.1.cmml">→</mo><mn id="S6.SS3.SSS1.p2.2.m2.1.1.3" xref="S6.SS3.SSS1.p2.2.m2.1.1.3.cmml">1</mn></mrow><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS1.p2.2.m2.1b"><apply id="S6.SS3.SSS1.p2.2.m2.1.1.cmml" xref="S6.SS3.SSS1.p2.2.m2.1.1"><ci id="S6.SS3.SSS1.p2.2.m2.1.1.1.cmml" xref="S6.SS3.SSS1.p2.2.m2.1.1.1">→</ci><ci id="S6.SS3.SSS1.p2.2.m2.1.1.2.cmml" xref="S6.SS3.SSS1.p2.2.m2.1.1.2">𝐿</ci><cn type="integer" id="S6.SS3.SSS1.p2.2.m2.1.1.3.cmml" xref="S6.SS3.SSS1.p2.2.m2.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS1.p2.2.m2.1c">L\rightarrow 1</annotation></semantics></math>, <math id="S6.SS3.SSS1.p2.3.m3.1" class="ltx_Math" alttext="H\rightarrow 2" display="inline"><semantics id="S6.SS3.SSS1.p2.3.m3.1a"><mrow id="S6.SS3.SSS1.p2.3.m3.1.1" xref="S6.SS3.SSS1.p2.3.m3.1.1.cmml"><mi id="S6.SS3.SSS1.p2.3.m3.1.1.2" xref="S6.SS3.SSS1.p2.3.m3.1.1.2.cmml">H</mi><mo stretchy="false" id="S6.SS3.SSS1.p2.3.m3.1.1.1" xref="S6.SS3.SSS1.p2.3.m3.1.1.1.cmml">→</mo><mn id="S6.SS3.SSS1.p2.3.m3.1.1.3" xref="S6.SS3.SSS1.p2.3.m3.1.1.3.cmml">2</mn></mrow><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS1.p2.3.m3.1b"><apply id="S6.SS3.SSS1.p2.3.m3.1.1.cmml" xref="S6.SS3.SSS1.p2.3.m3.1.1"><ci id="S6.SS3.SSS1.p2.3.m3.1.1.1.cmml" xref="S6.SS3.SSS1.p2.3.m3.1.1.1">→</ci><ci id="S6.SS3.SSS1.p2.3.m3.1.1.2.cmml" xref="S6.SS3.SSS1.p2.3.m3.1.1.2">𝐻</ci><cn type="integer" id="S6.SS3.SSS1.p2.3.m3.1.1.3.cmml" xref="S6.SS3.SSS1.p2.3.m3.1.1.3">2</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS1.p2.3.m3.1c">H\rightarrow 2</annotation></semantics></math>, and <math id="S6.SS3.SSS1.p2.4.m4.1" class="ltx_Math" alttext="C\rightarrow 4" display="inline"><semantics id="S6.SS3.SSS1.p2.4.m4.1a"><mrow id="S6.SS3.SSS1.p2.4.m4.1.1" xref="S6.SS3.SSS1.p2.4.m4.1.1.cmml"><mi id="S6.SS3.SSS1.p2.4.m4.1.1.2" xref="S6.SS3.SSS1.p2.4.m4.1.1.2.cmml">C</mi><mo stretchy="false" id="S6.SS3.SSS1.p2.4.m4.1.1.1" xref="S6.SS3.SSS1.p2.4.m4.1.1.1.cmml">→</mo><mn id="S6.SS3.SSS1.p2.4.m4.1.1.3" xref="S6.SS3.SSS1.p2.4.m4.1.1.3.cmml">4</mn></mrow><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS1.p2.4.m4.1b"><apply id="S6.SS3.SSS1.p2.4.m4.1.1.cmml" xref="S6.SS3.SSS1.p2.4.m4.1.1"><ci id="S6.SS3.SSS1.p2.4.m4.1.1.1.cmml" xref="S6.SS3.SSS1.p2.4.m4.1.1.1">→</ci><ci id="S6.SS3.SSS1.p2.4.m4.1.1.2.cmml" xref="S6.SS3.SSS1.p2.4.m4.1.1.2">𝐶</ci><cn type="integer" id="S6.SS3.SSS1.p2.4.m4.1.1.3.cmml" xref="S6.SS3.SSS1.p2.4.m4.1.1.3">4</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS1.p2.4.m4.1c">C\rightarrow 4</annotation></semantics></math>. Our mapping thus favours conservation in sheets, and gives default weight to loops. Although the active site of proteins can be found in loops, these regions often contain <span id="S6.SS3.SSS1.p2.4.3" class="ltx_text ltx_font_italic">indel</span> segments. Figure <a href="#S6.F2" title="Figure 2 ‣ 6.3.1 Secondary structural elements ‣ 6.3 The 𝑀_𝑠 structural weight matrices ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a> shows an example of structural weight attributions for proteins in a partial alignment.

<br class="ltx_break"></p>
</div>
<figure id="S6.F2" class="ltx_figure"><img src="/html/0704.2010/assets/x2.png" id="S6.F2.g1" class="ltx_graphics ltx_centering ltx_img_portrait" width="236" height="305" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 2: </span>Representation of the secondary structure elements through aligned numbers. Loop regions (L) is 1, helices (H) 2 and sheets (C) 4</figcaption>
</figure>
</section>
<section id="S6.SS3.SSS2" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">6.3.2 </span>Solvent Inaccessibility</h4>

<div id="S6.SS3.SSS2.p1" class="ltx_para">
<p id="S6.SS3.SSS2.p1.1" class="ltx_p">The hydrophobic interactions of non-polar side chains in amino-acids are believed to contribute significantly to the stability of the tertiary structures in proteins. Hydrophobic amino-acids will tend to cluster together, not as a result of attraction, but as a result of their repulsion by the hydrogen bond water network in which the protein is dissolved. Therefore, these amino-acids will preferentially be located away from the surface of the molecule. Since they form the core of protein, they tend to be more conserved and are, thus, more useful for identifying remote evolutionary relationships.</p>
</div>
<div id="S6.SS3.SSS2.p2" class="ltx_para">
<p id="S6.SS3.SSS2.p2.2" class="ltx_p">We have utilized the PSA <cite class="ltx_cite ltx_citemacro_citep">(Lee <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib28" title="" class="ltx_ref">1971</a>)</cite> program to provide solvent inaccessibility information. PSA is part of the JOY package. The <math id="S6.SS3.SSS2.p2.1.m1.1" class="ltx_Math" alttext="M_{s}" display="inline"><semantics id="S6.SS3.SSS2.p2.1.m1.1a"><msub id="S6.SS3.SSS2.p2.1.m1.1.1" xref="S6.SS3.SSS2.p2.1.m1.1.1.cmml"><mi id="S6.SS3.SSS2.p2.1.m1.1.1.2" xref="S6.SS3.SSS2.p2.1.m1.1.1.2.cmml">M</mi><mi id="S6.SS3.SSS2.p2.1.m1.1.1.3" xref="S6.SS3.SSS2.p2.1.m1.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS2.p2.1.m1.1b"><apply id="S6.SS3.SSS2.p2.1.m1.1.1.cmml" xref="S6.SS3.SSS2.p2.1.m1.1.1"><csymbol cd="ambiguous" id="S6.SS3.SSS2.p2.1.m1.1.1.1.cmml" xref="S6.SS3.SSS2.p2.1.m1.1.1">subscript</csymbol><ci id="S6.SS3.SSS2.p2.1.m1.1.1.2.cmml" xref="S6.SS3.SSS2.p2.1.m1.1.1.2">𝑀</ci><ci id="S6.SS3.SSS2.p2.1.m1.1.1.3.cmml" xref="S6.SS3.SSS2.p2.1.m1.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS2.p2.1.m1.1c">M_{s}</annotation></semantics></math> matrix was built giving weight 3 for inaccessible residues and weight one to the others. The weights are based on <cite class="ltx_cite ltx_citemacro_citep">(Chakrabarti <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib12" title="" class="ltx_ref">2004</a>)</cite>, which demonstrated empirically that inaccessible amino-acids are three times more conserved than accessible amino-acids. The <math id="S6.SS3.SSS2.p2.2.m2.1" class="ltx_Math" alttext="M_{s}" display="inline"><semantics id="S6.SS3.SSS2.p2.2.m2.1a"><msub id="S6.SS3.SSS2.p2.2.m2.1.1" xref="S6.SS3.SSS2.p2.2.m2.1.1.cmml"><mi id="S6.SS3.SSS2.p2.2.m2.1.1.2" xref="S6.SS3.SSS2.p2.2.m2.1.1.2.cmml">M</mi><mi id="S6.SS3.SSS2.p2.2.m2.1.1.3" xref="S6.SS3.SSS2.p2.2.m2.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS2.p2.2.m2.1b"><apply id="S6.SS3.SSS2.p2.2.m2.1.1.cmml" xref="S6.SS3.SSS2.p2.2.m2.1.1"><csymbol cd="ambiguous" id="S6.SS3.SSS2.p2.2.m2.1.1.1.cmml" xref="S6.SS3.SSS2.p2.2.m2.1.1">subscript</csymbol><ci id="S6.SS3.SSS2.p2.2.m2.1.1.2.cmml" xref="S6.SS3.SSS2.p2.2.m2.1.1.2">𝑀</ci><ci id="S6.SS3.SSS2.p2.2.m2.1.1.3.cmml" xref="S6.SS3.SSS2.p2.2.m2.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS2.p2.2.m2.1c">M_{s}</annotation></semantics></math> matrix represents structural weights that were used to build the model <span id="S6.SS3.SSS2.p2.2.1" class="ltx_text ltx_font_italic">pHMMAcc</span>, as shown in figure <a href="#S6.F1" title="Figure 1 ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.

<br class="ltx_break"></p>
</div>
</section>
<section id="S6.SS3.SSS3" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">6.3.3 </span>Packing density</h4>

<div id="S6.SS3.SSS3.p1" class="ltx_para">
<p id="S6.SS3.SSS3.p1.2" class="ltx_p">The tertiary structure of proteins stems from a very large number of atomic interactions. In regions where the interactions are stronger residues tend to be packed together. It is well known that densely packed regions tend to be preserved, and hence that amino-acids belonging to those regions are usually more conserved than other amino-acids. TJ Ooi created a measure, called the Ooi Number <cite class="ltx_cite ltx_citemacro_citep">(Nishikawa <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib32" title="" class="ltx_ref">1986</a>)</cite>, that estimates the amino-acid packing density. Essentially, the Ooi number counts for a residue counts the number of neighboring C-<math id="S6.SS3.SSS3.p1.1.m1.1" class="ltx_Math" alttext="\alpha" display="inline"><semantics id="S6.SS3.SSS3.p1.1.m1.1a"><mi id="S6.SS3.SSS3.p1.1.m1.1.1" xref="S6.SS3.SSS3.p1.1.m1.1.1.cmml">α</mi><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS3.p1.1.m1.1b"><ci id="S6.SS3.SSS3.p1.1.m1.1.1.cmml" xref="S6.SS3.SSS3.p1.1.m1.1.1">𝛼</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS3.p1.1.m1.1c">\alpha</annotation></semantics></math> atoms within a radius of 14Å of the given residue’s own C-<math id="S6.SS3.SSS3.p1.2.m2.1" class="ltx_Math" alttext="\alpha" display="inline"><semantics id="S6.SS3.SSS3.p1.2.m2.1a"><mi id="S6.SS3.SSS3.p1.2.m2.1.1" xref="S6.SS3.SSS3.p1.2.m2.1.1.cmml">α</mi><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS3.p1.2.m2.1b"><ci id="S6.SS3.SSS3.p1.2.m2.1.1.cmml" xref="S6.SS3.SSS3.p1.2.m2.1.1">𝛼</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS3.p1.2.m2.1c">\alpha</annotation></semantics></math>. Although crude, this measure does give a good impression of which parts of the structure are buried and which are exposed on the surface.</p>
</div>
<div id="S6.SS3.SSS3.p2" class="ltx_para">
<p id="S6.SS3.SSS3.p2.1" class="ltx_p">We again use the JOY package to obtain the Ooi number and estimate packing density. Figure <a href="#S6.F3" title="Figure 3 ‣ 6.3.3 Packing density ‣ 6.3 The 𝑀_𝑠 structural weight matrices ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a> shows a stretch of JOY output, in which the numbers represent the Ooi measure for the Dehaloperoxidase protein in the Globins family (16wc PDB code). We used these numbers to build the structural weight matrix <math id="S6.SS3.SSS3.p2.1.m1.1" class="ltx_Math" alttext="M_{s}" display="inline"><semantics id="S6.SS3.SSS3.p2.1.m1.1a"><msub id="S6.SS3.SSS3.p2.1.m1.1.1" xref="S6.SS3.SSS3.p2.1.m1.1.1.cmml"><mi id="S6.SS3.SSS3.p2.1.m1.1.1.2" xref="S6.SS3.SSS3.p2.1.m1.1.1.2.cmml">M</mi><mi id="S6.SS3.SSS3.p2.1.m1.1.1.3" xref="S6.SS3.SSS3.p2.1.m1.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS3.p2.1.m1.1b"><apply id="S6.SS3.SSS3.p2.1.m1.1.1.cmml" xref="S6.SS3.SSS3.p2.1.m1.1.1"><csymbol cd="ambiguous" id="S6.SS3.SSS3.p2.1.m1.1.1.1.cmml" xref="S6.SS3.SSS3.p2.1.m1.1.1">subscript</csymbol><ci id="S6.SS3.SSS3.p2.1.m1.1.1.2.cmml" xref="S6.SS3.SSS3.p2.1.m1.1.1.2">𝑀</ci><ci id="S6.SS3.SSS3.p2.1.m1.1.1.3.cmml" xref="S6.SS3.SSS3.p2.1.m1.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS3.p2.1.m1.1c">M_{s}</annotation></semantics></math>. The structural weights were than used to build the model <span id="S6.SS3.SSS3.p2.1.1" class="ltx_text ltx_font_italic">pHMMOoi</span>, as shown in figure <a href="#S6.F1" title="Figure 1 ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.</p>
</div>
<figure id="S6.F3" class="ltx_figure"><img src="/html/0704.2010/assets/x3.png" id="S6.F3.g1" class="ltx_graphics ltx_centering ltx_img_portrait" width="283" height="366" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 3: </span>Ooi measure for the Dehaloperoxidase protein of Globins family (16wc PDB code), each number represents the amount of neighbor amino-acids inside a radius of 14Å.</figcaption>
</figure>
</section>
<section id="S6.SS3.SSS4" class="ltx_subsubsection">
<h4 class="ltx_title ltx_title_subsubsection">
<span class="ltx_tag ltx_tag_subsubsection">6.3.4 </span>Homologuous Core Structure</h4>

<div id="S6.SS3.SSS4.p1" class="ltx_para">
<p id="S6.SS3.SSS4.p1.1" class="ltx_p">Structural similarity among proteins can provide valuable insights into their functionality. One way to provide structural similarities is through three-dimensional alignment of proteins, also called <em id="S6.SS3.SSS4.p1.1.1" class="ltx_emph ltx_font_italic">structural alignment</em>. The goal is to align two or more proteins by trying to overlap the three-dimensional coordinates of their atoms. When multiple homologue proteins are structurally aligned, we tend to observe that there is a subset of coordinates whose spatial locations are better conserved across structural alignment. This subset is called the <em id="S6.SS3.SSS4.p1.1.2" class="ltx_emph ltx_font_italic">homolog core structure (HCS)</em> <cite class="ltx_cite ltx_citemacro_citep">(Matsuo <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib29" title="" class="ltx_ref">1999</a>)</cite>. According to the result reported by <cite class="ltx_cite ltx_citemacro_cite">Gerstein <span class="ltx_text ltx_font_italic">et al</span>., (<a href="#bib.bib18" title="" class="ltx_ref">1995</a>)</cite>, HCS can be utilized to detect homologue proteins.</p>
</div>
<div id="S6.SS3.SSS4.p2" class="ltx_para">
<p id="S6.SS3.SSS4.p2.2" class="ltx_p">Our goal was to estimate the HCS of a set of protein. As a first approximation, we propose a method to extract it from structural alignment by calculating how much aligned residues from different proteins tend to be close together. Following MAMMOTH, we represent residues through the coordinates of their C-<math id="S6.SS3.SSS4.p2.1.m1.1" class="ltx_Math" alttext="\alpha" display="inline"><semantics id="S6.SS3.SSS4.p2.1.m1.1a"><mi id="S6.SS3.SSS4.p2.1.m1.1.1" xref="S6.SS3.SSS4.p2.1.m1.1.1.cmml">α</mi><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS4.p2.1.m1.1b"><ci id="S6.SS3.SSS4.p2.1.m1.1.1.cmml" xref="S6.SS3.SSS4.p2.1.m1.1.1">𝛼</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS4.p2.1.m1.1c">\alpha</annotation></semantics></math> atoms. In other words, we assume that closeness between C-<math id="S6.SS3.SSS4.p2.2.m2.1" class="ltx_Math" alttext="\alpha" display="inline"><semantics id="S6.SS3.SSS4.p2.2.m2.1a"><mi id="S6.SS3.SSS4.p2.2.m2.1.1" xref="S6.SS3.SSS4.p2.2.m2.1.1.cmml">α</mi><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS4.p2.2.m2.1b"><ci id="S6.SS3.SSS4.p2.2.m2.1.1.cmml" xref="S6.SS3.SSS4.p2.2.m2.1.1">𝛼</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS4.p2.2.m2.1c">\alpha</annotation></semantics></math> atoms will approximate overlapping among amino-acids. To find out how much amino-acids are close together, we utilize the Euclidian distance measure, as shown in the equation <a href="#S6.E8" title="In 6.3.4 Homologuous Core Structure ‣ 6.3 The 𝑀_𝑠 structural weight matrices ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>. It represents the shortest distance between two points in the space.

<br class="ltx_break"></p>
</div>
<div id="S6.SS3.SSS4.p3" class="ltx_para">
<table id="S6.E8" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S6.E8.m1.5" class="ltx_Math" alttext="de_{a,b}=\sqrt{(x_{a}-x_{b})^{2}+(y_{a}-y_{b})^{2}+(z_{a}-z_{b})^{2}}" display="block"><semantics id="S6.E8.m1.5a"><mrow id="S6.E8.m1.5.6" xref="S6.E8.m1.5.6.cmml"><mrow id="S6.E8.m1.5.6.2" xref="S6.E8.m1.5.6.2.cmml"><mi id="S6.E8.m1.5.6.2.2" xref="S6.E8.m1.5.6.2.2.cmml">d</mi><mo lspace="0em" rspace="0em" id="S6.E8.m1.5.6.2.1" xref="S6.E8.m1.5.6.2.1.cmml">​</mo><msub id="S6.E8.m1.5.6.2.3" xref="S6.E8.m1.5.6.2.3.cmml"><mi id="S6.E8.m1.5.6.2.3.2" xref="S6.E8.m1.5.6.2.3.2.cmml">e</mi><mrow id="S6.E8.m1.2.2.2.4" xref="S6.E8.m1.2.2.2.3.cmml"><mi id="S6.E8.m1.1.1.1.1" xref="S6.E8.m1.1.1.1.1.cmml">a</mi><mo id="S6.E8.m1.2.2.2.4.1" xref="S6.E8.m1.2.2.2.3.cmml">,</mo><mi id="S6.E8.m1.2.2.2.2" xref="S6.E8.m1.2.2.2.2.cmml">b</mi></mrow></msub></mrow><mo id="S6.E8.m1.5.6.1" xref="S6.E8.m1.5.6.1.cmml">=</mo><msqrt id="S6.E8.m1.5.5" xref="S6.E8.m1.5.5.cmml"><mrow id="S6.E8.m1.5.5.3" xref="S6.E8.m1.5.5.3.cmml"><msup id="S6.E8.m1.3.3.1.1" xref="S6.E8.m1.3.3.1.1.cmml"><mrow id="S6.E8.m1.3.3.1.1.1.1" xref="S6.E8.m1.3.3.1.1.1.1.1.cmml"><mo stretchy="false" id="S6.E8.m1.3.3.1.1.1.1.2" xref="S6.E8.m1.3.3.1.1.1.1.1.cmml">(</mo><mrow id="S6.E8.m1.3.3.1.1.1.1.1" xref="S6.E8.m1.3.3.1.1.1.1.1.cmml"><msub id="S6.E8.m1.3.3.1.1.1.1.1.2" xref="S6.E8.m1.3.3.1.1.1.1.1.2.cmml"><mi id="S6.E8.m1.3.3.1.1.1.1.1.2.2" xref="S6.E8.m1.3.3.1.1.1.1.1.2.2.cmml">x</mi><mi id="S6.E8.m1.3.3.1.1.1.1.1.2.3" xref="S6.E8.m1.3.3.1.1.1.1.1.2.3.cmml">a</mi></msub><mo id="S6.E8.m1.3.3.1.1.1.1.1.1" xref="S6.E8.m1.3.3.1.1.1.1.1.1.cmml">−</mo><msub id="S6.E8.m1.3.3.1.1.1.1.1.3" xref="S6.E8.m1.3.3.1.1.1.1.1.3.cmml"><mi id="S6.E8.m1.3.3.1.1.1.1.1.3.2" xref="S6.E8.m1.3.3.1.1.1.1.1.3.2.cmml">x</mi><mi id="S6.E8.m1.3.3.1.1.1.1.1.3.3" xref="S6.E8.m1.3.3.1.1.1.1.1.3.3.cmml">b</mi></msub></mrow><mo stretchy="false" id="S6.E8.m1.3.3.1.1.1.1.3" xref="S6.E8.m1.3.3.1.1.1.1.1.cmml">)</mo></mrow><mn id="S6.E8.m1.3.3.1.1.3" xref="S6.E8.m1.3.3.1.1.3.cmml">2</mn></msup><mo id="S6.E8.m1.5.5.3.4" xref="S6.E8.m1.5.5.3.4.cmml">+</mo><msup id="S6.E8.m1.4.4.2.2" xref="S6.E8.m1.4.4.2.2.cmml"><mrow id="S6.E8.m1.4.4.2.2.1.1" xref="S6.E8.m1.4.4.2.2.1.1.1.cmml"><mo stretchy="false" id="S6.E8.m1.4.4.2.2.1.1.2" xref="S6.E8.m1.4.4.2.2.1.1.1.cmml">(</mo><mrow id="S6.E8.m1.4.4.2.2.1.1.1" xref="S6.E8.m1.4.4.2.2.1.1.1.cmml"><msub id="S6.E8.m1.4.4.2.2.1.1.1.2" xref="S6.E8.m1.4.4.2.2.1.1.1.2.cmml"><mi id="S6.E8.m1.4.4.2.2.1.1.1.2.2" xref="S6.E8.m1.4.4.2.2.1.1.1.2.2.cmml">y</mi><mi id="S6.E8.m1.4.4.2.2.1.1.1.2.3" xref="S6.E8.m1.4.4.2.2.1.1.1.2.3.cmml">a</mi></msub><mo id="S6.E8.m1.4.4.2.2.1.1.1.1" xref="S6.E8.m1.4.4.2.2.1.1.1.1.cmml">−</mo><msub id="S6.E8.m1.4.4.2.2.1.1.1.3" xref="S6.E8.m1.4.4.2.2.1.1.1.3.cmml"><mi id="S6.E8.m1.4.4.2.2.1.1.1.3.2" xref="S6.E8.m1.4.4.2.2.1.1.1.3.2.cmml">y</mi><mi id="S6.E8.m1.4.4.2.2.1.1.1.3.3" xref="S6.E8.m1.4.4.2.2.1.1.1.3.3.cmml">b</mi></msub></mrow><mo stretchy="false" id="S6.E8.m1.4.4.2.2.1.1.3" xref="S6.E8.m1.4.4.2.2.1.1.1.cmml">)</mo></mrow><mn id="S6.E8.m1.4.4.2.2.3" xref="S6.E8.m1.4.4.2.2.3.cmml">2</mn></msup><mo id="S6.E8.m1.5.5.3.4a" xref="S6.E8.m1.5.5.3.4.cmml">+</mo><msup id="S6.E8.m1.5.5.3.3" xref="S6.E8.m1.5.5.3.3.cmml"><mrow id="S6.E8.m1.5.5.3.3.1.1" xref="S6.E8.m1.5.5.3.3.1.1.1.cmml"><mo stretchy="false" id="S6.E8.m1.5.5.3.3.1.1.2" xref="S6.E8.m1.5.5.3.3.1.1.1.cmml">(</mo><mrow id="S6.E8.m1.5.5.3.3.1.1.1" xref="S6.E8.m1.5.5.3.3.1.1.1.cmml"><msub id="S6.E8.m1.5.5.3.3.1.1.1.2" xref="S6.E8.m1.5.5.3.3.1.1.1.2.cmml"><mi id="S6.E8.m1.5.5.3.3.1.1.1.2.2" xref="S6.E8.m1.5.5.3.3.1.1.1.2.2.cmml">z</mi><mi id="S6.E8.m1.5.5.3.3.1.1.1.2.3" xref="S6.E8.m1.5.5.3.3.1.1.1.2.3.cmml">a</mi></msub><mo id="S6.E8.m1.5.5.3.3.1.1.1.1" xref="S6.E8.m1.5.5.3.3.1.1.1.1.cmml">−</mo><msub id="S6.E8.m1.5.5.3.3.1.1.1.3" xref="S6.E8.m1.5.5.3.3.1.1.1.3.cmml"><mi id="S6.E8.m1.5.5.3.3.1.1.1.3.2" xref="S6.E8.m1.5.5.3.3.1.1.1.3.2.cmml">z</mi><mi id="S6.E8.m1.5.5.3.3.1.1.1.3.3" xref="S6.E8.m1.5.5.3.3.1.1.1.3.3.cmml">b</mi></msub></mrow><mo stretchy="false" id="S6.E8.m1.5.5.3.3.1.1.3" xref="S6.E8.m1.5.5.3.3.1.1.1.cmml">)</mo></mrow><mn id="S6.E8.m1.5.5.3.3.3" xref="S6.E8.m1.5.5.3.3.3.cmml">2</mn></msup></mrow></msqrt></mrow><annotation-xml encoding="MathML-Content" id="S6.E8.m1.5b"><apply id="S6.E8.m1.5.6.cmml" xref="S6.E8.m1.5.6"><eq id="S6.E8.m1.5.6.1.cmml" xref="S6.E8.m1.5.6.1"></eq><apply id="S6.E8.m1.5.6.2.cmml" xref="S6.E8.m1.5.6.2"><times id="S6.E8.m1.5.6.2.1.cmml" xref="S6.E8.m1.5.6.2.1"></times><ci id="S6.E8.m1.5.6.2.2.cmml" xref="S6.E8.m1.5.6.2.2">𝑑</ci><apply id="S6.E8.m1.5.6.2.3.cmml" xref="S6.E8.m1.5.6.2.3"><csymbol cd="ambiguous" id="S6.E8.m1.5.6.2.3.1.cmml" xref="S6.E8.m1.5.6.2.3">subscript</csymbol><ci id="S6.E8.m1.5.6.2.3.2.cmml" xref="S6.E8.m1.5.6.2.3.2">𝑒</ci><list id="S6.E8.m1.2.2.2.3.cmml" xref="S6.E8.m1.2.2.2.4"><ci id="S6.E8.m1.1.1.1.1.cmml" xref="S6.E8.m1.1.1.1.1">𝑎</ci><ci id="S6.E8.m1.2.2.2.2.cmml" xref="S6.E8.m1.2.2.2.2">𝑏</ci></list></apply></apply><apply id="S6.E8.m1.5.5.cmml" xref="S6.E8.m1.5.5"><root id="S6.E8.m1.5.5a.cmml" xref="S6.E8.m1.5.5"></root><apply id="S6.E8.m1.5.5.3.cmml" xref="S6.E8.m1.5.5.3"><plus id="S6.E8.m1.5.5.3.4.cmml" xref="S6.E8.m1.5.5.3.4"></plus><apply id="S6.E8.m1.3.3.1.1.cmml" xref="S6.E8.m1.3.3.1.1"><csymbol cd="ambiguous" id="S6.E8.m1.3.3.1.1.2.cmml" xref="S6.E8.m1.3.3.1.1">superscript</csymbol><apply id="S6.E8.m1.3.3.1.1.1.1.1.cmml" xref="S6.E8.m1.3.3.1.1.1.1"><minus id="S6.E8.m1.3.3.1.1.1.1.1.1.cmml" xref="S6.E8.m1.3.3.1.1.1.1.1.1"></minus><apply id="S6.E8.m1.3.3.1.1.1.1.1.2.cmml" xref="S6.E8.m1.3.3.1.1.1.1.1.2"><csymbol cd="ambiguous" id="S6.E8.m1.3.3.1.1.1.1.1.2.1.cmml" xref="S6.E8.m1.3.3.1.1.1.1.1.2">subscript</csymbol><ci id="S6.E8.m1.3.3.1.1.1.1.1.2.2.cmml" xref="S6.E8.m1.3.3.1.1.1.1.1.2.2">𝑥</ci><ci id="S6.E8.m1.3.3.1.1.1.1.1.2.3.cmml" xref="S6.E8.m1.3.3.1.1.1.1.1.2.3">𝑎</ci></apply><apply id="S6.E8.m1.3.3.1.1.1.1.1.3.cmml" xref="S6.E8.m1.3.3.1.1.1.1.1.3"><csymbol cd="ambiguous" id="S6.E8.m1.3.3.1.1.1.1.1.3.1.cmml" xref="S6.E8.m1.3.3.1.1.1.1.1.3">subscript</csymbol><ci id="S6.E8.m1.3.3.1.1.1.1.1.3.2.cmml" xref="S6.E8.m1.3.3.1.1.1.1.1.3.2">𝑥</ci><ci id="S6.E8.m1.3.3.1.1.1.1.1.3.3.cmml" xref="S6.E8.m1.3.3.1.1.1.1.1.3.3">𝑏</ci></apply></apply><cn type="integer" id="S6.E8.m1.3.3.1.1.3.cmml" xref="S6.E8.m1.3.3.1.1.3">2</cn></apply><apply id="S6.E8.m1.4.4.2.2.cmml" xref="S6.E8.m1.4.4.2.2"><csymbol cd="ambiguous" id="S6.E8.m1.4.4.2.2.2.cmml" xref="S6.E8.m1.4.4.2.2">superscript</csymbol><apply id="S6.E8.m1.4.4.2.2.1.1.1.cmml" xref="S6.E8.m1.4.4.2.2.1.1"><minus id="S6.E8.m1.4.4.2.2.1.1.1.1.cmml" xref="S6.E8.m1.4.4.2.2.1.1.1.1"></minus><apply id="S6.E8.m1.4.4.2.2.1.1.1.2.cmml" xref="S6.E8.m1.4.4.2.2.1.1.1.2"><csymbol cd="ambiguous" id="S6.E8.m1.4.4.2.2.1.1.1.2.1.cmml" xref="S6.E8.m1.4.4.2.2.1.1.1.2">subscript</csymbol><ci id="S6.E8.m1.4.4.2.2.1.1.1.2.2.cmml" xref="S6.E8.m1.4.4.2.2.1.1.1.2.2">𝑦</ci><ci id="S6.E8.m1.4.4.2.2.1.1.1.2.3.cmml" xref="S6.E8.m1.4.4.2.2.1.1.1.2.3">𝑎</ci></apply><apply id="S6.E8.m1.4.4.2.2.1.1.1.3.cmml" xref="S6.E8.m1.4.4.2.2.1.1.1.3"><csymbol cd="ambiguous" id="S6.E8.m1.4.4.2.2.1.1.1.3.1.cmml" xref="S6.E8.m1.4.4.2.2.1.1.1.3">subscript</csymbol><ci id="S6.E8.m1.4.4.2.2.1.1.1.3.2.cmml" xref="S6.E8.m1.4.4.2.2.1.1.1.3.2">𝑦</ci><ci id="S6.E8.m1.4.4.2.2.1.1.1.3.3.cmml" xref="S6.E8.m1.4.4.2.2.1.1.1.3.3">𝑏</ci></apply></apply><cn type="integer" id="S6.E8.m1.4.4.2.2.3.cmml" xref="S6.E8.m1.4.4.2.2.3">2</cn></apply><apply id="S6.E8.m1.5.5.3.3.cmml" xref="S6.E8.m1.5.5.3.3"><csymbol cd="ambiguous" id="S6.E8.m1.5.5.3.3.2.cmml" xref="S6.E8.m1.5.5.3.3">superscript</csymbol><apply id="S6.E8.m1.5.5.3.3.1.1.1.cmml" xref="S6.E8.m1.5.5.3.3.1.1"><minus id="S6.E8.m1.5.5.3.3.1.1.1.1.cmml" xref="S6.E8.m1.5.5.3.3.1.1.1.1"></minus><apply id="S6.E8.m1.5.5.3.3.1.1.1.2.cmml" xref="S6.E8.m1.5.5.3.3.1.1.1.2"><csymbol cd="ambiguous" id="S6.E8.m1.5.5.3.3.1.1.1.2.1.cmml" xref="S6.E8.m1.5.5.3.3.1.1.1.2">subscript</csymbol><ci id="S6.E8.m1.5.5.3.3.1.1.1.2.2.cmml" xref="S6.E8.m1.5.5.3.3.1.1.1.2.2">𝑧</ci><ci id="S6.E8.m1.5.5.3.3.1.1.1.2.3.cmml" xref="S6.E8.m1.5.5.3.3.1.1.1.2.3">𝑎</ci></apply><apply id="S6.E8.m1.5.5.3.3.1.1.1.3.cmml" xref="S6.E8.m1.5.5.3.3.1.1.1.3"><csymbol cd="ambiguous" id="S6.E8.m1.5.5.3.3.1.1.1.3.1.cmml" xref="S6.E8.m1.5.5.3.3.1.1.1.3">subscript</csymbol><ci id="S6.E8.m1.5.5.3.3.1.1.1.3.2.cmml" xref="S6.E8.m1.5.5.3.3.1.1.1.3.2">𝑧</ci><ci id="S6.E8.m1.5.5.3.3.1.1.1.3.3.cmml" xref="S6.E8.m1.5.5.3.3.1.1.1.3.3">𝑏</ci></apply></apply><cn type="integer" id="S6.E8.m1.5.5.3.3.3.cmml" xref="S6.E8.m1.5.5.3.3.3">2</cn></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.E8.m1.5c">de_{a,b}=\sqrt{(x_{a}-x_{b})^{2}+(y_{a}-y_{b})^{2}+(z_{a}-z_{b})^{2}}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(8)</span></td>
</tr></tbody>
</table>
<br class="ltx_break">
<p id="S6.SS3.SSS4.p3.3" class="ltx_p">The degree of overlap between aligned residues in the structural alignment was calculated through the relative distance <math id="S6.SS3.SSS4.p3.1.m1.1" class="ltx_Math" alttext="di_{j}" display="inline"><semantics id="S6.SS3.SSS4.p3.1.m1.1a"><mrow id="S6.SS3.SSS4.p3.1.m1.1.1" xref="S6.SS3.SSS4.p3.1.m1.1.1.cmml"><mi id="S6.SS3.SSS4.p3.1.m1.1.1.2" xref="S6.SS3.SSS4.p3.1.m1.1.1.2.cmml">d</mi><mo lspace="0em" rspace="0em" id="S6.SS3.SSS4.p3.1.m1.1.1.1" xref="S6.SS3.SSS4.p3.1.m1.1.1.1.cmml">​</mo><msub id="S6.SS3.SSS4.p3.1.m1.1.1.3" xref="S6.SS3.SSS4.p3.1.m1.1.1.3.cmml"><mi id="S6.SS3.SSS4.p3.1.m1.1.1.3.2" xref="S6.SS3.SSS4.p3.1.m1.1.1.3.2.cmml">i</mi><mi id="S6.SS3.SSS4.p3.1.m1.1.1.3.3" xref="S6.SS3.SSS4.p3.1.m1.1.1.3.3.cmml">j</mi></msub></mrow><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS4.p3.1.m1.1b"><apply id="S6.SS3.SSS4.p3.1.m1.1.1.cmml" xref="S6.SS3.SSS4.p3.1.m1.1.1"><times id="S6.SS3.SSS4.p3.1.m1.1.1.1.cmml" xref="S6.SS3.SSS4.p3.1.m1.1.1.1"></times><ci id="S6.SS3.SSS4.p3.1.m1.1.1.2.cmml" xref="S6.SS3.SSS4.p3.1.m1.1.1.2">𝑑</ci><apply id="S6.SS3.SSS4.p3.1.m1.1.1.3.cmml" xref="S6.SS3.SSS4.p3.1.m1.1.1.3"><csymbol cd="ambiguous" id="S6.SS3.SSS4.p3.1.m1.1.1.3.1.cmml" xref="S6.SS3.SSS4.p3.1.m1.1.1.3">subscript</csymbol><ci id="S6.SS3.SSS4.p3.1.m1.1.1.3.2.cmml" xref="S6.SS3.SSS4.p3.1.m1.1.1.3.2">𝑖</ci><ci id="S6.SS3.SSS4.p3.1.m1.1.1.3.3.cmml" xref="S6.SS3.SSS4.p3.1.m1.1.1.3.3">𝑗</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS4.p3.1.m1.1c">di_{j}</annotation></semantics></math>, equation <a href="#S6.E9" title="In 6.3.4 Homologuous Core Structure ‣ 6.3 The 𝑀_𝑠 structural weight matrices ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">9</span></a>. This distance can be found through the average distance among the amino-acid in the position <math id="S6.SS3.SSS4.p3.2.m2.1" class="ltx_Math" alttext="ij" display="inline"><semantics id="S6.SS3.SSS4.p3.2.m2.1a"><mrow id="S6.SS3.SSS4.p3.2.m2.1.1" xref="S6.SS3.SSS4.p3.2.m2.1.1.cmml"><mi id="S6.SS3.SSS4.p3.2.m2.1.1.2" xref="S6.SS3.SSS4.p3.2.m2.1.1.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS3.SSS4.p3.2.m2.1.1.1" xref="S6.SS3.SSS4.p3.2.m2.1.1.1.cmml">​</mo><mi id="S6.SS3.SSS4.p3.2.m2.1.1.3" xref="S6.SS3.SSS4.p3.2.m2.1.1.3.cmml">j</mi></mrow><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS4.p3.2.m2.1b"><apply id="S6.SS3.SSS4.p3.2.m2.1.1.cmml" xref="S6.SS3.SSS4.p3.2.m2.1.1"><times id="S6.SS3.SSS4.p3.2.m2.1.1.1.cmml" xref="S6.SS3.SSS4.p3.2.m2.1.1.1"></times><ci id="S6.SS3.SSS4.p3.2.m2.1.1.2.cmml" xref="S6.SS3.SSS4.p3.2.m2.1.1.2">𝑖</ci><ci id="S6.SS3.SSS4.p3.2.m2.1.1.3.cmml" xref="S6.SS3.SSS4.p3.2.m2.1.1.3">𝑗</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS4.p3.2.m2.1c">ij</annotation></semantics></math> and other amino-acids in the <math id="S6.SS3.SSS4.p3.3.m3.1" class="ltx_Math" alttext="j" display="inline"><semantics id="S6.SS3.SSS4.p3.3.m3.1a"><mi id="S6.SS3.SSS4.p3.3.m3.1.1" xref="S6.SS3.SSS4.p3.3.m3.1.1.cmml">j</mi><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS4.p3.3.m3.1b"><ci id="S6.SS3.SSS4.p3.3.m3.1.1.cmml" xref="S6.SS3.SSS4.p3.3.m3.1.1">𝑗</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS4.p3.3.m3.1c">j</annotation></semantics></math> column of alignment.</p>
</div>
<div id="S6.SS3.SSS4.p4" class="ltx_para">
<table id="S6.E9" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S6.E9.m1.5" class="ltx_Math" alttext="di_{j}=\dfrac{\sum_{b=j}^{n-1}de_{(i,b),(i,b+1)}}{n-1}" display="block"><semantics id="S6.E9.m1.5a"><mrow id="S6.E9.m1.5.6" xref="S6.E9.m1.5.6.cmml"><mrow id="S6.E9.m1.5.6.2" xref="S6.E9.m1.5.6.2.cmml"><mi id="S6.E9.m1.5.6.2.2" xref="S6.E9.m1.5.6.2.2.cmml">d</mi><mo lspace="0em" rspace="0em" id="S6.E9.m1.5.6.2.1" xref="S6.E9.m1.5.6.2.1.cmml">​</mo><msub id="S6.E9.m1.5.6.2.3" xref="S6.E9.m1.5.6.2.3.cmml"><mi id="S6.E9.m1.5.6.2.3.2" xref="S6.E9.m1.5.6.2.3.2.cmml">i</mi><mi id="S6.E9.m1.5.6.2.3.3" xref="S6.E9.m1.5.6.2.3.3.cmml">j</mi></msub></mrow><mo id="S6.E9.m1.5.6.1" xref="S6.E9.m1.5.6.1.cmml">=</mo><mfrac id="S6.E9.m1.5.5" xref="S6.E9.m1.5.5.cmml"><mrow id="S6.E9.m1.5.5.5" xref="S6.E9.m1.5.5.5.cmml"><msubsup id="S6.E9.m1.5.5.5.6" xref="S6.E9.m1.5.5.5.6.cmml"><mo id="S6.E9.m1.5.5.5.6.2.2" xref="S6.E9.m1.5.5.5.6.2.2.cmml">∑</mo><mrow id="S6.E9.m1.5.5.5.6.2.3" xref="S6.E9.m1.5.5.5.6.2.3.cmml"><mi id="S6.E9.m1.5.5.5.6.2.3.2" xref="S6.E9.m1.5.5.5.6.2.3.2.cmml">b</mi><mo id="S6.E9.m1.5.5.5.6.2.3.1" xref="S6.E9.m1.5.5.5.6.2.3.1.cmml">=</mo><mi id="S6.E9.m1.5.5.5.6.2.3.3" xref="S6.E9.m1.5.5.5.6.2.3.3.cmml">j</mi></mrow><mrow id="S6.E9.m1.5.5.5.6.3" xref="S6.E9.m1.5.5.5.6.3.cmml"><mi id="S6.E9.m1.5.5.5.6.3.2" xref="S6.E9.m1.5.5.5.6.3.2.cmml">n</mi><mo id="S6.E9.m1.5.5.5.6.3.1" xref="S6.E9.m1.5.5.5.6.3.1.cmml">−</mo><mn id="S6.E9.m1.5.5.5.6.3.3" xref="S6.E9.m1.5.5.5.6.3.3.cmml">1</mn></mrow></msubsup><mrow id="S6.E9.m1.5.5.5.7" xref="S6.E9.m1.5.5.5.7.cmml"><mi id="S6.E9.m1.5.5.5.7.2" xref="S6.E9.m1.5.5.5.7.2.cmml">d</mi><mo lspace="0em" rspace="0em" id="S6.E9.m1.5.5.5.7.1" xref="S6.E9.m1.5.5.5.7.1.cmml">​</mo><msub id="S6.E9.m1.5.5.5.7.3" xref="S6.E9.m1.5.5.5.7.3.cmml"><mi id="S6.E9.m1.5.5.5.7.3.2" xref="S6.E9.m1.5.5.5.7.3.2.cmml">e</mi><mrow id="S6.E9.m1.5.5.5.5.5.5" xref="S6.E9.m1.5.5.5.5.5.6.cmml"><mrow id="S6.E9.m1.4.4.4.4.4.4.1.2" xref="S6.E9.m1.4.4.4.4.4.4.1.1.cmml"><mo stretchy="false" id="S6.E9.m1.4.4.4.4.4.4.1.2.1" xref="S6.E9.m1.4.4.4.4.4.4.1.1.cmml">(</mo><mi id="S6.E9.m1.1.1.1.1.1.1" xref="S6.E9.m1.1.1.1.1.1.1.cmml">i</mi><mo id="S6.E9.m1.4.4.4.4.4.4.1.2.2" xref="S6.E9.m1.4.4.4.4.4.4.1.1.cmml">,</mo><mi id="S6.E9.m1.2.2.2.2.2.2" xref="S6.E9.m1.2.2.2.2.2.2.cmml">b</mi><mo stretchy="false" id="S6.E9.m1.4.4.4.4.4.4.1.2.3" xref="S6.E9.m1.4.4.4.4.4.4.1.1.cmml">)</mo></mrow><mo id="S6.E9.m1.5.5.5.5.5.5.3" xref="S6.E9.m1.5.5.5.5.5.6.cmml">,</mo><mrow id="S6.E9.m1.5.5.5.5.5.5.2.1" xref="S6.E9.m1.5.5.5.5.5.5.2.2.cmml"><mo stretchy="false" id="S6.E9.m1.5.5.5.5.5.5.2.1.2" xref="S6.E9.m1.5.5.5.5.5.5.2.2.cmml">(</mo><mi id="S6.E9.m1.3.3.3.3.3.3" xref="S6.E9.m1.3.3.3.3.3.3.cmml">i</mi><mo id="S6.E9.m1.5.5.5.5.5.5.2.1.3" xref="S6.E9.m1.5.5.5.5.5.5.2.2.cmml">,</mo><mrow id="S6.E9.m1.5.5.5.5.5.5.2.1.1" xref="S6.E9.m1.5.5.5.5.5.5.2.1.1.cmml"><mi id="S6.E9.m1.5.5.5.5.5.5.2.1.1.2" xref="S6.E9.m1.5.5.5.5.5.5.2.1.1.2.cmml">b</mi><mo id="S6.E9.m1.5.5.5.5.5.5.2.1.1.1" xref="S6.E9.m1.5.5.5.5.5.5.2.1.1.1.cmml">+</mo><mn id="S6.E9.m1.5.5.5.5.5.5.2.1.1.3" xref="S6.E9.m1.5.5.5.5.5.5.2.1.1.3.cmml">1</mn></mrow><mo stretchy="false" id="S6.E9.m1.5.5.5.5.5.5.2.1.4" xref="S6.E9.m1.5.5.5.5.5.5.2.2.cmml">)</mo></mrow></mrow></msub></mrow></mrow><mrow id="S6.E9.m1.5.5.7" xref="S6.E9.m1.5.5.7.cmml"><mi id="S6.E9.m1.5.5.7.2" xref="S6.E9.m1.5.5.7.2.cmml">n</mi><mo id="S6.E9.m1.5.5.7.1" xref="S6.E9.m1.5.5.7.1.cmml">−</mo><mn id="S6.E9.m1.5.5.7.3" xref="S6.E9.m1.5.5.7.3.cmml">1</mn></mrow></mfrac></mrow><annotation-xml encoding="MathML-Content" id="S6.E9.m1.5b"><apply id="S6.E9.m1.5.6.cmml" xref="S6.E9.m1.5.6"><eq id="S6.E9.m1.5.6.1.cmml" xref="S6.E9.m1.5.6.1"></eq><apply id="S6.E9.m1.5.6.2.cmml" xref="S6.E9.m1.5.6.2"><times id="S6.E9.m1.5.6.2.1.cmml" xref="S6.E9.m1.5.6.2.1"></times><ci id="S6.E9.m1.5.6.2.2.cmml" xref="S6.E9.m1.5.6.2.2">𝑑</ci><apply id="S6.E9.m1.5.6.2.3.cmml" xref="S6.E9.m1.5.6.2.3"><csymbol cd="ambiguous" id="S6.E9.m1.5.6.2.3.1.cmml" xref="S6.E9.m1.5.6.2.3">subscript</csymbol><ci id="S6.E9.m1.5.6.2.3.2.cmml" xref="S6.E9.m1.5.6.2.3.2">𝑖</ci><ci id="S6.E9.m1.5.6.2.3.3.cmml" xref="S6.E9.m1.5.6.2.3.3">𝑗</ci></apply></apply><apply id="S6.E9.m1.5.5.cmml" xref="S6.E9.m1.5.5"><divide id="S6.E9.m1.5.5.6.cmml" xref="S6.E9.m1.5.5"></divide><apply id="S6.E9.m1.5.5.5.cmml" xref="S6.E9.m1.5.5.5"><apply id="S6.E9.m1.5.5.5.6.cmml" xref="S6.E9.m1.5.5.5.6"><csymbol cd="ambiguous" id="S6.E9.m1.5.5.5.6.1.cmml" xref="S6.E9.m1.5.5.5.6">superscript</csymbol><apply id="S6.E9.m1.5.5.5.6.2.cmml" xref="S6.E9.m1.5.5.5.6"><csymbol cd="ambiguous" id="S6.E9.m1.5.5.5.6.2.1.cmml" xref="S6.E9.m1.5.5.5.6">subscript</csymbol><sum id="S6.E9.m1.5.5.5.6.2.2.cmml" xref="S6.E9.m1.5.5.5.6.2.2"></sum><apply id="S6.E9.m1.5.5.5.6.2.3.cmml" xref="S6.E9.m1.5.5.5.6.2.3"><eq id="S6.E9.m1.5.5.5.6.2.3.1.cmml" xref="S6.E9.m1.5.5.5.6.2.3.1"></eq><ci id="S6.E9.m1.5.5.5.6.2.3.2.cmml" xref="S6.E9.m1.5.5.5.6.2.3.2">𝑏</ci><ci id="S6.E9.m1.5.5.5.6.2.3.3.cmml" xref="S6.E9.m1.5.5.5.6.2.3.3">𝑗</ci></apply></apply><apply id="S6.E9.m1.5.5.5.6.3.cmml" xref="S6.E9.m1.5.5.5.6.3"><minus id="S6.E9.m1.5.5.5.6.3.1.cmml" xref="S6.E9.m1.5.5.5.6.3.1"></minus><ci id="S6.E9.m1.5.5.5.6.3.2.cmml" xref="S6.E9.m1.5.5.5.6.3.2">𝑛</ci><cn type="integer" id="S6.E9.m1.5.5.5.6.3.3.cmml" xref="S6.E9.m1.5.5.5.6.3.3">1</cn></apply></apply><apply id="S6.E9.m1.5.5.5.7.cmml" xref="S6.E9.m1.5.5.5.7"><times id="S6.E9.m1.5.5.5.7.1.cmml" xref="S6.E9.m1.5.5.5.7.1"></times><ci id="S6.E9.m1.5.5.5.7.2.cmml" xref="S6.E9.m1.5.5.5.7.2">𝑑</ci><apply id="S6.E9.m1.5.5.5.7.3.cmml" xref="S6.E9.m1.5.5.5.7.3"><csymbol cd="ambiguous" id="S6.E9.m1.5.5.5.7.3.1.cmml" xref="S6.E9.m1.5.5.5.7.3">subscript</csymbol><ci id="S6.E9.m1.5.5.5.7.3.2.cmml" xref="S6.E9.m1.5.5.5.7.3.2">𝑒</ci><list id="S6.E9.m1.5.5.5.5.5.6.cmml" xref="S6.E9.m1.5.5.5.5.5.5"><interval closure="open" id="S6.E9.m1.4.4.4.4.4.4.1.1.cmml" xref="S6.E9.m1.4.4.4.4.4.4.1.2"><ci id="S6.E9.m1.1.1.1.1.1.1.cmml" xref="S6.E9.m1.1.1.1.1.1.1">𝑖</ci><ci id="S6.E9.m1.2.2.2.2.2.2.cmml" xref="S6.E9.m1.2.2.2.2.2.2">𝑏</ci></interval><interval closure="open" id="S6.E9.m1.5.5.5.5.5.5.2.2.cmml" xref="S6.E9.m1.5.5.5.5.5.5.2.1"><ci id="S6.E9.m1.3.3.3.3.3.3.cmml" xref="S6.E9.m1.3.3.3.3.3.3">𝑖</ci><apply id="S6.E9.m1.5.5.5.5.5.5.2.1.1.cmml" xref="S6.E9.m1.5.5.5.5.5.5.2.1.1"><plus id="S6.E9.m1.5.5.5.5.5.5.2.1.1.1.cmml" xref="S6.E9.m1.5.5.5.5.5.5.2.1.1.1"></plus><ci id="S6.E9.m1.5.5.5.5.5.5.2.1.1.2.cmml" xref="S6.E9.m1.5.5.5.5.5.5.2.1.1.2">𝑏</ci><cn type="integer" id="S6.E9.m1.5.5.5.5.5.5.2.1.1.3.cmml" xref="S6.E9.m1.5.5.5.5.5.5.2.1.1.3">1</cn></apply></interval></list></apply></apply></apply><apply id="S6.E9.m1.5.5.7.cmml" xref="S6.E9.m1.5.5.7"><minus id="S6.E9.m1.5.5.7.1.cmml" xref="S6.E9.m1.5.5.7.1"></minus><ci id="S6.E9.m1.5.5.7.2.cmml" xref="S6.E9.m1.5.5.7.2">𝑛</ci><cn type="integer" id="S6.E9.m1.5.5.7.3.cmml" xref="S6.E9.m1.5.5.7.3">1</cn></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.E9.m1.5c">di_{j}=\dfrac{\sum_{b=j}^{n-1}de_{(i,b),(i,b+1)}}{n-1}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(9)</span></td>
</tr></tbody>
</table>
<br class="ltx_break">
<p id="S6.SS3.SSS4.p4.3" class="ltx_p">Finally, the relative distance was normalized according to <a href="#S6.E10" title="In 6.3.4 Homologuous Core Structure ‣ 6.3 The 𝑀_𝑠 structural weight matrices ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">10</span></a>, and it was used to determine the degree of overlap of each residue. These measures were normalized by using the equation <a href="#S6.E10" title="In 6.3.4 Homologuous Core Structure ‣ 6.3 The 𝑀_𝑠 structural weight matrices ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">10</span></a>, where <math id="S6.SS3.SSS4.p4.1.m1.1" class="ltx_Math" alttext="d_{min}" display="inline"><semantics id="S6.SS3.SSS4.p4.1.m1.1a"><msub id="S6.SS3.SSS4.p4.1.m1.1.1" xref="S6.SS3.SSS4.p4.1.m1.1.1.cmml"><mi id="S6.SS3.SSS4.p4.1.m1.1.1.2" xref="S6.SS3.SSS4.p4.1.m1.1.1.2.cmml">d</mi><mrow id="S6.SS3.SSS4.p4.1.m1.1.1.3" xref="S6.SS3.SSS4.p4.1.m1.1.1.3.cmml"><mi id="S6.SS3.SSS4.p4.1.m1.1.1.3.2" xref="S6.SS3.SSS4.p4.1.m1.1.1.3.2.cmml">m</mi><mo lspace="0em" rspace="0em" id="S6.SS3.SSS4.p4.1.m1.1.1.3.1" xref="S6.SS3.SSS4.p4.1.m1.1.1.3.1.cmml">​</mo><mi id="S6.SS3.SSS4.p4.1.m1.1.1.3.3" xref="S6.SS3.SSS4.p4.1.m1.1.1.3.3.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS3.SSS4.p4.1.m1.1.1.3.1a" xref="S6.SS3.SSS4.p4.1.m1.1.1.3.1.cmml">​</mo><mi id="S6.SS3.SSS4.p4.1.m1.1.1.3.4" xref="S6.SS3.SSS4.p4.1.m1.1.1.3.4.cmml">n</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS4.p4.1.m1.1b"><apply id="S6.SS3.SSS4.p4.1.m1.1.1.cmml" xref="S6.SS3.SSS4.p4.1.m1.1.1"><csymbol cd="ambiguous" id="S6.SS3.SSS4.p4.1.m1.1.1.1.cmml" xref="S6.SS3.SSS4.p4.1.m1.1.1">subscript</csymbol><ci id="S6.SS3.SSS4.p4.1.m1.1.1.2.cmml" xref="S6.SS3.SSS4.p4.1.m1.1.1.2">𝑑</ci><apply id="S6.SS3.SSS4.p4.1.m1.1.1.3.cmml" xref="S6.SS3.SSS4.p4.1.m1.1.1.3"><times id="S6.SS3.SSS4.p4.1.m1.1.1.3.1.cmml" xref="S6.SS3.SSS4.p4.1.m1.1.1.3.1"></times><ci id="S6.SS3.SSS4.p4.1.m1.1.1.3.2.cmml" xref="S6.SS3.SSS4.p4.1.m1.1.1.3.2">𝑚</ci><ci id="S6.SS3.SSS4.p4.1.m1.1.1.3.3.cmml" xref="S6.SS3.SSS4.p4.1.m1.1.1.3.3">𝑖</ci><ci id="S6.SS3.SSS4.p4.1.m1.1.1.3.4.cmml" xref="S6.SS3.SSS4.p4.1.m1.1.1.3.4">𝑛</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS4.p4.1.m1.1c">d_{min}</annotation></semantics></math> is the minimal distance and <math id="S6.SS3.SSS4.p4.2.m2.1" class="ltx_Math" alttext="O_{max_{i}}" display="inline"><semantics id="S6.SS3.SSS4.p4.2.m2.1a"><msub id="S6.SS3.SSS4.p4.2.m2.1.1" xref="S6.SS3.SSS4.p4.2.m2.1.1.cmml"><mi id="S6.SS3.SSS4.p4.2.m2.1.1.2" xref="S6.SS3.SSS4.p4.2.m2.1.1.2.cmml">O</mi><mrow id="S6.SS3.SSS4.p4.2.m2.1.1.3" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.cmml"><mi id="S6.SS3.SSS4.p4.2.m2.1.1.3.2" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.2.cmml">m</mi><mo lspace="0em" rspace="0em" id="S6.SS3.SSS4.p4.2.m2.1.1.3.1" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.1.cmml">​</mo><mi id="S6.SS3.SSS4.p4.2.m2.1.1.3.3" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.3.cmml">a</mi><mo lspace="0em" rspace="0em" id="S6.SS3.SSS4.p4.2.m2.1.1.3.1a" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.1.cmml">​</mo><msub id="S6.SS3.SSS4.p4.2.m2.1.1.3.4" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.4.cmml"><mi id="S6.SS3.SSS4.p4.2.m2.1.1.3.4.2" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.4.2.cmml">x</mi><mi id="S6.SS3.SSS4.p4.2.m2.1.1.3.4.3" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.4.3.cmml">i</mi></msub></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS4.p4.2.m2.1b"><apply id="S6.SS3.SSS4.p4.2.m2.1.1.cmml" xref="S6.SS3.SSS4.p4.2.m2.1.1"><csymbol cd="ambiguous" id="S6.SS3.SSS4.p4.2.m2.1.1.1.cmml" xref="S6.SS3.SSS4.p4.2.m2.1.1">subscript</csymbol><ci id="S6.SS3.SSS4.p4.2.m2.1.1.2.cmml" xref="S6.SS3.SSS4.p4.2.m2.1.1.2">𝑂</ci><apply id="S6.SS3.SSS4.p4.2.m2.1.1.3.cmml" xref="S6.SS3.SSS4.p4.2.m2.1.1.3"><times id="S6.SS3.SSS4.p4.2.m2.1.1.3.1.cmml" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.1"></times><ci id="S6.SS3.SSS4.p4.2.m2.1.1.3.2.cmml" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.2">𝑚</ci><ci id="S6.SS3.SSS4.p4.2.m2.1.1.3.3.cmml" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.3">𝑎</ci><apply id="S6.SS3.SSS4.p4.2.m2.1.1.3.4.cmml" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.4"><csymbol cd="ambiguous" id="S6.SS3.SSS4.p4.2.m2.1.1.3.4.1.cmml" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.4">subscript</csymbol><ci id="S6.SS3.SSS4.p4.2.m2.1.1.3.4.2.cmml" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.4.2">𝑥</ci><ci id="S6.SS3.SSS4.p4.2.m2.1.1.3.4.3.cmml" xref="S6.SS3.SSS4.p4.2.m2.1.1.3.4.3">𝑖</ci></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS4.p4.2.m2.1c">O_{max_{i}}</annotation></semantics></math> is the maximal Ooi measure for protein <math id="S6.SS3.SSS4.p4.3.m3.1" class="ltx_Math" alttext="i" display="inline"><semantics id="S6.SS3.SSS4.p4.3.m3.1a"><mi id="S6.SS3.SSS4.p4.3.m3.1.1" xref="S6.SS3.SSS4.p4.3.m3.1.1.cmml">i</mi><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS4.p4.3.m3.1b"><ci id="S6.SS3.SSS4.p4.3.m3.1.1.cmml" xref="S6.SS3.SSS4.p4.3.m3.1.1">𝑖</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS4.p4.3.m3.1c">i</annotation></semantics></math>.

<br class="ltx_break"></p>
<table id="S6.E10" class="ltx_equation ltx_eqn_table">

<tbody><tr class="ltx_equation ltx_eqn_row ltx_align_baseline">
<td class="ltx_eqn_cell ltx_eqn_center_padleft"></td>
<td class="ltx_eqn_cell ltx_align_center"><math id="S6.E10.m1.1" class="ltx_Math" alttext="m_{ij}=\dfrac{d_{min}*O_{max_{i}}}{di_{j}}" display="block"><semantics id="S6.E10.m1.1a"><mrow id="S6.E10.m1.1.1" xref="S6.E10.m1.1.1.cmml"><msub id="S6.E10.m1.1.1.2" xref="S6.E10.m1.1.1.2.cmml"><mi id="S6.E10.m1.1.1.2.2" xref="S6.E10.m1.1.1.2.2.cmml">m</mi><mrow id="S6.E10.m1.1.1.2.3" xref="S6.E10.m1.1.1.2.3.cmml"><mi id="S6.E10.m1.1.1.2.3.2" xref="S6.E10.m1.1.1.2.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.E10.m1.1.1.2.3.1" xref="S6.E10.m1.1.1.2.3.1.cmml">​</mo><mi id="S6.E10.m1.1.1.2.3.3" xref="S6.E10.m1.1.1.2.3.3.cmml">j</mi></mrow></msub><mo id="S6.E10.m1.1.1.1" xref="S6.E10.m1.1.1.1.cmml">=</mo><mfrac id="S6.E10.m1.1.1.3" xref="S6.E10.m1.1.1.3.cmml"><mrow id="S6.E10.m1.1.1.3.2" xref="S6.E10.m1.1.1.3.2.cmml"><msub id="S6.E10.m1.1.1.3.2.2" xref="S6.E10.m1.1.1.3.2.2.cmml"><mi id="S6.E10.m1.1.1.3.2.2.2" xref="S6.E10.m1.1.1.3.2.2.2.cmml">d</mi><mrow id="S6.E10.m1.1.1.3.2.2.3" xref="S6.E10.m1.1.1.3.2.2.3.cmml"><mi id="S6.E10.m1.1.1.3.2.2.3.2" xref="S6.E10.m1.1.1.3.2.2.3.2.cmml">m</mi><mo lspace="0em" rspace="0em" id="S6.E10.m1.1.1.3.2.2.3.1" xref="S6.E10.m1.1.1.3.2.2.3.1.cmml">​</mo><mi id="S6.E10.m1.1.1.3.2.2.3.3" xref="S6.E10.m1.1.1.3.2.2.3.3.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.E10.m1.1.1.3.2.2.3.1a" xref="S6.E10.m1.1.1.3.2.2.3.1.cmml">​</mo><mi id="S6.E10.m1.1.1.3.2.2.3.4" xref="S6.E10.m1.1.1.3.2.2.3.4.cmml">n</mi></mrow></msub><mo lspace="0.222em" rspace="0.222em" id="S6.E10.m1.1.1.3.2.1" xref="S6.E10.m1.1.1.3.2.1.cmml">∗</mo><msub id="S6.E10.m1.1.1.3.2.3" xref="S6.E10.m1.1.1.3.2.3.cmml"><mi id="S6.E10.m1.1.1.3.2.3.2" xref="S6.E10.m1.1.1.3.2.3.2.cmml">O</mi><mrow id="S6.E10.m1.1.1.3.2.3.3" xref="S6.E10.m1.1.1.3.2.3.3.cmml"><mi id="S6.E10.m1.1.1.3.2.3.3.2" xref="S6.E10.m1.1.1.3.2.3.3.2.cmml">m</mi><mo lspace="0em" rspace="0em" id="S6.E10.m1.1.1.3.2.3.3.1" xref="S6.E10.m1.1.1.3.2.3.3.1.cmml">​</mo><mi id="S6.E10.m1.1.1.3.2.3.3.3" xref="S6.E10.m1.1.1.3.2.3.3.3.cmml">a</mi><mo lspace="0em" rspace="0em" id="S6.E10.m1.1.1.3.2.3.3.1a" xref="S6.E10.m1.1.1.3.2.3.3.1.cmml">​</mo><msub id="S6.E10.m1.1.1.3.2.3.3.4" xref="S6.E10.m1.1.1.3.2.3.3.4.cmml"><mi id="S6.E10.m1.1.1.3.2.3.3.4.2" xref="S6.E10.m1.1.1.3.2.3.3.4.2.cmml">x</mi><mi id="S6.E10.m1.1.1.3.2.3.3.4.3" xref="S6.E10.m1.1.1.3.2.3.3.4.3.cmml">i</mi></msub></mrow></msub></mrow><mrow id="S6.E10.m1.1.1.3.3" xref="S6.E10.m1.1.1.3.3.cmml"><mi id="S6.E10.m1.1.1.3.3.2" xref="S6.E10.m1.1.1.3.3.2.cmml">d</mi><mo lspace="0em" rspace="0em" id="S6.E10.m1.1.1.3.3.1" xref="S6.E10.m1.1.1.3.3.1.cmml">​</mo><msub id="S6.E10.m1.1.1.3.3.3" xref="S6.E10.m1.1.1.3.3.3.cmml"><mi id="S6.E10.m1.1.1.3.3.3.2" xref="S6.E10.m1.1.1.3.3.3.2.cmml">i</mi><mi id="S6.E10.m1.1.1.3.3.3.3" xref="S6.E10.m1.1.1.3.3.3.3.cmml">j</mi></msub></mrow></mfrac></mrow><annotation-xml encoding="MathML-Content" id="S6.E10.m1.1b"><apply id="S6.E10.m1.1.1.cmml" xref="S6.E10.m1.1.1"><eq id="S6.E10.m1.1.1.1.cmml" xref="S6.E10.m1.1.1.1"></eq><apply id="S6.E10.m1.1.1.2.cmml" xref="S6.E10.m1.1.1.2"><csymbol cd="ambiguous" id="S6.E10.m1.1.1.2.1.cmml" xref="S6.E10.m1.1.1.2">subscript</csymbol><ci id="S6.E10.m1.1.1.2.2.cmml" xref="S6.E10.m1.1.1.2.2">𝑚</ci><apply id="S6.E10.m1.1.1.2.3.cmml" xref="S6.E10.m1.1.1.2.3"><times id="S6.E10.m1.1.1.2.3.1.cmml" xref="S6.E10.m1.1.1.2.3.1"></times><ci id="S6.E10.m1.1.1.2.3.2.cmml" xref="S6.E10.m1.1.1.2.3.2">𝑖</ci><ci id="S6.E10.m1.1.1.2.3.3.cmml" xref="S6.E10.m1.1.1.2.3.3">𝑗</ci></apply></apply><apply id="S6.E10.m1.1.1.3.cmml" xref="S6.E10.m1.1.1.3"><divide id="S6.E10.m1.1.1.3.1.cmml" xref="S6.E10.m1.1.1.3"></divide><apply id="S6.E10.m1.1.1.3.2.cmml" xref="S6.E10.m1.1.1.3.2"><times id="S6.E10.m1.1.1.3.2.1.cmml" xref="S6.E10.m1.1.1.3.2.1"></times><apply id="S6.E10.m1.1.1.3.2.2.cmml" xref="S6.E10.m1.1.1.3.2.2"><csymbol cd="ambiguous" id="S6.E10.m1.1.1.3.2.2.1.cmml" xref="S6.E10.m1.1.1.3.2.2">subscript</csymbol><ci id="S6.E10.m1.1.1.3.2.2.2.cmml" xref="S6.E10.m1.1.1.3.2.2.2">𝑑</ci><apply id="S6.E10.m1.1.1.3.2.2.3.cmml" xref="S6.E10.m1.1.1.3.2.2.3"><times id="S6.E10.m1.1.1.3.2.2.3.1.cmml" xref="S6.E10.m1.1.1.3.2.2.3.1"></times><ci id="S6.E10.m1.1.1.3.2.2.3.2.cmml" xref="S6.E10.m1.1.1.3.2.2.3.2">𝑚</ci><ci id="S6.E10.m1.1.1.3.2.2.3.3.cmml" xref="S6.E10.m1.1.1.3.2.2.3.3">𝑖</ci><ci id="S6.E10.m1.1.1.3.2.2.3.4.cmml" xref="S6.E10.m1.1.1.3.2.2.3.4">𝑛</ci></apply></apply><apply id="S6.E10.m1.1.1.3.2.3.cmml" xref="S6.E10.m1.1.1.3.2.3"><csymbol cd="ambiguous" id="S6.E10.m1.1.1.3.2.3.1.cmml" xref="S6.E10.m1.1.1.3.2.3">subscript</csymbol><ci id="S6.E10.m1.1.1.3.2.3.2.cmml" xref="S6.E10.m1.1.1.3.2.3.2">𝑂</ci><apply id="S6.E10.m1.1.1.3.2.3.3.cmml" xref="S6.E10.m1.1.1.3.2.3.3"><times id="S6.E10.m1.1.1.3.2.3.3.1.cmml" xref="S6.E10.m1.1.1.3.2.3.3.1"></times><ci id="S6.E10.m1.1.1.3.2.3.3.2.cmml" xref="S6.E10.m1.1.1.3.2.3.3.2">𝑚</ci><ci id="S6.E10.m1.1.1.3.2.3.3.3.cmml" xref="S6.E10.m1.1.1.3.2.3.3.3">𝑎</ci><apply id="S6.E10.m1.1.1.3.2.3.3.4.cmml" xref="S6.E10.m1.1.1.3.2.3.3.4"><csymbol cd="ambiguous" id="S6.E10.m1.1.1.3.2.3.3.4.1.cmml" xref="S6.E10.m1.1.1.3.2.3.3.4">subscript</csymbol><ci id="S6.E10.m1.1.1.3.2.3.3.4.2.cmml" xref="S6.E10.m1.1.1.3.2.3.3.4.2">𝑥</ci><ci id="S6.E10.m1.1.1.3.2.3.3.4.3.cmml" xref="S6.E10.m1.1.1.3.2.3.3.4.3">𝑖</ci></apply></apply></apply></apply><apply id="S6.E10.m1.1.1.3.3.cmml" xref="S6.E10.m1.1.1.3.3"><times id="S6.E10.m1.1.1.3.3.1.cmml" xref="S6.E10.m1.1.1.3.3.1"></times><ci id="S6.E10.m1.1.1.3.3.2.cmml" xref="S6.E10.m1.1.1.3.3.2">𝑑</ci><apply id="S6.E10.m1.1.1.3.3.3.cmml" xref="S6.E10.m1.1.1.3.3.3"><csymbol cd="ambiguous" id="S6.E10.m1.1.1.3.3.3.1.cmml" xref="S6.E10.m1.1.1.3.3.3">subscript</csymbol><ci id="S6.E10.m1.1.1.3.3.3.2.cmml" xref="S6.E10.m1.1.1.3.3.3.2">𝑖</ci><ci id="S6.E10.m1.1.1.3.3.3.3.cmml" xref="S6.E10.m1.1.1.3.3.3.3">𝑗</ci></apply></apply></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.E10.m1.1c">m_{ij}=\dfrac{d_{min}*O_{max_{i}}}{di_{j}}</annotation></semantics></math></td>
<td class="ltx_eqn_cell ltx_eqn_center_padright"></td>
<td rowspan="1" class="ltx_eqn_cell ltx_eqn_eqno ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation ltx_align_right">(10)</span></td>
</tr></tbody>
</table>
<br class="ltx_break">
<p id="S6.SS3.SSS4.p4.6" class="ltx_p">After this step, we built the <math id="S6.SS3.SSS4.p4.4.m1.1" class="ltx_Math" alttext="M_{s}" display="inline"><semantics id="S6.SS3.SSS4.p4.4.m1.1a"><msub id="S6.SS3.SSS4.p4.4.m1.1.1" xref="S6.SS3.SSS4.p4.4.m1.1.1.cmml"><mi id="S6.SS3.SSS4.p4.4.m1.1.1.2" xref="S6.SS3.SSS4.p4.4.m1.1.1.2.cmml">M</mi><mi id="S6.SS3.SSS4.p4.4.m1.1.1.3" xref="S6.SS3.SSS4.p4.4.m1.1.1.3.cmml">s</mi></msub><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS4.p4.4.m1.1b"><apply id="S6.SS3.SSS4.p4.4.m1.1.1.cmml" xref="S6.SS3.SSS4.p4.4.m1.1.1"><csymbol cd="ambiguous" id="S6.SS3.SSS4.p4.4.m1.1.1.1.cmml" xref="S6.SS3.SSS4.p4.4.m1.1.1">subscript</csymbol><ci id="S6.SS3.SSS4.p4.4.m1.1.1.2.cmml" xref="S6.SS3.SSS4.p4.4.m1.1.1.2">𝑀</ci><ci id="S6.SS3.SSS4.p4.4.m1.1.1.3.cmml" xref="S6.SS3.SSS4.p4.4.m1.1.1.3">𝑠</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS4.p4.4.m1.1c">M_{s}</annotation></semantics></math> matrix, where each <math id="S6.SS3.SSS4.p4.5.m2.1" class="ltx_Math" alttext="m_{ij}" display="inline"><semantics id="S6.SS3.SSS4.p4.5.m2.1a"><msub id="S6.SS3.SSS4.p4.5.m2.1.1" xref="S6.SS3.SSS4.p4.5.m2.1.1.cmml"><mi id="S6.SS3.SSS4.p4.5.m2.1.1.2" xref="S6.SS3.SSS4.p4.5.m2.1.1.2.cmml">m</mi><mrow id="S6.SS3.SSS4.p4.5.m2.1.1.3" xref="S6.SS3.SSS4.p4.5.m2.1.1.3.cmml"><mi id="S6.SS3.SSS4.p4.5.m2.1.1.3.2" xref="S6.SS3.SSS4.p4.5.m2.1.1.3.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS3.SSS4.p4.5.m2.1.1.3.1" xref="S6.SS3.SSS4.p4.5.m2.1.1.3.1.cmml">​</mo><mi id="S6.SS3.SSS4.p4.5.m2.1.1.3.3" xref="S6.SS3.SSS4.p4.5.m2.1.1.3.3.cmml">j</mi></mrow></msub><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS4.p4.5.m2.1b"><apply id="S6.SS3.SSS4.p4.5.m2.1.1.cmml" xref="S6.SS3.SSS4.p4.5.m2.1.1"><csymbol cd="ambiguous" id="S6.SS3.SSS4.p4.5.m2.1.1.1.cmml" xref="S6.SS3.SSS4.p4.5.m2.1.1">subscript</csymbol><ci id="S6.SS3.SSS4.p4.5.m2.1.1.2.cmml" xref="S6.SS3.SSS4.p4.5.m2.1.1.2">𝑚</ci><apply id="S6.SS3.SSS4.p4.5.m2.1.1.3.cmml" xref="S6.SS3.SSS4.p4.5.m2.1.1.3"><times id="S6.SS3.SSS4.p4.5.m2.1.1.3.1.cmml" xref="S6.SS3.SSS4.p4.5.m2.1.1.3.1"></times><ci id="S6.SS3.SSS4.p4.5.m2.1.1.3.2.cmml" xref="S6.SS3.SSS4.p4.5.m2.1.1.3.2">𝑖</ci><ci id="S6.SS3.SSS4.p4.5.m2.1.1.3.3.cmml" xref="S6.SS3.SSS4.p4.5.m2.1.1.3.3">𝑗</ci></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS4.p4.5.m2.1c">m_{ij}</annotation></semantics></math> matrix element corresponds to the relative distance of amino-acids <math id="S6.SS3.SSS4.p4.6.m3.1" class="ltx_Math" alttext="ij" display="inline"><semantics id="S6.SS3.SSS4.p4.6.m3.1a"><mrow id="S6.SS3.SSS4.p4.6.m3.1.1" xref="S6.SS3.SSS4.p4.6.m3.1.1.cmml"><mi id="S6.SS3.SSS4.p4.6.m3.1.1.2" xref="S6.SS3.SSS4.p4.6.m3.1.1.2.cmml">i</mi><mo lspace="0em" rspace="0em" id="S6.SS3.SSS4.p4.6.m3.1.1.1" xref="S6.SS3.SSS4.p4.6.m3.1.1.1.cmml">​</mo><mi id="S6.SS3.SSS4.p4.6.m3.1.1.3" xref="S6.SS3.SSS4.p4.6.m3.1.1.3.cmml">j</mi></mrow><annotation-xml encoding="MathML-Content" id="S6.SS3.SSS4.p4.6.m3.1b"><apply id="S6.SS3.SSS4.p4.6.m3.1.1.cmml" xref="S6.SS3.SSS4.p4.6.m3.1.1"><times id="S6.SS3.SSS4.p4.6.m3.1.1.1.cmml" xref="S6.SS3.SSS4.p4.6.m3.1.1.1"></times><ci id="S6.SS3.SSS4.p4.6.m3.1.1.2.cmml" xref="S6.SS3.SSS4.p4.6.m3.1.1.2">𝑖</ci><ci id="S6.SS3.SSS4.p4.6.m3.1.1.3.cmml" xref="S6.SS3.SSS4.p4.6.m3.1.1.3">𝑗</ci></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS3.SSS4.p4.6.m3.1c">ij</annotation></semantics></math> in the structural alignment. This matrix represents structural weights that were used to build the model <span id="S6.SS3.SSS4.p4.6.1" class="ltx_text ltx_font_italic">pHMM3D</span>, shown in the figure <a href="#S6.F1" title="Figure 1 ‣ 6 Methods ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.

<br class="ltx_break"></p>
</div>
</section>
</section>
<section id="S6.SS4" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">6.4 </span>Library of structural models</h3>

<div id="S6.SS4.p1" class="ltx_para">
<p id="S6.SS4.p1.1" class="ltx_p">In a second step, we join the models built from these matrices to form a library of structural models aiming at building a single model to represent the structural patterns under different aspects. We used the <span id="S6.SS4.p1.1.1" class="ltx_text ltx_font_typewriter">hmmpfam</span> HMMER tool to combine the models together. Library of models have been used in a number of studies, such as <cite class="ltx_cite ltx_citemacro_citep">(Bateman <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib6" title="" class="ltx_ref">2004</a>; Haft <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib22" title="" class="ltx_ref">2003</a>; Gough <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib19" title="" class="ltx_ref">2001</a>)</cite>, and they are known to achieve better results than those achieved by single models.</p>
</div>
</section>
<section id="S6.SS5" class="ltx_subsection">
<h3 class="ltx_title ltx_title_subsection">
<span class="ltx_tag ltx_tag_subsection">6.5 </span>Test Procedure</h3>

<div id="S6.SS5.p1" class="ltx_para">
<p id="S6.SS5.p1.1" class="ltx_p">The main concern of our study is to build pHMMs that can be helpful in remote homology detection. Therefore, our experiments considered proteins with identity below 30%. To do so, we used the SCOP database <cite class="ltx_cite ltx_citemacro_citep">(Andreeva <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib4" title="" class="ltx_ref">2004</a>)</cite>, and more specifically ASTRAL SCOP version 1.67 PDB40 (with 6600 protein sequences). ASTRAL SCOP is particularly interesting for our study because it describes structural and evolutionary relationships among proteins, such that none of the sequences in ASTRAL SCOP present <math id="S6.SS5.p1.1.m1.1" class="ltx_Math" alttext="&gt;40" display="inline"><semantics id="S6.SS5.p1.1.m1.1a"><mrow id="S6.SS5.p1.1.m1.1.1" xref="S6.SS5.p1.1.m1.1.1.cmml"><mi id="S6.SS5.p1.1.m1.1.1.2" xref="S6.SS5.p1.1.m1.1.1.2.cmml"></mi><mo id="S6.SS5.p1.1.m1.1.1.1" xref="S6.SS5.p1.1.m1.1.1.1.cmml">&gt;</mo><mn id="S6.SS5.p1.1.m1.1.1.3" xref="S6.SS5.p1.1.m1.1.1.3.cmml">40</mn></mrow><annotation-xml encoding="MathML-Content" id="S6.SS5.p1.1.m1.1b"><apply id="S6.SS5.p1.1.m1.1.1.cmml" xref="S6.SS5.p1.1.m1.1.1"><gt id="S6.SS5.p1.1.m1.1.1.1.cmml" xref="S6.SS5.p1.1.m1.1.1.1"></gt><csymbol cd="latexml" id="S6.SS5.p1.1.m1.1.1.2.cmml" xref="S6.SS5.p1.1.m1.1.1.2">absent</csymbol><cn type="integer" id="S6.SS5.p1.1.m1.1.1.3.cmml" xref="S6.SS5.p1.1.m1.1.1.3">40</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p1.1.m1.1c">&gt;40</annotation></semantics></math>% sequence identity. Thus, it is an excellent dataset to evaluate the performance of remote homology detection methods and has been widely used to reach this goal <cite class="ltx_cite ltx_citemacro_citep">(Espadaler <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib16" title="" class="ltx_ref">2005</a>; Wistrand <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib37" title="" class="ltx_ref">2005</a>; Hou <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib24" title="" class="ltx_ref">2004</a>; Alexandrov <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib1" title="" class="ltx_ref">2004</a>)</cite>.</p>
</div>
<div id="S6.SS5.p2" class="ltx_para">
<p id="S6.SS5.p2.1" class="ltx_p">SCOP classifies all protein domains of known structure into a hierarchy with four levels: class, fold, super family and family. In our study, we work at the super family level, which gathers families in such a way that a common evolutionary origin is not obvious from sequence identity, but probable from an analysis of structure and from functional features. We believe that this level better represents remote homolog.</p>
</div>
<div id="S6.SS5.p3" class="ltx_para">
<p id="S6.SS5.p3.8" class="ltx_p">Moreover, we used cross-validation <cite class="ltx_cite ltx_citemacro_citep">(Mitchell,, <a href="#bib.bib7" title="" class="ltx_ref">1997</a>)</cite> to compare the different approaches. First, we divided SCOP database by super family level. Next, from ASTRAL PDB40, we chose those super families containing at least three families and at least 20 sequences. We eventually tested 39 super families, as listed in Table 1. This whittled down the number of sequences we used to model building to 1137. Third, we implemented leave-one-family-out cross-validation. For any super family <math id="S6.SS5.p3.1.m1.1" class="ltx_Math" alttext="x" display="inline"><semantics id="S6.SS5.p3.1.m1.1a"><mi id="S6.SS5.p3.1.m1.1.1" xref="S6.SS5.p3.1.m1.1.1.cmml">x</mi><annotation-xml encoding="MathML-Content" id="S6.SS5.p3.1.m1.1b"><ci id="S6.SS5.p3.1.m1.1.1.cmml" xref="S6.SS5.p3.1.m1.1.1">𝑥</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p3.1.m1.1c">x</annotation></semantics></math> having <math id="S6.SS5.p3.2.m2.1" class="ltx_Math" alttext="n" display="inline"><semantics id="S6.SS5.p3.2.m2.1a"><mi id="S6.SS5.p3.2.m2.1.1" xref="S6.SS5.p3.2.m2.1.1.cmml">n</mi><annotation-xml encoding="MathML-Content" id="S6.SS5.p3.2.m2.1b"><ci id="S6.SS5.p3.2.m2.1.1.cmml" xref="S6.SS5.p3.2.m2.1.1">𝑛</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p3.2.m2.1c">n</annotation></semantics></math> families, we built <math id="S6.SS5.p3.3.m3.1" class="ltx_Math" alttext="n" display="inline"><semantics id="S6.SS5.p3.3.m3.1a"><mi id="S6.SS5.p3.3.m3.1.1" xref="S6.SS5.p3.3.m3.1.1.cmml">n</mi><annotation-xml encoding="MathML-Content" id="S6.SS5.p3.3.m3.1b"><ci id="S6.SS5.p3.3.m3.1.1.cmml" xref="S6.SS5.p3.3.m3.1.1">𝑛</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p3.3.m3.1c">n</annotation></semantics></math> profiles so that each profile <math id="S6.SS5.p3.4.m4.1" class="ltx_Math" alttext="P" display="inline"><semantics id="S6.SS5.p3.4.m4.1a"><mi id="S6.SS5.p3.4.m4.1.1" xref="S6.SS5.p3.4.m4.1.1.cmml">P</mi><annotation-xml encoding="MathML-Content" id="S6.SS5.p3.4.m4.1b"><ci id="S6.SS5.p3.4.m4.1.1.cmml" xref="S6.SS5.p3.4.m4.1.1">𝑃</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p3.4.m4.1c">P</annotation></semantics></math> was built from the sequences in the remaining <math id="S6.SS5.p3.5.m5.1" class="ltx_Math" alttext="n-1" display="inline"><semantics id="S6.SS5.p3.5.m5.1a"><mrow id="S6.SS5.p3.5.m5.1.1" xref="S6.SS5.p3.5.m5.1.1.cmml"><mi id="S6.SS5.p3.5.m5.1.1.2" xref="S6.SS5.p3.5.m5.1.1.2.cmml">n</mi><mo id="S6.SS5.p3.5.m5.1.1.1" xref="S6.SS5.p3.5.m5.1.1.1.cmml">−</mo><mn id="S6.SS5.p3.5.m5.1.1.3" xref="S6.SS5.p3.5.m5.1.1.3.cmml">1</mn></mrow><annotation-xml encoding="MathML-Content" id="S6.SS5.p3.5.m5.1b"><apply id="S6.SS5.p3.5.m5.1.1.cmml" xref="S6.SS5.p3.5.m5.1.1"><minus id="S6.SS5.p3.5.m5.1.1.1.cmml" xref="S6.SS5.p3.5.m5.1.1.1"></minus><ci id="S6.SS5.p3.5.m5.1.1.2.cmml" xref="S6.SS5.p3.5.m5.1.1.2">𝑛</ci><cn type="integer" id="S6.SS5.p3.5.m5.1.1.3.cmml" xref="S6.SS5.p3.5.m5.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p3.5.m5.1c">n-1</annotation></semantics></math> families. Thus, the <math id="S6.SS5.p3.6.m6.1" class="ltx_Math" alttext="n-1" display="inline"><semantics id="S6.SS5.p3.6.m6.1a"><mrow id="S6.SS5.p3.6.m6.1.1" xref="S6.SS5.p3.6.m6.1.1.cmml"><mi id="S6.SS5.p3.6.m6.1.1.2" xref="S6.SS5.p3.6.m6.1.1.2.cmml">n</mi><mo id="S6.SS5.p3.6.m6.1.1.1" xref="S6.SS5.p3.6.m6.1.1.1.cmml">−</mo><mn id="S6.SS5.p3.6.m6.1.1.3" xref="S6.SS5.p3.6.m6.1.1.3.cmml">1</mn></mrow><annotation-xml encoding="MathML-Content" id="S6.SS5.p3.6.m6.1b"><apply id="S6.SS5.p3.6.m6.1.1.cmml" xref="S6.SS5.p3.6.m6.1.1"><minus id="S6.SS5.p3.6.m6.1.1.1.cmml" xref="S6.SS5.p3.6.m6.1.1.1"></minus><ci id="S6.SS5.p3.6.m6.1.1.2.cmml" xref="S6.SS5.p3.6.m6.1.1.2">𝑛</ci><cn type="integer" id="S6.SS5.p3.6.m6.1.1.3.cmml" xref="S6.SS5.p3.6.m6.1.1.3">1</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p3.6.m6.1c">n-1</annotation></semantics></math> sequences form the training set for profile <math id="S6.SS5.p3.7.m7.1" class="ltx_Math" alttext="P" display="inline"><semantics id="S6.SS5.p3.7.m7.1a"><mi id="S6.SS5.p3.7.m7.1.1" xref="S6.SS5.p3.7.m7.1.1.cmml">P</mi><annotation-xml encoding="MathML-Content" id="S6.SS5.p3.7.m7.1b"><ci id="S6.SS5.p3.7.m7.1.1.cmml" xref="S6.SS5.p3.7.m7.1.1">𝑃</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p3.7.m7.1c">P</annotation></semantics></math>. The test set for profile <math id="S6.SS5.p3.8.m8.1" class="ltx_Math" alttext="P" display="inline"><semantics id="S6.SS5.p3.8.m8.1a"><mi id="S6.SS5.p3.8.m8.1.1" xref="S6.SS5.p3.8.m8.1.1.cmml">P</mi><annotation-xml encoding="MathML-Content" id="S6.SS5.p3.8.m8.1b"><ci id="S6.SS5.p3.8.m8.1.1.cmml" xref="S6.SS5.p3.8.m8.1.1">𝑃</ci></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p3.8.m8.1c">P</annotation></semantics></math> will be the remaining sequences (test positives) plus all other database sequences (test negatives).</p>
</div>
<figure id="S6.SS5.tab1" class="ltx_table">
<div class="ltx_flex_figure ltx_flex_table">
<div class="ltx_flex_cell ltx_flex_size_1"><span id="S6.SS5.tab1.1" class="ltx_ERROR ltx_figure_panel undefined">\processtable</span></div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<p id="S6.SS5.tab1.2" class="ltx_p ltx_figure_panel">Superfamily SCOP-Ids

<span id="S6.SS5.tab1.2.1" class="ltx_tabular ltx_guessed_headers ltx_align_middle">
<span class="ltx_tbody">
<span id="S6.SS5.tab1.2.1.1.1" class="ltx_tr">
<span id="S6.SS5.tab1.2.1.1.1.1" class="ltx_td ltx_align_left ltx_th ltx_th_row"><span id="S6.SS5.tab1.2.1.1.1.1.1" class="ltx_ERROR undefined">\toprule</span><span id="S6.SS5.tab1.2.1.1.1.1.2" class="ltx_text" style="font-size:70%;">a.1.1.</span></span>
<span id="S6.SS5.tab1.2.1.1.1.2" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.1.1.2.1" class="ltx_text" style="font-size:70%;">a.138.1.</span></span>
<span id="S6.SS5.tab1.2.1.1.1.3" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.1.1.3.1" class="ltx_text" style="font-size:70%;">a.25.1.</span></span>
<span id="S6.SS5.tab1.2.1.1.1.4" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.1.1.4.1" class="ltx_text" style="font-size:70%;">a.26.1.</span></span>
<span id="S6.SS5.tab1.2.1.1.1.5" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.1.1.5.1" class="ltx_text" style="font-size:70%;">a.3.1.</span></span>
<span id="S6.SS5.tab1.2.1.1.1.6" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.1.1.6.1" class="ltx_text" style="font-size:70%;">a.39.1.</span></span>
<span id="S6.SS5.tab1.2.1.1.1.7" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.1.1.7.1" class="ltx_text" style="font-size:70%;">a.4.1.</span></span>
<span id="S6.SS5.tab1.2.1.1.1.8" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.1.1.8.1" class="ltx_text" style="font-size:70%;">b.121.4.</span></span></span>
<span id="S6.SS5.tab1.2.1.2.2" class="ltx_tr">
<span id="S6.SS5.tab1.2.1.2.2.1" class="ltx_td ltx_align_left ltx_th ltx_th_row"><span id="S6.SS5.tab1.2.1.2.2.1.1" class="ltx_text" style="font-size:70%;">b.18.1.</span></span>
<span id="S6.SS5.tab1.2.1.2.2.2" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.2.2.2.1" class="ltx_text" style="font-size:70%;">b.29.1.</span></span>
<span id="S6.SS5.tab1.2.1.2.2.3" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.2.2.3.1" class="ltx_text" style="font-size:70%;">b.36.1.</span></span>
<span id="S6.SS5.tab1.2.1.2.2.4" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.2.2.4.1" class="ltx_text" style="font-size:70%;">b.47.1.</span></span>
<span id="S6.SS5.tab1.2.1.2.2.5" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.2.2.5.1" class="ltx_text" style="font-size:70%;">b.55.1.</span></span>
<span id="S6.SS5.tab1.2.1.2.2.6" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.2.2.6.1" class="ltx_text" style="font-size:70%;">b.60.1.</span></span>
<span id="S6.SS5.tab1.2.1.2.2.7" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.2.2.7.1" class="ltx_text" style="font-size:70%;">b.6.1.</span></span>
<span id="S6.SS5.tab1.2.1.2.2.8" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.2.2.8.1" class="ltx_text" style="font-size:70%;">b.71.1.</span></span></span>
<span id="S6.SS5.tab1.2.1.3.3" class="ltx_tr">
<span id="S6.SS5.tab1.2.1.3.3.1" class="ltx_td ltx_align_left ltx_th ltx_th_row"><span id="S6.SS5.tab1.2.1.3.3.1.1" class="ltx_text" style="font-size:70%;">b.82.1.</span></span>
<span id="S6.SS5.tab1.2.1.3.3.2" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.3.3.2.1" class="ltx_text" style="font-size:70%;">c.1.10.</span></span>
<span id="S6.SS5.tab1.2.1.3.3.3" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.3.3.3.1" class="ltx_text" style="font-size:70%;">c.23.1.</span></span>
<span id="S6.SS5.tab1.2.1.3.3.4" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.3.3.4.1" class="ltx_text" style="font-size:70%;">c.26.1.</span></span>
<span id="S6.SS5.tab1.2.1.3.3.5" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.3.3.5.1" class="ltx_text" style="font-size:70%;">c.36.1.</span></span>
<span id="S6.SS5.tab1.2.1.3.3.6" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.3.3.6.1" class="ltx_text" style="font-size:70%;">c.52.1.</span></span>
<span id="S6.SS5.tab1.2.1.3.3.7" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.3.3.7.1" class="ltx_text" style="font-size:70%;">c.55.1.</span></span>
<span id="S6.SS5.tab1.2.1.3.3.8" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.3.3.8.1" class="ltx_text" style="font-size:70%;">c.55.3.</span></span></span>
<span id="S6.SS5.tab1.2.1.4.4" class="ltx_tr">
<span id="S6.SS5.tab1.2.1.4.4.1" class="ltx_td ltx_align_left ltx_th ltx_th_row"><span id="S6.SS5.tab1.2.1.4.4.1.1" class="ltx_text" style="font-size:70%;">c.67.1.</span></span>
<span id="S6.SS5.tab1.2.1.4.4.2" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.4.4.2.1" class="ltx_text" style="font-size:70%;">d.108.1.</span></span>
<span id="S6.SS5.tab1.2.1.4.4.3" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.4.4.3.1" class="ltx_text" style="font-size:70%;">d.14.1.</span></span>
<span id="S6.SS5.tab1.2.1.4.4.4" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.4.4.4.1" class="ltx_text" style="font-size:70%;">d.144.1.</span></span>
<span id="S6.SS5.tab1.2.1.4.4.5" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.4.4.5.1" class="ltx_text" style="font-size:70%;">d.15.1.</span></span>
<span id="S6.SS5.tab1.2.1.4.4.6" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.4.4.6.1" class="ltx_text" style="font-size:70%;">d.153.1.</span></span>
<span id="S6.SS5.tab1.2.1.4.4.7" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.4.4.7.1" class="ltx_text" style="font-size:70%;">d.169.1.</span></span>
<span id="S6.SS5.tab1.2.1.4.4.8" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.4.4.8.1" class="ltx_text" style="font-size:70%;">d.3.1.</span></span></span>
<span id="S6.SS5.tab1.2.1.5.5" class="ltx_tr">
<span id="S6.SS5.tab1.2.1.5.5.1" class="ltx_td ltx_align_left ltx_th ltx_th_row"><span id="S6.SS5.tab1.2.1.5.5.1.1" class="ltx_text" style="font-size:70%;">d.58.7.</span></span>
<span id="S6.SS5.tab1.2.1.5.5.2" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.5.5.2.1" class="ltx_text" style="font-size:70%;">d.92.1.</span></span>
<span id="S6.SS5.tab1.2.1.5.5.3" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.5.5.3.1" class="ltx_text" style="font-size:70%;">g.3.11.</span></span>
<span id="S6.SS5.tab1.2.1.5.5.4" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.5.5.4.1" class="ltx_text" style="font-size:70%;">g.3.6.</span></span>
<span id="S6.SS5.tab1.2.1.5.5.5" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.5.5.5.1" class="ltx_text" style="font-size:70%;">g.3.7.</span></span>
<span id="S6.SS5.tab1.2.1.5.5.6" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.5.5.6.1" class="ltx_text" style="font-size:70%;">g.37.1.</span></span>
<span id="S6.SS5.tab1.2.1.5.5.7" class="ltx_td ltx_align_left"><span id="S6.SS5.tab1.2.1.5.5.7.1" class="ltx_text" style="font-size:70%;">g.39.1.</span></span>
<span id="S6.SS5.tab1.2.1.5.5.8" class="ltx_td"></span></span>
<span id="S6.SS5.tab1.2.1.6.6" class="ltx_tr">
<span id="S6.SS5.tab1.2.1.6.6.1" class="ltx_td ltx_align_left ltx_th ltx_th_row"><span id="S6.SS5.tab1.2.1.6.6.1.1" class="ltx_ERROR undefined">\botrule</span></span>
<span id="S6.SS5.tab1.2.1.6.6.2" class="ltx_td"></span>
<span id="S6.SS5.tab1.2.1.6.6.3" class="ltx_td"></span>
<span id="S6.SS5.tab1.2.1.6.6.4" class="ltx_td"></span>
<span id="S6.SS5.tab1.2.1.6.6.5" class="ltx_td"></span>
<span id="S6.SS5.tab1.2.1.6.6.6" class="ltx_td"></span>
<span id="S6.SS5.tab1.2.1.6.6.7" class="ltx_td"></span>
<span id="S6.SS5.tab1.2.1.6.6.8" class="ltx_td"></span></span>
</span>
</span>SCOP Super families used in our experiments. We only considered super families with at least 20 proteins and three or more families.</p>
</div>
</div>
</figure>
<div id="S6.SS5.p4" class="ltx_para">
<p id="S6.SS5.p4.1" class="ltx_p">In order to assess HMMER-STRUCT performance, we used the HMMER package. We did not compare with SAM <cite class="ltx_cite ltx_citemacro_citep">(Hughey <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib25" title="" class="ltx_ref">1996</a>)</cite> package. First, because our goal was to evaluate whether structural properties can improve pHMMs, not to compare the two packages, and second, because a related previous study on the same dataset actually showed HMMER outperforming SAM <cite class="ltx_cite ltx_citemacro_citep">(Bernardes <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib8" title="" class="ltx_ref">2007</a>)</cite>. The same study also indicated better results on the “twilight zone” using structural alignment tools, such as MAMMOTH-mult and 3DCOFFEE. We used MAMMOTH in this study.</p>
</div>
<div id="S6.SS5.p5" class="ltx_para">
<p id="S6.SS5.p5.3" class="ltx_p">Results were graphically analyzed by building ROC and Precision/Recall curves. ROC curves are a common measure of performance that is very used in bioinformatics application. They are based on the relation of the false positives (non homologue proteins) and of true positives (homologue proteins), and are obtained by varying a parameter that affect these relationships. We further present Precision/Recall curves, as they give a good perspective on true positives, false positives and false negatives hits. In both cases, the bigger the area under the curve (AUC), the more efficient the analyzed tool is. In both cases we used the minimal <em id="S6.SS5.p5.3.1" class="ltx_emph ltx_font_italic">e-value</em> required to accept a match as the parameter used to build both curves. We ranged e-values between <math id="S6.SS5.p5.1.m1.1" class="ltx_Math" alttext="10^{-50}" display="inline"><semantics id="S6.SS5.p5.1.m1.1a"><msup id="S6.SS5.p5.1.m1.1.1" xref="S6.SS5.p5.1.m1.1.1.cmml"><mn id="S6.SS5.p5.1.m1.1.1.2" xref="S6.SS5.p5.1.m1.1.1.2.cmml">10</mn><mrow id="S6.SS5.p5.1.m1.1.1.3" xref="S6.SS5.p5.1.m1.1.1.3.cmml"><mo id="S6.SS5.p5.1.m1.1.1.3a" xref="S6.SS5.p5.1.m1.1.1.3.cmml">−</mo><mn id="S6.SS5.p5.1.m1.1.1.3.2" xref="S6.SS5.p5.1.m1.1.1.3.2.cmml">50</mn></mrow></msup><annotation-xml encoding="MathML-Content" id="S6.SS5.p5.1.m1.1b"><apply id="S6.SS5.p5.1.m1.1.1.cmml" xref="S6.SS5.p5.1.m1.1.1"><csymbol cd="ambiguous" id="S6.SS5.p5.1.m1.1.1.1.cmml" xref="S6.SS5.p5.1.m1.1.1">superscript</csymbol><cn type="integer" id="S6.SS5.p5.1.m1.1.1.2.cmml" xref="S6.SS5.p5.1.m1.1.1.2">10</cn><apply id="S6.SS5.p5.1.m1.1.1.3.cmml" xref="S6.SS5.p5.1.m1.1.1.3"><minus id="S6.SS5.p5.1.m1.1.1.3.1.cmml" xref="S6.SS5.p5.1.m1.1.1.3"></minus><cn type="integer" id="S6.SS5.p5.1.m1.1.1.3.2.cmml" xref="S6.SS5.p5.1.m1.1.1.3.2">50</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p5.1.m1.1c">10^{-50}</annotation></semantics></math> and <math id="S6.SS5.p5.2.m2.1" class="ltx_Math" alttext="10" display="inline"><semantics id="S6.SS5.p5.2.m2.1a"><mn id="S6.SS5.p5.2.m2.1.1" xref="S6.SS5.p5.2.m2.1.1.cmml">10</mn><annotation-xml encoding="MathML-Content" id="S6.SS5.p5.2.m2.1b"><cn type="integer" id="S6.SS5.p5.2.m2.1.1.cmml" xref="S6.SS5.p5.2.m2.1.1">10</cn></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p5.2.m2.1c">10</annotation></semantics></math>. Finally, we used the paired two tailed t-test to assess significance, and assumed that results with <math id="S6.SS5.p5.3.m3.1" class="ltx_Math" alttext="p\leq 0.05" display="inline"><semantics id="S6.SS5.p5.3.m3.1a"><mrow id="S6.SS5.p5.3.m3.1.1" xref="S6.SS5.p5.3.m3.1.1.cmml"><mi id="S6.SS5.p5.3.m3.1.1.2" xref="S6.SS5.p5.3.m3.1.1.2.cmml">p</mi><mo id="S6.SS5.p5.3.m3.1.1.1" xref="S6.SS5.p5.3.m3.1.1.1.cmml">≤</mo><mn id="S6.SS5.p5.3.m3.1.1.3" xref="S6.SS5.p5.3.m3.1.1.3.cmml">0.05</mn></mrow><annotation-xml encoding="MathML-Content" id="S6.SS5.p5.3.m3.1b"><apply id="S6.SS5.p5.3.m3.1.1.cmml" xref="S6.SS5.p5.3.m3.1.1"><leq id="S6.SS5.p5.3.m3.1.1.1.cmml" xref="S6.SS5.p5.3.m3.1.1.1"></leq><ci id="S6.SS5.p5.3.m3.1.1.2.cmml" xref="S6.SS5.p5.3.m3.1.1.2">𝑝</ci><cn type="float" id="S6.SS5.p5.3.m3.1.1.3.cmml" xref="S6.SS5.p5.3.m3.1.1.3">0.05</cn></apply></annotation-xml><annotation encoding="application/x-tex" id="S6.SS5.p5.3.m3.1c">p\leq 0.05</annotation></semantics></math> (I.e. 95% of confidence) are significant.</p>
</div>
</section>
</section>
<section id="S7" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">7 </span>Results</h2>

<div id="S7.p1" class="ltx_para">
<p id="S7.p1.1" class="ltx_p">As a first step, we build a model for each structural property and evaluate it according to the methodology described in the Methods section. The ROC curves are presented in figure <a href="#S7.F4" title="Figure 4 ‣ 7 Results ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a> and the Precision/Recall curves in figure <a href="#S7.F5" title="Figure 5 ‣ 7 Results ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>. Both figures show all models, that is, <span id="S7.p1.1.1" class="ltx_text ltx_font_italic">pHMM2D</span> (secondary structural model), <span id="S7.p1.1.2" class="ltx_text ltx_font_italic">pHMMOi</span> (Ooi measure model), <span id="S7.p1.1.3" class="ltx_text ltx_font_italic">pHMMAcc</span> (inaccessibility model) and <span id="S7.p1.1.4" class="ltx_text ltx_font_italic">pHMM3D</span> (three-dimensional structure model) outperforming the HMMER model. Table <a href="#S7" title="7 Results ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a> shows the paired two tailed t-test between each model. All models built from structural properties perform significantly when compared to HMMER. Only, the <span id="S7.p1.1.5" class="ltx_text ltx_font_italic">pHMM3D</span> and <span id="S7.p1.1.6" class="ltx_text ltx_font_italic">pHMMAcc</span> results are not significant in relation to each other.</p>
</div>
<figure id="S7.F4" class="ltx_figure"><img src="/html/0704.2010/assets/x4.png" id="S7.F4.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="180" height="126" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 4: </span>Performance of each model in HMMER-STRUCT tool, for MAMMOTH aligner, as measured by ROC Curves</figcaption>
</figure>
<figure id="S7.F5" class="ltx_figure"><img src="/html/0704.2010/assets/x5.png" id="S7.F5.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="180" height="126" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 5: </span>Performance of each model in HMMER-STRUCT tool, for MAMMOTH aligner, as measured by Precision/Recall Curves</figcaption>
</figure>
<figure id="S7.tab1" class="ltx_table">
<div class="ltx_flex_figure ltx_flex_table">
<div class="ltx_flex_cell ltx_flex_size_1"><span id="S7.tab1.1" class="ltx_ERROR ltx_figure_panel undefined">\processtable</span></div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<p id="S7.tab1.2" class="ltx_p ltx_figure_panel">HMMER-STRUCT paired t-test

<span id="S7.tab1.2.1" class="ltx_tabular ltx_align_middle">
<span class="ltx_tbody">
<span id="S7.tab1.2.1.1.1" class="ltx_tr">
<span id="S7.tab1.2.1.1.1.1" class="ltx_td ltx_border_l ltx_border_r ltx_border_t"></span>
<span id="S7.tab1.2.1.1.1.2" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.1.1.2.1" class="ltx_text" style="font-size:70%;">HMMER</span></span>
<span id="S7.tab1.2.1.1.1.3" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.1.1.3.1" class="ltx_text" style="font-size:70%;">pHMM2D</span></span>
<span id="S7.tab1.2.1.1.1.4" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.1.1.4.1" class="ltx_text" style="font-size:70%;">pHMM3D</span></span>
<span id="S7.tab1.2.1.1.1.5" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.1.1.5.1" class="ltx_text" style="font-size:70%;">pHMMAcc</span></span></span>
<span id="S7.tab1.2.1.2.2" class="ltx_tr">
<span id="S7.tab1.2.1.2.2.1" class="ltx_td ltx_align_left ltx_border_l ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.2.2.1.1" class="ltx_text" style="font-size:70%;">pHMMOi</span></span>
<span id="S7.tab1.2.1.2.2.2" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.2.2.2.1" class="ltx_text" style="font-size:70%;">0,00741</span></span>
<span id="S7.tab1.2.1.2.2.3" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.2.2.3.1" class="ltx_text" style="font-size:70%;">0,04</span></span>
<span id="S7.tab1.2.1.2.2.4" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.2.2.4.1" class="ltx_text" style="font-size:70%;">0,01</span></span>
<span id="S7.tab1.2.1.2.2.5" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.2.2.5.1" class="ltx_text" style="font-size:70%;">0,01</span></span></span>
<span id="S7.tab1.2.1.3.3" class="ltx_tr">
<span id="S7.tab1.2.1.3.3.1" class="ltx_td ltx_align_left ltx_border_l ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.3.3.1.1" class="ltx_text" style="font-size:70%;">pHMMAcc</span></span>
<span id="S7.tab1.2.1.3.3.2" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.3.3.2.1" class="ltx_text" style="font-size:70%;">0,03877</span></span>
<span id="S7.tab1.2.1.3.3.3" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.3.3.3.1" class="ltx_text" style="font-size:70%;">0,01</span></span>
<span id="S7.tab1.2.1.3.3.4" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.3.3.4.1" class="ltx_text" style="font-size:70%;">0,05</span></span>
<span id="S7.tab1.2.1.3.3.5" class="ltx_td ltx_border_r ltx_border_t"></span></span>
<span id="S7.tab1.2.1.4.4" class="ltx_tr">
<span id="S7.tab1.2.1.4.4.1" class="ltx_td ltx_align_left ltx_border_l ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.4.4.1.1" class="ltx_text" style="font-size:70%;">pHMM3D</span></span>
<span id="S7.tab1.2.1.4.4.2" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.4.4.2.1" class="ltx_text" style="font-size:70%;">0,01</span></span>
<span id="S7.tab1.2.1.4.4.3" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.4.4.3.1" class="ltx_text" style="font-size:70%;">0,01</span></span>
<span id="S7.tab1.2.1.4.4.4" class="ltx_td ltx_border_r ltx_border_t"></span>
<span id="S7.tab1.2.1.4.4.5" class="ltx_td ltx_border_r ltx_border_t"></span></span>
<span id="S7.tab1.2.1.5.5" class="ltx_tr">
<span id="S7.tab1.2.1.5.5.1" class="ltx_td ltx_align_left ltx_border_b ltx_border_l ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.5.5.1.1" class="ltx_text" style="font-size:70%;">pHMM2D</span></span>
<span id="S7.tab1.2.1.5.5.2" class="ltx_td ltx_align_left ltx_border_b ltx_border_r ltx_border_t"><span id="S7.tab1.2.1.5.5.2.1" class="ltx_text" style="font-size:70%;">0,00660</span></span>
<span id="S7.tab1.2.1.5.5.3" class="ltx_td ltx_border_b ltx_border_r ltx_border_t"></span>
<span id="S7.tab1.2.1.5.5.4" class="ltx_td ltx_border_b ltx_border_r ltx_border_t"></span>
<span id="S7.tab1.2.1.5.5.5" class="ltx_td ltx_border_b ltx_border_r ltx_border_t"></span></span>
</span>
</span>Paired two tailed t-test when comparing performance of each HMMER-STRUCT model all against all.</p>
</div>
</div>
</figure>
<div id="S7.p2" class="ltx_para">
<p id="S7.p2.1" class="ltx_p">Next, we compare the performance of the model library with respect to the initial HMMER model. To do so, we joined the five models, one for each structural property, and scored the test sequences using <span id="S7.p2.1.1" class="ltx_text ltx_font_italic">hmmpfam</span>. Figure <a href="#S7.F6" title="Figure 6 ‣ 7 Results ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">6</span></a> shows the ROC curve for the results. Figure <a href="#S7.F7" title="Figure 7 ‣ 7 Results ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a> shows graphically the results through Precison/Recall curves. Both figures show HMMER-STRUCT outperforming HMMER. Table <a href="#S7" title="7 Results ‣ A study of structural properties on profile HMMs" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a> displays significance results. The difference between HMMER-STRUCT and HMMER results are statistically significant according to paired two tailed t-test. The two tailed t-test also indicate significant differences between HMMER-STRUCT and each HMMER-STRUCT component, i.e, HMMER, <span id="S7.p2.1.2" class="ltx_text ltx_font_italic">pHMM2D</span>, <span id="S7.p2.1.3" class="ltx_text ltx_font_italic">pHMM3D</span>, <span id="S7.p2.1.4" class="ltx_text ltx_font_italic">pHMMAcc</span> and <span id="S7.p2.1.5" class="ltx_text ltx_font_italic">pHMMOi</span>.</p>
</div>
<figure id="S7.F6" class="ltx_figure"><img src="/html/0704.2010/assets/x6.png" id="S7.F6.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="194" height="136" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 6: </span>HMMER-STRUCT Performance for MAMMOTH aligner, as measured by ROC Curves</figcaption>
</figure>
<figure id="S7.F7" class="ltx_figure"><img src="/html/0704.2010/assets/x7.png" id="S7.F7.g1" class="ltx_graphics ltx_centering ltx_img_landscape" width="194" height="136" alt="Refer to caption">
<figcaption class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure 7: </span>HMMER-STRUCT Performance for MAMMOTH aligner, as measured by Precision/Recall Curves</figcaption>
</figure>
<figure id="S7.5" class="ltx_table">
<div class="ltx_flex_figure ltx_flex_table">
<div class="ltx_flex_cell ltx_flex_size_1"><span id="S7.5.6" class="ltx_ERROR ltx_figure_panel undefined">\processtable</span></div>
<div class="ltx_flex_break"></div>
<div class="ltx_flex_cell ltx_flex_size_1">
<p id="S7.5.5" class="ltx_p ltx_figure_panel">HMMER-STRUCT paired t-test

<span id="S7.5.5.5" class="ltx_tabular ltx_guessed_headers ltx_align_middle">
<span class="ltx_thead">
<span id="S7.5.5.5.6.1" class="ltx_tr">
<span id="S7.5.5.5.6.1.1" class="ltx_td ltx_th ltx_th_column ltx_th_row ltx_border_l ltx_border_r ltx_border_t"></span>
<span id="S7.5.5.5.6.1.2" class="ltx_td ltx_align_left ltx_th ltx_th_column ltx_border_r ltx_border_t"><span id="S7.5.5.5.6.1.2.1" class="ltx_text" style="font-size:70%;">HMMER-STRUCT</span></span></span>
</span>
<span class="ltx_tbody">
<span id="S7.1.1.1.1" class="ltx_tr">
<span id="S7.1.1.1.1.2" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t"><span id="S7.1.1.1.1.2.1" class="ltx_text" style="font-size:70%;">HMMER</span></span>
<span id="S7.1.1.1.1.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><math id="S7.1.1.1.1.1.m1.1" class="ltx_Math" alttext="10^{-4}" display="inline"><semantics id="S7.1.1.1.1.1.m1.1a"><msup id="S7.1.1.1.1.1.m1.1.1" xref="S7.1.1.1.1.1.m1.1.1.cmml"><mn mathsize="70%" id="S7.1.1.1.1.1.m1.1.1.2" xref="S7.1.1.1.1.1.m1.1.1.2.cmml">10</mn><mrow id="S7.1.1.1.1.1.m1.1.1.3" xref="S7.1.1.1.1.1.m1.1.1.3.cmml"><mo mathsize="70%" id="S7.1.1.1.1.1.m1.1.1.3a" xref="S7.1.1.1.1.1.m1.1.1.3.cmml">−</mo><mn mathsize="70%" id="S7.1.1.1.1.1.m1.1.1.3.2" xref="S7.1.1.1.1.1.m1.1.1.3.2.cmml">4</mn></mrow></msup><annotation-xml encoding="MathML-Content" id="S7.1.1.1.1.1.m1.1b"><apply id="S7.1.1.1.1.1.m1.1.1.cmml" xref="S7.1.1.1.1.1.m1.1.1"><csymbol cd="ambiguous" id="S7.1.1.1.1.1.m1.1.1.1.cmml" xref="S7.1.1.1.1.1.m1.1.1">superscript</csymbol><cn type="integer" id="S7.1.1.1.1.1.m1.1.1.2.cmml" xref="S7.1.1.1.1.1.m1.1.1.2">10</cn><apply id="S7.1.1.1.1.1.m1.1.1.3.cmml" xref="S7.1.1.1.1.1.m1.1.1.3"><minus id="S7.1.1.1.1.1.m1.1.1.3.1.cmml" xref="S7.1.1.1.1.1.m1.1.1.3"></minus><cn type="integer" id="S7.1.1.1.1.1.m1.1.1.3.2.cmml" xref="S7.1.1.1.1.1.m1.1.1.3.2">4</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S7.1.1.1.1.1.m1.1c">10^{-4}</annotation></semantics></math></span></span>
<span id="S7.2.2.2.2" class="ltx_tr">
<span id="S7.2.2.2.2.2" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t"><span id="S7.2.2.2.2.2.1" class="ltx_text" style="font-size:70%;">pHMMAcc</span></span>
<span id="S7.2.2.2.2.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><math id="S7.2.2.2.2.1.m1.1" class="ltx_Math" alttext="10^{-4}" display="inline"><semantics id="S7.2.2.2.2.1.m1.1a"><msup id="S7.2.2.2.2.1.m1.1.1" xref="S7.2.2.2.2.1.m1.1.1.cmml"><mn mathsize="70%" id="S7.2.2.2.2.1.m1.1.1.2" xref="S7.2.2.2.2.1.m1.1.1.2.cmml">10</mn><mrow id="S7.2.2.2.2.1.m1.1.1.3" xref="S7.2.2.2.2.1.m1.1.1.3.cmml"><mo mathsize="70%" id="S7.2.2.2.2.1.m1.1.1.3a" xref="S7.2.2.2.2.1.m1.1.1.3.cmml">−</mo><mn mathsize="70%" id="S7.2.2.2.2.1.m1.1.1.3.2" xref="S7.2.2.2.2.1.m1.1.1.3.2.cmml">4</mn></mrow></msup><annotation-xml encoding="MathML-Content" id="S7.2.2.2.2.1.m1.1b"><apply id="S7.2.2.2.2.1.m1.1.1.cmml" xref="S7.2.2.2.2.1.m1.1.1"><csymbol cd="ambiguous" id="S7.2.2.2.2.1.m1.1.1.1.cmml" xref="S7.2.2.2.2.1.m1.1.1">superscript</csymbol><cn type="integer" id="S7.2.2.2.2.1.m1.1.1.2.cmml" xref="S7.2.2.2.2.1.m1.1.1.2">10</cn><apply id="S7.2.2.2.2.1.m1.1.1.3.cmml" xref="S7.2.2.2.2.1.m1.1.1.3"><minus id="S7.2.2.2.2.1.m1.1.1.3.1.cmml" xref="S7.2.2.2.2.1.m1.1.1.3"></minus><cn type="integer" id="S7.2.2.2.2.1.m1.1.1.3.2.cmml" xref="S7.2.2.2.2.1.m1.1.1.3.2">4</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S7.2.2.2.2.1.m1.1c">10^{-4}</annotation></semantics></math></span></span>
<span id="S7.3.3.3.3" class="ltx_tr">
<span id="S7.3.3.3.3.2" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t"><span id="S7.3.3.3.3.2.1" class="ltx_text" style="font-size:70%;">pHMMOi</span></span>
<span id="S7.3.3.3.3.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><math id="S7.3.3.3.3.1.m1.1" class="ltx_Math" alttext="10^{-3}" display="inline"><semantics id="S7.3.3.3.3.1.m1.1a"><msup id="S7.3.3.3.3.1.m1.1.1" xref="S7.3.3.3.3.1.m1.1.1.cmml"><mn mathsize="70%" id="S7.3.3.3.3.1.m1.1.1.2" xref="S7.3.3.3.3.1.m1.1.1.2.cmml">10</mn><mrow id="S7.3.3.3.3.1.m1.1.1.3" xref="S7.3.3.3.3.1.m1.1.1.3.cmml"><mo mathsize="70%" id="S7.3.3.3.3.1.m1.1.1.3a" xref="S7.3.3.3.3.1.m1.1.1.3.cmml">−</mo><mn mathsize="70%" id="S7.3.3.3.3.1.m1.1.1.3.2" xref="S7.3.3.3.3.1.m1.1.1.3.2.cmml">3</mn></mrow></msup><annotation-xml encoding="MathML-Content" id="S7.3.3.3.3.1.m1.1b"><apply id="S7.3.3.3.3.1.m1.1.1.cmml" xref="S7.3.3.3.3.1.m1.1.1"><csymbol cd="ambiguous" id="S7.3.3.3.3.1.m1.1.1.1.cmml" xref="S7.3.3.3.3.1.m1.1.1">superscript</csymbol><cn type="integer" id="S7.3.3.3.3.1.m1.1.1.2.cmml" xref="S7.3.3.3.3.1.m1.1.1.2">10</cn><apply id="S7.3.3.3.3.1.m1.1.1.3.cmml" xref="S7.3.3.3.3.1.m1.1.1.3"><minus id="S7.3.3.3.3.1.m1.1.1.3.1.cmml" xref="S7.3.3.3.3.1.m1.1.1.3"></minus><cn type="integer" id="S7.3.3.3.3.1.m1.1.1.3.2.cmml" xref="S7.3.3.3.3.1.m1.1.1.3.2">3</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S7.3.3.3.3.1.m1.1c">10^{-3}</annotation></semantics></math></span></span>
<span id="S7.4.4.4.4" class="ltx_tr">
<span id="S7.4.4.4.4.2" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_l ltx_border_r ltx_border_t"><span id="S7.4.4.4.4.2.1" class="ltx_text" style="font-size:70%;">pHMM2D</span></span>
<span id="S7.4.4.4.4.1" class="ltx_td ltx_align_left ltx_border_r ltx_border_t"><math id="S7.4.4.4.4.1.m1.1" class="ltx_Math" alttext="10^{-3}" display="inline"><semantics id="S7.4.4.4.4.1.m1.1a"><msup id="S7.4.4.4.4.1.m1.1.1" xref="S7.4.4.4.4.1.m1.1.1.cmml"><mn mathsize="70%" id="S7.4.4.4.4.1.m1.1.1.2" xref="S7.4.4.4.4.1.m1.1.1.2.cmml">10</mn><mrow id="S7.4.4.4.4.1.m1.1.1.3" xref="S7.4.4.4.4.1.m1.1.1.3.cmml"><mo mathsize="70%" id="S7.4.4.4.4.1.m1.1.1.3a" xref="S7.4.4.4.4.1.m1.1.1.3.cmml">−</mo><mn mathsize="70%" id="S7.4.4.4.4.1.m1.1.1.3.2" xref="S7.4.4.4.4.1.m1.1.1.3.2.cmml">3</mn></mrow></msup><annotation-xml encoding="MathML-Content" id="S7.4.4.4.4.1.m1.1b"><apply id="S7.4.4.4.4.1.m1.1.1.cmml" xref="S7.4.4.4.4.1.m1.1.1"><csymbol cd="ambiguous" id="S7.4.4.4.4.1.m1.1.1.1.cmml" xref="S7.4.4.4.4.1.m1.1.1">superscript</csymbol><cn type="integer" id="S7.4.4.4.4.1.m1.1.1.2.cmml" xref="S7.4.4.4.4.1.m1.1.1.2">10</cn><apply id="S7.4.4.4.4.1.m1.1.1.3.cmml" xref="S7.4.4.4.4.1.m1.1.1.3"><minus id="S7.4.4.4.4.1.m1.1.1.3.1.cmml" xref="S7.4.4.4.4.1.m1.1.1.3"></minus><cn type="integer" id="S7.4.4.4.4.1.m1.1.1.3.2.cmml" xref="S7.4.4.4.4.1.m1.1.1.3.2">3</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S7.4.4.4.4.1.m1.1c">10^{-3}</annotation></semantics></math></span></span>
<span id="S7.5.5.5.5" class="ltx_tr">
<span id="S7.5.5.5.5.2" class="ltx_td ltx_align_left ltx_th ltx_th_row ltx_border_b ltx_border_l ltx_border_r ltx_border_t"><span id="S7.5.5.5.5.2.1" class="ltx_text" style="font-size:70%;">pHMM3D</span></span>
<span id="S7.5.5.5.5.1" class="ltx_td ltx_align_left ltx_border_b ltx_border_r ltx_border_t"><math id="S7.5.5.5.5.1.m1.1" class="ltx_Math" alttext="10^{-4}" display="inline"><semantics id="S7.5.5.5.5.1.m1.1a"><msup id="S7.5.5.5.5.1.m1.1.1" xref="S7.5.5.5.5.1.m1.1.1.cmml"><mn mathsize="70%" id="S7.5.5.5.5.1.m1.1.1.2" xref="S7.5.5.5.5.1.m1.1.1.2.cmml">10</mn><mrow id="S7.5.5.5.5.1.m1.1.1.3" xref="S7.5.5.5.5.1.m1.1.1.3.cmml"><mo mathsize="70%" id="S7.5.5.5.5.1.m1.1.1.3a" xref="S7.5.5.5.5.1.m1.1.1.3.cmml">−</mo><mn mathsize="70%" id="S7.5.5.5.5.1.m1.1.1.3.2" xref="S7.5.5.5.5.1.m1.1.1.3.2.cmml">4</mn></mrow></msup><annotation-xml encoding="MathML-Content" id="S7.5.5.5.5.1.m1.1b"><apply id="S7.5.5.5.5.1.m1.1.1.cmml" xref="S7.5.5.5.5.1.m1.1.1"><csymbol cd="ambiguous" id="S7.5.5.5.5.1.m1.1.1.1.cmml" xref="S7.5.5.5.5.1.m1.1.1">superscript</csymbol><cn type="integer" id="S7.5.5.5.5.1.m1.1.1.2.cmml" xref="S7.5.5.5.5.1.m1.1.1.2">10</cn><apply id="S7.5.5.5.5.1.m1.1.1.3.cmml" xref="S7.5.5.5.5.1.m1.1.1.3"><minus id="S7.5.5.5.5.1.m1.1.1.3.1.cmml" xref="S7.5.5.5.5.1.m1.1.1.3"></minus><cn type="integer" id="S7.5.5.5.5.1.m1.1.1.3.2.cmml" xref="S7.5.5.5.5.1.m1.1.1.3.2">4</cn></apply></apply></annotation-xml><annotation encoding="application/x-tex" id="S7.5.5.5.5.1.m1.1c">10^{-4}</annotation></semantics></math></span></span>
</span>
</span>Paired two tailed t-test when comparing performance of each HMMER-STRUCT component with the combined model.</p>
</div>
</div>
</figure>
</section>
<section id="S8" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">8 </span>Discussion</h2>

<div id="S8.p1" class="ltx_para">
<p id="S8.p1.1" class="ltx_p">The accuracy of homology detection methods is essential for the problem of inferring the function of unknown-function proteins. However, improving accuracy becomes hard when similarity between sequences is low. We proposed a method to improve pHMMs sensitivity by adding structural properties in the model building stage. We showed that the pHMMs trained according to this method are more sensitive than pHMMs trained from multiple sequence alignments, even if the alignment itself relied on structural properties. Our experiments demonstrated best performance for <span id="S8.p1.1.1" class="ltx_text ltx_font_italic">pHMM2D</span>, that used secondary structural properties, and for <span id="S8.p1.1.2" class="ltx_text ltx_font_italic">pHMMOi</span>, that used packing density residues. Both pHMMs present similar performance. We believe that the good results obtained with the <span id="S8.p1.1.3" class="ltx_text ltx_font_italic">pHMMoi</span> model can be attributed to the fact that tight packing is important for protein stability, and follow well-known results that indicate that amino-acids located in the core protein are more conserved than amino-acids located in other sites <cite class="ltx_cite ltx_citemacro_citep">(Privalov,, <a href="#bib.bib35" title="" class="ltx_ref">2000</a>)</cite>. In the same way, the <span id="S8.p1.1.4" class="ltx_text ltx_font_italic">pHMM2D</span> model achieve good performance as secondary structure elements are responsible for maintaining the form in homologue proteins. These elements form motifs and domains, which are related with protein function. Conserved sites may point to functionally and structurally important regions. These observations may explain the higher performance of models based on packing residues and on secondary structural properties.</p>
</div>
<div id="S8.p2" class="ltx_para">
<p id="S8.p2.1" class="ltx_p">The <span id="S8.p2.1.1" class="ltx_text ltx_font_italic">pHMMAcc</span> models, based on amino-acid inaccessibility, and the <span id="S8.p2.1.2" class="ltx_text ltx_font_italic">pHMM3D</span> models, based on three-dimensional coordinates, did not perform as well. The <span id="S8.p2.1.3" class="ltx_text ltx_font_italic">pHMMAcc</span> models did not achieve statistical significance results, when they were compared with HMMER. On the other hand, we observe that the inaccessibility property can be explained by hydrophobic effects, as are the amino-acids with hydrophobic side-chain that go toward the core protein by forming packages. Therefore hydrophobicity was represented in the <span id="S8.p2.1.4" class="ltx_text ltx_font_italic">pHMMOi</span> model, that achieved good performance. Our results suggest the difference between models stems from the <span id="S8.p2.1.5" class="ltx_text ltx_font_italic">pHMMOi</span> models to be more accurate and precise than what is used when building <span id="S8.p2.1.6" class="ltx_text ltx_font_italic">pHMMAcc</span>.</p>
</div>
<div id="S8.p3" class="ltx_para">
<p id="S8.p3.1" class="ltx_p">However, we believe the inaccessibility property is already represented appropriately by <span id="S8.p3.1.1" class="ltx_text ltx_font_italic">pHMMOi</span> model. Since amino-acids with high packing density already are inaccessible. Therefore, <span id="S8.p3.1.2" class="ltx_text ltx_font_italic">pHMMOi</span> outperformed the <span id="S8.p3.1.3" class="ltx_text ltx_font_italic">pHMMAcc</span>, as <span id="S8.p3.1.4" class="ltx_text ltx_font_italic">pHMMoi</span> has more information than <span id="S8.p3.1.5" class="ltx_text ltx_font_italic">pHMMAcc</span>.</p>
</div>
<div id="S8.p4" class="ltx_para">
<p id="S8.p4.1" class="ltx_p">The chief contribution of our method was achieved when all the models work together. The combined models performed significantly better than any single model. We believe that this results from the fact that each trained pHMMs represents a different structural property. Therefore, combining the models increases sensitivity by exploring the different structural properties.</p>
</div>
<div id="S8.p5" class="ltx_para">
<p id="S8.p5.1" class="ltx_p">Our method shows that structural information can be added during the training phase of pHMM to improve sensitivity, without much changes to the usage of pHMM methodology, and applied to recently discovered proteins for which there is little structural information.</p>
</div>
</section>
<section id="S9" class="ltx_section">
<h2 class="ltx_title ltx_title_section">
<span class="ltx_tag ltx_tag_section">9 </span>Conclusion</h2>

<div id="S9.p1" class="ltx_para">
<p id="S9.p1.1" class="ltx_p">The increasing number of studies involving pHMMs and the use of structural information has been quite remarkable <cite class="ltx_cite ltx_citemacro_citep">(Hou <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib24" title="" class="ltx_ref">2004</a>; Alexandrov <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib1" title="" class="ltx_ref">2004</a>; Bystroff <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib11" title="" class="ltx_ref">2000</a>)</cite>. Most of these approaches build structural models based on three-dimensional coordinates. In contrast, we present a novel methodology to train pHMMs based on structural alignment and other structural properties using a set of homologue protein sequences. Our method builds five models from an aligned homologue sequence set. Each model represents a different structural property, and the union of the models represent the structural context of aligned proteins. The properties used were primary, secondary and tertiary structures, accessibility and packing residue. Note that previous attempts have already used secondary and tertiary structural properties to train pHMM, though in quite a different way. However, accessibility and packing residue properties were used for the first time in pHMM training, with good results in the latter case.</p>
</div>
<div id="S9.p2" class="ltx_para">
<p id="S9.p2.1" class="ltx_p">In order, to build each model, we developed a novel sequence-weighting algorithm based on structural weights that are attributed for each amino-acid. Traditional weighting-algorithm works gives the same weight for every residue in the protein. Instead, we propose a method that gives a different weight to each amino-acid into a protein, according to structural properties that suggest it may be in a conserved region. Our results relied on prior work <cite class="ltx_cite ltx_citemacro_citep">(Chakrabarti <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib12" title="" class="ltx_ref">2004</a>; Deane <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib13" title="" class="ltx_ref">2003</a>; Nishikawa <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib32" title="" class="ltx_ref">1986</a>)</cite> that suggested interesting properties and estimated their weight.</p>
</div>
<div id="S9.p3" class="ltx_para">
<p id="S9.p3.1" class="ltx_p">Nowadays, the most popular approach to discovering the function of a newly found protein is through sequence similarity search. In fact, it is well known that structure is more conserved than sequence, and thus structural similarity can suggest function similarity. On the other hand, structural data is sparse and are usually not available for proteins with unknown function. Therefore, it is very important that methods that uses structural properties to build models will not need to rely on structural information for a new protein. Our method makes use of structural properties only at the model building stage, but not at scoring.</p>
</div>
<div id="S9.p4" class="ltx_para">
<p id="S9.p4.1" class="ltx_p">Our results show that the use of structural properties can improve the sensitivity of remote homology methods. Moreover, the combination of different model (one for each property) outperforms the use of individual properties. A number of future research directions present themselves. It will be interesting to include more models, such as that based on bond-hydrogen properties. Also, it will be interesting to apply our methodology to other remote homology tools, such as SAM <cite class="ltx_cite ltx_citemacro_citep">(Hughey <span class="ltx_text ltx_font_italic">et al</span>.,, <a href="#bib.bib25" title="" class="ltx_ref">1996</a>)</cite> and T-HMM <cite class="ltx_cite ltx_citemacro_cite">Qian <span class="ltx_text ltx_font_italic">et al</span>., (<a href="#bib.bib36" title="" class="ltx_ref">2004</a>)</cite>. Ultimately, we believe that our work is a step in the major challenge of finding the set of structural properties or features that represent precisely membership of a super family.</p>
</div>
</section>
<section id="Sx1" class="ltx_section">
<h2 class="ltx_title ltx_title_section">Acknowledgement</h2>

<div id="Sx1.p1" class="ltx_para">
<p id="Sx1.p1.1" class="ltx_p">We are grateful to CNPq for financial support.</p>
</div>
</section>
<section id="bib" class="ltx_bibliography">
<h2 class="ltx_title ltx_title_bibliography">References</h2>

<ul class="ltx_biblist">
<li id="bib.bib1" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Alexandrov <span id="bib.bib1.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2004)</span>
<span class="ltx_bibblock"> Alexandrov,V., Gerstein,M. (2004) Using 3D Hidden Markov Models that explicitly represent spatial coordinates to model and compare protein structures, <span id="bib.bib1.3.1" class="ltx_text ltx_font_italic">BMC Bioinformatics</span>, <span id="bib.bib1.4.2" class="ltx_text ltx_font_bold">5</span>, 1–10.

</span>
</li>
<li id="bib.bib2" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Altschul <span id="bib.bib2.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1990)</span>
<span class="ltx_bibblock"> Altschul,F., Gish,W., Miller,W., Myers,E., Lipman,D. (1990) A basic local alignment search tool, <span id="bib.bib2.3.1" class="ltx_text ltx_font_italic">Journal of Molecular Biology</span>, <span id="bib.bib2.4.2" class="ltx_text ltx_font_bold">215</span>, 403-410.

</span>
</li>
<li id="bib.bib3" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Altschul <span id="bib.bib3.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2000)</span>
<span class="ltx_bibblock"> Altschul,S., Madden,T., Schaffer,A., Zhang,J., Zhang,Z., Miller,W., Lipman,D. (2000) PSI-BLAST searches using hidden markov models of structural repeats: prediction of an unusual sliding DNA clamp and of beta-propellers in UV-damaged DNA-binding protein, <span id="bib.bib3.3.1" class="ltx_text ltx_font_italic">Nucleic Acids Research</span>, <span id="bib.bib3.4.2" class="ltx_text ltx_font_bold">28</span>, 3570-3580.

</span>
</li>
<li id="bib.bib4" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Andreeva <span id="bib.bib4.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2004)</span>
<span class="ltx_bibblock"> Andreeva,A., Howorth,D., Brenner,S., Hubbard,T., Chothia,C., Murzin,A. (2004) SCOP database in 2004: refinements integrate structure and sequence family data, <span id="bib.bib4.3.1" class="ltx_text ltx_font_italic">Nucleic Acids Research</span>, <span id="bib.bib4.4.2" class="ltx_text ltx_font_bold">32</span>, 226-229.

</span>
</li>
<li id="bib.bib5" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Attwood <span id="bib.bib5.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2005)</span>
<span class="ltx_bibblock"> Attwood,T., Bradley,P., Flower,D., Gaulton,A., Maudling,N., Mitchell,A. (2005) Article title, <span id="bib.bib5.3.1" class="ltx_text ltx_font_italic">Bioinformatics</span>, <span id="bib.bib5.4.2" class="ltx_text ltx_font_bold">21</span>, 3255–3263.

</span>
</li>
<li id="bib.bib6" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Bateman <span id="bib.bib6.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2004)</span>
<span class="ltx_bibblock"> Bateman,A., Coin,L., Durbin,R., Finn,R., Hollich,V., Griffiths-Jones,S., Khanna,A., Marshall,M., Moxon,S., Sonnhammer,E., Studholme,D., Yeats,C., Eddy,S. (2004) The Pfam Protein Families Database, <span id="bib.bib6.3.1" class="ltx_text ltx_font_italic">Nucleic Acids Research</span>, <span id="bib.bib6.4.2" class="ltx_text ltx_font_bold">32</span>, 138-141.

</span>
</li>
<li id="bib.bib7" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mitchell,  (1997)</span>
<span class="ltx_bibblock"> Mitchell, T. (1997) Machine Learning, <span id="bib.bib7.1.1" class="ltx_text ltx_font_italic">McGraw-Hill</span>.

</span>
</li>
<li id="bib.bib8" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Bernardes <span id="bib.bib8.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2007)</span>
<span class="ltx_bibblock"> Bernardes, J., Dàvila, A., Costa, V., Zaverucha, G. (2007) Improving model construction of profile HMMs for remote homology detection through structural alignment, <span id="bib.bib8.3.1" class="ltx_text ltx_font_italic">BMC Bioinformatics</span>, <span id="bib.bib8.4.2" class="ltx_text ltx_font_bold">8</span>, 435:1-12.

</span>
</li>
<li id="bib.bib9" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Branden <span id="bib.bib9.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1991)</span>
<span class="ltx_bibblock"> Branden,C., Tooze,J. (1991) Introduction to protein Structure, chapter Motifs of protein structure, <span id="bib.bib9.3.1" class="ltx_text ltx_font_italic">Garland Publishing</span>, 11-29.

</span>
</li>
<li id="bib.bib10" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Brown <span id="bib.bib10.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1993)</span>
<span class="ltx_bibblock"> Brown,M., Hughey,R., Krogh,A., Mian,I., Sjölander,K., Haussler,D. (1993) Using Dirichlet mixture priors to derive hidden markov models for protein families, <span id="bib.bib10.3.1" class="ltx_text ltx_font_italic">Proc. of First Int. Conf. on Intelligent Systems for Molecular Biology</span>, <span id="bib.bib10.4.2" class="ltx_text ltx_font_bold">1</span>, 47–55.

</span>
</li>
<li id="bib.bib11" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Bystroff <span id="bib.bib11.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2000)</span>
<span class="ltx_bibblock"> Bystroff,C., Baker,D. (2000) HMMSTR:A hidden Markov model for local sequence-structure correlation in proteins, <span id="bib.bib11.3.1" class="ltx_text ltx_font_italic">Journal of Molecular Biology</span>, <span id="bib.bib11.4.2" class="ltx_text ltx_font_bold">301</span>, 173–190.

</span>
</li>
<li id="bib.bib12" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Chakrabarti <span id="bib.bib12.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2004)</span>
<span class="ltx_bibblock"> Chakrabarti,S., Sowdhamini,R. (2004) Regions of minimal structural variation among members of protein domain superfamilies: application to remote homology detection and modelling using distant relationships, <span id="bib.bib12.3.1" class="ltx_text ltx_font_italic">FEBS</span>, <span id="bib.bib12.4.2" class="ltx_text ltx_font_bold">569</span>, 31-36.

</span>
</li>
<li id="bib.bib13" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Deane <span id="bib.bib13.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2003)</span>
<span class="ltx_bibblock"> Deane,C., Perdersen,J., Lunter, G. (2003) Insertions and deletions in protein alignment, <span id="bib.bib13.3.1" class="ltx_text ltx_font_italic">unpublished</span>.

</span>
</li>
<li id="bib.bib14" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Eddy,  (1996)</span>
<span class="ltx_bibblock"> Eddy,S. (1996) Hidden markov models, <span id="bib.bib14.1.1" class="ltx_text ltx_font_italic">Current Opinion in Structural Biology</span>, <span id="bib.bib14.2.2" class="ltx_text ltx_font_bold">6</span>, 361-365.

</span>
</li>
<li id="bib.bib15" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Eddy,  (1998)</span>
<span class="ltx_bibblock"> Eddy,S. (1998) Profile hidden Markov models, <span id="bib.bib15.1.1" class="ltx_text ltx_font_italic">Current Opinion in Structural Biology</span>, <span id="bib.bib15.2.2" class="ltx_text ltx_font_bold">14</span>, 755-763.

</span>
</li>
<li id="bib.bib16" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Espadaler <span id="bib.bib16.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2005)</span>
<span class="ltx_bibblock"> Espadaler,J., Aragues,R., Eswar,N., Marti-Renom,M., Querol,E., Aviles,F., Sali,A., Oliva,B. (2005) Detecting remotely related proteins by their interactions and sequence similarity, <span id="bib.bib16.3.1" class="ltx_text ltx_font_italic">National Academy of Sciences</span>, <span id="bib.bib16.4.2" class="ltx_text ltx_font_bold">102</span>, 7151-7156.

</span>
</li>
<li id="bib.bib17" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gerstein <span id="bib.bib17.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1994)</span>
<span class="ltx_bibblock"> Gerstein,M., Sonnhammer,E., Chothia,C. (1994) Volume changes in protein evolution, <span id="bib.bib17.3.1" class="ltx_text ltx_font_italic">Journal of Molecular Biology</span>, <span id="bib.bib17.4.2" class="ltx_text ltx_font_bold">236</span>, 1067-1078.

</span>
</li>
<li id="bib.bib18" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gerstein <span id="bib.bib18.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1995)</span>
<span class="ltx_bibblock"> Gerstein,M., Altman,R. (1995) Average core structures and variability measures for protein families: application to the immunoglobulins, <span id="bib.bib18.3.1" class="ltx_text ltx_font_italic">Journal of Molecular Biology</span>, <span id="bib.bib18.4.2" class="ltx_text ltx_font_bold">251</span>, 165-175.

</span>
</li>
<li id="bib.bib19" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gough <span id="bib.bib19.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2001)</span>
<span class="ltx_bibblock"> Gough,J., Karplus,K., Hughey,R., Chothia,C. (2001) Assignment od homology to genome sequences using a library of hidden Markov models that represent all proteins of known structure, <span id="bib.bib19.3.1" class="ltx_text ltx_font_italic">Journal of Molecular Biology</span>, <span id="bib.bib19.4.2" class="ltx_text ltx_font_bold">313</span>, 903–919.

</span>
</li>
<li id="bib.bib20" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Goyon <span id="bib.bib20.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2004)</span>
<span class="ltx_bibblock"> Goyon,F., Tufféry,P. (2004) SA-Search: A web tool for protein structure mining based on structural alphabet, <span id="bib.bib20.3.1" class="ltx_text ltx_font_italic">Nucleic Acids Research</span>, <span id="bib.bib20.4.2" class="ltx_text ltx_font_bold">32</span>, 545–548.

</span>
</li>
<li id="bib.bib21" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Gribskov <span id="bib.bib21.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1987)</span>
<span class="ltx_bibblock"> Gribskov,M., McLachlan,A., Eisenberg,D. (1987) Profile analysis: detection of distantly related proteins, <span id="bib.bib21.3.1" class="ltx_text ltx_font_italic">National Academy of Sciences</span>, <span id="bib.bib21.4.2" class="ltx_text ltx_font_bold">84</span>,4355–4358.

</span>
</li>
<li id="bib.bib22" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Haft <span id="bib.bib22.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2003)</span>
<span class="ltx_bibblock"> Haft,D., Selengut,J., White,O. (2003) The TIGRFAMs database of protein families, <span id="bib.bib22.3.1" class="ltx_text ltx_font_italic">Nucleic Acids Research</span>, <span id="bib.bib22.4.2" class="ltx_text ltx_font_bold">31</span>, 371-373.

</span>
</li>
<li id="bib.bib23" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Helen <span id="bib.bib23.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2000)</span>
<span class="ltx_bibblock"> Helen,M., Westbrook,J., Feng,Z., Gilliland,G., Bhat,T., Weissig,H., Shindyalov,I., Bourne,P. (2000) The Protein Data Bank, <span id="bib.bib23.3.1" class="ltx_text ltx_font_italic">Nucleic Acids Research</span>, <span id="bib.bib23.4.2" class="ltx_text ltx_font_bold">28</span>, 235-242.

</span>
</li>
<li id="bib.bib24" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hou <span id="bib.bib24.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2004)</span>
<span class="ltx_bibblock"> Hou,Y., Hsu,W., Lee,M., Bystroff,C. (2004) Remote homolog detection using local sequence-structure correlations, <span id="bib.bib24.3.1" class="ltx_text ltx_font_italic">Journal of Molecular Biology</span>, <span id="bib.bib24.4.2" class="ltx_text ltx_font_bold">340</span>, 385-395.

</span>
</li>
<li id="bib.bib25" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Hughey <span id="bib.bib25.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1996)</span>
<span class="ltx_bibblock"> Hughey,R., Krogh,A. (1996) Hidden Markov models for sequence analysis: extension and analysis of the basic method, <span id="bib.bib25.3.1" class="ltx_text ltx_font_italic">Computer Applications in the Biosciences</span>, <span id="bib.bib25.4.2" class="ltx_text ltx_font_bold">12</span>, 95-107.

</span>
</li>
<li id="bib.bib26" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Karplus <span id="bib.bib26.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2005)</span>
<span class="ltx_bibblock"> Karplus,K., Karchin,R., Shackelford,G., Hughey,R. (2005) Calibrating E-values for hidden Markov models using reverse-sequence null models, <span id="bib.bib26.3.1" class="ltx_text ltx_font_italic">Bioinformatics</span>, <span id="bib.bib26.4.2" class="ltx_text ltx_font_bold">21</span>, 4107-4115.

</span>
</li>
<li id="bib.bib27" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Krogh <span id="bib.bib27.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1994)</span>
<span class="ltx_bibblock"> Krogh,A., Brown,M., Mian,I., Sjolander,K., Haussler,D. (1994) Hidden markov models in computational biology applications to protein modeling, <span id="bib.bib27.3.1" class="ltx_text ltx_font_italic">Journal of Molecular Biology</span>, <span id="bib.bib27.4.2" class="ltx_text ltx_font_bold">235</span>, 1501-1531.

</span>
</li>
<li id="bib.bib28" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Lee <span id="bib.bib28.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1971)</span>
<span class="ltx_bibblock"> Lee,B., Richards,F. (1971) The interpretation of protein structure: estimation of static accessibility, <span id="bib.bib28.3.1" class="ltx_text ltx_font_italic">Journal of Molecular Biology</span>, <span id="bib.bib28.4.2" class="ltx_text ltx_font_bold">14</span>, 379-400.

</span>
</li>
<li id="bib.bib29" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Matsuo <span id="bib.bib29.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1999)</span>
<span class="ltx_bibblock"> Matsuo,Y., Bryant,S. (1999) Identification of homolog core structures, <span id="bib.bib29.3.1" class="ltx_text ltx_font_italic">Proteins</span>, <span id="bib.bib29.4.2" class="ltx_text ltx_font_bold">35</span>, 70-79.

</span>
</li>
<li id="bib.bib30" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mitchell,  (1997)</span>
<span class="ltx_bibblock"> Mitchell,T. (1997) Machine Learning, <span id="bib.bib30.1.1" class="ltx_text ltx_font_italic">McGraw-Hill</span>

</span>
</li>
<li id="bib.bib31" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Mizuguchi <span id="bib.bib31.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1998)</span>
<span class="ltx_bibblock"> Mizuguchi,K., Deane,C., Johnson,M., Blundell,T., Overington,J. (1998) Article title, <span id="bib.bib31.3.1" class="ltx_text ltx_font_italic">Journal Name</span>, <span id="bib.bib31.4.2" class="ltx_text ltx_font_bold">14</span>, 617-623.

</span>
</li>
<li id="bib.bib32" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Nishikawa <span id="bib.bib32.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1986)</span>
<span class="ltx_bibblock"> Nishikawa,K., Ooi,T. (1986) Radial locations of amino acid residues in a globular protein: correlation with the sequence, <span id="bib.bib32.3.1" class="ltx_text ltx_font_italic">Journal of Biochemistry</span>, <span id="bib.bib32.4.2" class="ltx_text ltx_font_bold">100</span>, 1043-1047.

</span>
</li>
<li id="bib.bib33" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Park <span id="bib.bib33.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1998)</span>
<span class="ltx_bibblock"> Park,J., Karplus,K., Barrett,C., Hughey,R., Haussler,D., Hubbard,T., Chothia,C. (1998) Sequence comparisons using multiples sequence detect three times as many remote homologues as pairwise methods, <span id="bib.bib33.3.1" class="ltx_text ltx_font_italic">Journal of Molecular Biology</span>, <span id="bib.bib33.4.2" class="ltx_text ltx_font_bold">284</span>, 1201–1210.

</span>
</li>
<li id="bib.bib34" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Pearson,  (1985)</span>
<span class="ltx_bibblock"> Pearson,W. (1985) Rapid and sensitive sequence comparisons with FASTP and FASTA, <span id="bib.bib34.1.1" class="ltx_text ltx_font_italic">Methods Enzymol</span>, <span id="bib.bib34.2.2" class="ltx_text ltx_font_bold">183</span>, 63-98.

</span>
</li>
<li id="bib.bib35" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Privalov,  (2000)</span>
<span class="ltx_bibblock"> Privalov,P. (1996) Intermediate states in protein folding, <span id="bib.bib35.1.1" class="ltx_text ltx_font_italic">Journal Name</span>, <span id="bib.bib35.2.2" class="ltx_text ltx_font_bold">258</span>, 707-725.

</span>
</li>
<li id="bib.bib36" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Qian <span id="bib.bib36.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2004)</span>
<span class="ltx_bibblock"> Qian,B., Goldstein,R. (2004) Performance of an iterated T-HMM for homology detection, <span id="bib.bib36.3.1" class="ltx_text ltx_font_italic">Bioinformatics</span>, <span id="bib.bib36.4.2" class="ltx_text ltx_font_bold">20</span>, 2175–2180.

</span>
</li>
<li id="bib.bib37" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Wistrand <span id="bib.bib37.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (2005)</span>
<span class="ltx_bibblock"> Wistrand,M., Sonnhammer,E. (2005) Improved profile HMM performance by assessment of critical algorithmic in SAM and HMMER, <span id="bib.bib37.3.1" class="ltx_text ltx_font_italic">BMC Bioinformatics</span>, <span id="bib.bib37.4.2" class="ltx_text ltx_font_bold">6</span>, 1-10.

</span>
</li>
<li id="bib.bib38" class="ltx_bibitem">
<span class="ltx_tag ltx_role_refnum ltx_tag_bibitem">Schölkopf <span id="bib.bib38.2.2.1" class="ltx_text ltx_font_italic">et al</span>.,  (1999)</span>
<span class="ltx_bibblock"> Schölkopf,B., Burges,C., Smola,A. (1999) Advances in kernel methods: support vector learning, <span id="bib.bib38.3.1" class="ltx_text ltx_font_italic">MIT Press</span>.

</span>
</li>
</ul>
</section>
</article>
</div>
<div class="ar5iv-footer"><a href="/html/0704.2009" class="ar5iv-nav-button ar5iv-nav-button-prev">◄</a>
    <a class="ar5iv-home-button" href="/"><img height="40" alt="ar5iv homepage" src="/assets/ar5iv.png"></a>
    <a href="/feeling_lucky" class="ar5iv-text-button">Feeling<br>lucky?</a>
    <a href="/log/0704.2010" class="ar5iv-text-button ar5iv-severity-error">Conversion<br>report</a>
    <a class="ar5iv-text-button" target="_blank" href="https://github.com/dginev/ar5iv/issues/new?template=improve-article--arxiv-id-.md&title=Improve+article+0704.2010">Report<br>an issue</a>
    <a href="https://arxiv.org/abs/0704.2010" class="ar5iv-text-button arxiv-ui-theme">View&nbsp;original<br>on&nbsp;arXiv</a><a href="/html/0704.2011" class="ar5iv-nav-button ar5iv-nav-button-next">►</a>
</div><footer class="ltx_page_footer">
<a class="ar5iv-toggle-color-scheme" href="javascript:toggleColorScheme()" title="Toggle ar5iv color scheme"><span class="color-scheme-icon"></span></a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/license" target="_blank">Copyright</a>
<a class="ar5iv-footer-button" href="https://arxiv.org/help/policies/privacy_policy" target="_blank">Privacy Policy</a>

<div class="ltx_page_logo">Generated  on Sun Mar 10 12:20:08 2024 by <a target="_blank" href="http://dlmf.nist.gov/LaTeXML/" class="ltx_LaTeXML_logo"><span style="letter-spacing:-0.2em; margin-right:0.1em;">L<span class="ltx_font_smallcaps" style="position:relative; bottom:2.2pt;">a</span>T<span class="ltx_font_smallcaps" style="font-size:120%;position:relative; bottom:-0.2ex;">e</span></span><span style="font-size:90%; position:relative; bottom:-0.2ex;">XML</span><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="Mascot Sammy"></a>
</div></footer>
</div>

    <script>
      var canMathML = typeof(MathMLElement) == "function";
      if (!canMathML) {
        var body = document.querySelector("body");
        body.firstElementChild.setAttribute('style', 'opacity: 0;');
        var loading = document.createElement("div");
        loading.setAttribute("id", "mathjax-loading-spinner");
        var message = document.createElement("div");
        message.setAttribute("id", "mathjax-loading-message");
        message.innerText = "Typesetting Equations...";
        body.prepend(loading);
        body.prepend(message);

        var el = document.createElement("script");
        el.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js";
        document.querySelector("head").appendChild(el);

        window.MathJax = {
          startup: {
            pageReady: () => {
              return MathJax.startup.defaultPageReady().then(() => {
                body.removeChild(loading);
                body.removeChild(message);
                body.firstElementChild.removeAttribute('style');
              }); } } };
      }
    </script>
    <script>
    // Auxiliary function, building the preview feature when
    // an inline citation is clicked
    function clicked_cite(e) {
      e.preventDefault();
      let cite = this.closest('.ltx_cite');
      let next = cite.nextSibling;
      if (next && next.nodeType == Node.ELEMENT_NODE && next.getAttribute('class') == "ar5iv-bibitem-preview") {
        next.remove();
        return; }
      // Before adding a preview modal,
      // cleanup older previews, in case they're still open
      document.querySelectorAll('span.ar5iv-bibitem-preview').forEach(function(node) {
        node.remove();
      })

      // Create the preview
      preview = document.createElement('span');
      preview.setAttribute('class','ar5iv-bibitem-preview');
      let target = document.getElementById(this.getAttribute('href').slice(1));
      target.childNodes.forEach(function (child) {
        preview.append(child.cloneNode(true));
      });
      let close_x = document.createElement('button');
      close_x.setAttribute("aria-label","Close modal for bibliography item preview");
      close_x.textContent = "×";
      close_x.setAttribute('class', 'ar5iv-button-close-preview');
      close_x.setAttribute('onclick','this.parentNode.remove()');
      preview.append(close_x);
      preview.querySelectorAll('.ltx_tag_bibitem').forEach(function(node) {
        node.remove();
      });
      cite.parentNode.insertBefore(preview, cite.nextSibling);
      return;
    }
    // Global Document initialization:
    // - assign the preview feature to all inline citation links
    document.querySelectorAll(".ltx_cite .ltx_ref").forEach(function (link) {
      link.addEventListener("click", clicked_cite);
    });
    </script>
    </body>
</html>
